---
layout: default
title: Genome Assembly Class (2006)
---

<span class="contenthead">Genome Assembly Class (2006)</span><br><hr>

An 8 part lecture series given at the <a href="http://www.hawaii.edu"
target="_blank">University of Hawaii</a> between August 13 - 18 2006. The
lecture series covers the entire assembly process, from sequencing reactions, to
assembly, and finishing.  The discussion begins with an overview of the assembly
process, and its theoretical foundations of Lander-Waterman statistics and
Shortest-Common-Superstring.  Next there is an indepth discussion of the Celera
Assembler, covering the details of overlapping, unitigging, and scaffolding.
Next an Introduction to AMOS is given describing the motivation, framework, and
a brief discussion of some of the currently available tools.  Lecture 5
discusses current methods to discover mis-assemblies and the Interactive Genome
Visual Analytics tool Hawkeye, which acts as a visual portal to understanding
and validating your assembly data.  Next, I discuss two common problems in
assembly, that of base calling and trimming and describe AutoEditor and
AutoJoiner which are second generation assembly tools to address these areas.
Lecture 6 is provided by Adam Phillippy and covers all aspects of Whole Genome
Alignment, centered around the MUMmer suite.  The following lecture, also by
Adam Phillippy, describes the AMOScmp Comparative Assembler which uses MUMmer to
assemble genomes without the costly overlapping step even at extremely low
coverage.  The Final lecture acts as a summary for the class, and a checklist
for potential problem areas one might encounter during whole genome assembly.
<br><br>

<table class="intro">
<tr><td valign=top align=right>1.</td><td>
<a href="01. Assembly Intro.pdf">Genome Assembly: Assembly Concepts and Methods</a>:
Lander-Waterman Statistics, Shortest-Common-Superstring
</td></tr>

<tr><td valign=top align=right>2.</td><td>
<a href="02. Celera Assembler.pdf">Celera Assembler: Theory and Practice</a>:
runCA, overlapper, unitigging, scaffolding<br>
</td></tr>

<tr><td valign=top align=right>3.</td><td>
<a href="03. AMOS.pdf">AMOS: A Modular Open Source Assembler</a>:
AMOS overview, runAMOS, AMOS banks, Converters<br>
</td></tr>

<tr><td valign=top align=right>4.</td><td>
<a href="04. AMOS Assembly Validation.pdf">AMOS Assembly Validation and Visualization</a>:
Mate-pairs, SNPs, Coverage levels, Hawkeye, stitchContigs, Assembly Repair<br>
</td></tr>

<tr><td valign=top align=right>5.</td><td>
<a href="05. Improving Assembly Without Sequencing.pdf">Improving Assembly without Sequencing</a>:
Basecalling, AutoEditor, Trimming, AutoJoiner<br>
</td></tr>

<tr><td valign=top align=right>6.</td><td>
<a href="06. Whole Genome Alignment.pdf">Whole Genome Alignment</a>:
Alignment, Smith-Waterman, MUMmer, Suffix Trees<br>
</td></tr>

<tr><td valign=top align=right>7.</td><td>
<a href="07. AMOScmp.pdf">Comparative Genome Assembly</a>:
AMOScmp, MUMmer, reference assembly<br>
</td></tr>

<tr><td valign=top align=right>8.</td><td>
<a href="08. Assembly Checklist.pdf">Assembly Checklist</a>:
Sequencing, Libraries, Biases, Coverage, Unitigging, Scaffolding<br>
</td></tr>
</table>


<!--#include virtual="/template/footer.shtml" -->
