---
layout: default
title: Schatzlab - Welcome
---

<table class="intro">
<tr> <td width="544" valign="top">

<span class="contenthead">Welcome</span><br><hr>

Our main research interest is in understanding the structure and function of genomes, especially those of medical or 
agricultural importance. The core strength of our research is in developing novel algorithms and computational systems 
for large-scale biological sequence analysis, including leading algorithms for de novo genome assembly, variant detection, 
and related –omics assays. Using these advances we have contributed to the de novo genome assemblies of dozens of species; 
probed the sequence variations related to autism, cancer, and other human diseases; mapped the transcriptional and epigenetic 
profiles of tomatoes, corn, and other important plant species; and explored the role of microbes in different environments. 
In response to the deluge of biological sequence data we are now facing, we have also been at the forefront of distributed 
and parallel computing in genomics, and have pioneered the use of cloud computing as an enabling platform to 
address the big data challenges we are all facing.

<br><br>

Looking forward, we see ourselves at the intersection of biotechnology and algorithmics, developing systems for probing 
the structure and function of genomes using the best technologies possible.  Our expertise spans from low level computer 
architecture, through sequencing, de novo assembly, variant identification, transcriptome &amp; other -omics data
and up to machine learning approaches to build predictive models of diseases and treatment response.  In addition to ongoing 
projects in autism, cancer, and other human diseases, we also study agricultrual systems to better understand the underlying
genotype to phenotype relations.  Altogether, we intend to develop powerful new methods for analyzing large collections 
of genomes to address questions of disease, development, and evolution. Among other recognition, for this work I was granted 
an NSF CAREER award, a Sloan Foundation Fellowship, and was named a TIME100 recipient in 2022.  

<br><br>

<span class="contenthead">Recent News</span><hr>

<table class="intro" width=100%>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top><a href="http://schatz-lab.org/people/">Welcome Nicole Brown to the lab!</a><br>Aug 26, 2024<td> </tr> 
<tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top><a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btae493/7729118">ModDotPlot—rapid and interactive visualization of tandem repeats</a><br>Aug 7, 2024<td> </tr> 
<tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top><a href="https://www.science.org/doi/10.1126/science.ado1663">Convergent evolution of plant prickles by repeated gene co-option over deep time</a><br>Aug 6, 2024<td> </tr> 
<tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top><a href="https://www.biorxiv.org/content/10.1101/2024.07.31.605654v1">Complete sequencing of ape genomes</a><br>July 31, 2024<td> </tr> 
<tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top><a href="https://genome.cshlp.org/content/early/2024/07/01/gr.277950.123.abstract">Interspecies regulatory landscapes and elements revealed by novel joint systematic integration of human and mouse blood cell epigenomes</a><br>July 1, 2024<td> </tr> 
<tr><td height=8 colspan=2></td></tr>

<tr><td colspan=2 align=right>(<a href="/news/">past news</a>)</td></tr>
</table>

<br>

<span class="contenthead">Upcoming Events</span><br> <hr>

<table class="intro" width=100%>


<tr><td height=8 colspan=2><b>~~ 2024 ~~<b><br><br></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top> <a href="https://www.ashg.org/meetings/">American Society of Human Genetics</a><br>
  Denver CO, November. 2024</td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top> <a href="https://anvilproject.org/events/anvil2024-community-conference">AnVIL Community Conference</a><br>
  CSHL, NY. November 12-13 2024</td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top> <a href="https://meetings.cshl.edu/meetings.aspx?meet=DATA&year=24">Biological Data Science</a><br>
  CSHL, NY. November 13-16 2024</td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr><td height=8 colspan=2><b>~~ 2025 ~~<b><br><br></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top> <a href="https://www.intlpag.org">Plant and Animal Genomes Conference (PAG)</a><br>
  San Diego, CA. Jan 2025</td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top> <a href="http://www.agbt.org">Advances in Genome Biology and Technology (AGBT)</a><br> 
  Marco Island, FL. Feb 2025</td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top> <a href="http://www.agbt.org">Advances in Genome Biology and Technology (AGBT) Ag</a><br>
  Orlando, FL. March 2025</td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top> <a href="https://meetings.cshl.edu/meetings.aspx?meet=GENOME&year=25">Biology of Genomes</a><br>
  CSHL, NY. May 2025</td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top> <a href="https://galaxyproject.org/gcc/">Galaxy Community Conference</a><br>
  CSHL, NY. June 2025</td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr><td colspan=2 align=right>(<a href="/presentations/">presentation archive</a>)</td></tr>
</table>

</td>

<td width="230" valign="top" align=center>
<br>

<img src="/images/2015-Schatz.JPG" width="198" height="300" border="2"> 

<br><br>

<table class="intro"> <tr> <td align="left">
<b>Michael Schatz</b><br>
<br>
Bloomberg Distinguished<br>
Professor of Computer Science<br>
and Biology<br>
<br>
CS office and Mailing address:<br>
Johns Hopkins University<br>
Department of Computer Science<br>
3400 N Charles St<br>
Malone Hall 323<br>
Baltimore, MD 21218<br>
<br>
Department of Biology:<br>
UTL 391 (office) <br>
UTL 398 (lab and conference room) <br>
<br>
Cell: (703) 966-1987 <br>
E-mail: mschatz &lt;at&gt; cs.jhu.edu <br>
Twitter: <a href="http://twitter.com/mike_schatz">@mike_schatz</a> <br>
LinkedIn:  <a href="https://www.linkedin.com/in/mschatz/">mschatz</a> <br>
BlueSky: <a href="https://bsky.app/profile/mikeschatz.bsky.social">mikeschatz.bsky.social</a> <br>
</td> </tr> </table>

</td>
</tr>
</table>



