---
layout: default
title: Welcome
---

<table class="intro">
<tr> <td width="544" valign="top">

<span class="contenthead">Welcome</span><br><hr>

Our main research interest is in understanding the structure and function of genomes, especially those of medical or 
agricultural importance. The core strength of our research is in developing novel algorithms and computational systems 
for large-scale biological sequence analysis, including leading algorithms for de novo genome assembly, variant detection, 
and related â€“omics assays. Using these advances we have contributed to the de novo genome assemblies of dozens of species; 
probed the sequence variations related to autism, cancer, and other human diseases; mapped the transcriptional and epigenetic 
profiles of tomatoes, corn, and other important plant species; and explored the role of microbes in different environments. 
In response to the deluge of biological sequence data we are now facing, we have also been at the forefront of distributed 
and parallel computing in genomics, and have pioneered the use of cloud computing and Hadoop/MapReduce as an enabling platform to 
address the big data challenges we are all facing.

<br><br>

Looking forward, we see ourselves at the intersection of biotechnology and algorithmics, developing systems for probing 
the structure and function of genomes using the best technologies possible.  Our expertise spans from low level computer 
architecture, through sequencing, de novo assembly, variant identification, transcriptome &amp; other -omics data
and up to machine learning approaches to build predictive models of diseases and treatment response.  In addition to ongoing 
projects in autism &amp; other human diseases, and developmental plant biology, I was granted an NSF CAREER award 
to research new approaches for analyzing single molecule sequencing, especially for genome and transcriptome analysis of crop 
species.  Another recent thrust has been to develop algorithms for single cell analysis, especially to use copy number variations 
within individual tumor cells to examine how cancer progresses. Altogether, we intend to develop powerful new methods 
for analyzing large collections of genomes to address questions of disease, development, and evolution. 

<br><br>

<span class="contenthead">Recent News</span><hr>

<table class="intro" width=100%>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top><a href="https://academic.oup.com/gigascience/article/doi/10.1093/gigascience/gix045/3868354/16GT-a-fast-and-sensitive-variant-caller-using-a">16GT: a fast and sensitive variant caller using a 16-genotype probabilistic model </a> <br> June 15, 2017</td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top><a href="/presentations/2017/2017.05.22.JIMB.PerfectGenomeSequencing.pdf">In pursuit of perfect genome sequencing</a> <br> May 22, 2017</td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top><a href="https://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3861.html">Recurrent noncoding regulatory mutations in pancreatic ductal adenocarcinoma</a> <br> May 8, 2017</td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top><a href="https://academic.oup.com/bioinformatics/article/3089939/GenomeScope-Fast-reference-free-genome-profiling">GenomeScope: Fast reference-free genome profiling from short reads</a> <br> April 4, 2017</td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top><a href="http://www.nature.com/nmeth/journal/v14/n4/full/nmeth.4240.html">Nanopore sequencing meets epigenetics</a> <br> March 31, 2017</td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr><td colspan=2 align=right>(<a href="/news/">past news</a>)</td></tr>
</table>

<br>

<span class="contenthead">Upcoming Events</span><br> <hr>

<table class="intro" width=100%>

<tr><td height=8 colspan=2><b>~~ 2017 ~~<b><br><br></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top> <a href="http://programs.pacificbiosciences.com/l/1652/2017-03-25/3sn5p2">PacBio Users Meeting</a><br>
  Baltimore, MD. June 28, 2017</td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top> <a href="http://computationalgenomics.bioinformatics.ucla.edu/">UCLA Computational Genomics Summer Institute</a><br>
  UCLA. July 10, 2017</td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top> <a href="http://meetings.cshl.edu/NYQB17">Making Use of Emerging Technologies</a><br>
  CSHL. Aug 2, 2017</td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top> <a href="https://meetings.cshl.edu/meetings.aspx?meet=INFO&year=17">Genome Informatics</a><br>
  CSHL. Nov 1-4, 2017</td>
</tr> <tr><td height=8 colspan=2></td></tr>


<tr><td height=8 colspan=2><b>~~ 2018 ~~<b><br><br></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top> <a href="http://www.intlpag.org/">International Plant and Animal Genomes (PAG)</a><br>
  San Diego CA. Jan 13 - 17, 2018</td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top> <a href="http://www.agbt.org">Advances in Genome Biology and Technology (AGBT)</a><br>
  Marco Island, FL. Feb 14 - Feb 20, 2018</td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top> <a href="https://meetings.cshl.edu/meetings.aspx?meet=DATA&year=16">Biological Data Science</a><br>
  CSHL. Nov 7-10, 2018</td>
</tr> <tr><td height=8 colspan=2></td></tr>


<tr><td colspan=2 align=right>(<a href="/presentations/">presentation archive</a>)</td></tr>
</table>

</td>

<td width="230" valign="top" align=center>
<br>

<img src="/images/2015-Schatz.JPG" width="198" height="300" border="2"> 

<br><br>

<table class="intro"> <tr> <td align="left">
<b>Michael Schatz</b><br>
<br>
Bloomberg Distinguished<br>
Associate Professor of<br> 
Computer Science<br>
and Biology<br>
<br>
Johns Hopkins University<br>
Department of Computer Science<br>
3400 N Charles St<br>
Malone Hall 323<br>
Baltimore, MD 21211<br>
Cell: (703) 966-1987 <br>
E-mail: mschatz &lt;at&gt; cs.jhu.edu <br>
<br>
Adjunct Associate Professor of<br>
Quantitative Biology<br>
<br>
Cold Spring Harbor Laboratory<br>
One Bungtown Road<br>
Koch Building 1121<br>
Cold Spring Harbor, NY 11724<br>
Tel: (516) 367-5218 <br>
Fax: (516) 367-8380 <br>
E-mail: mschatz &lt;at&gt; cshl.edu <br>
Twitter: <a href="http://twitter.com/mike_schatz">@mike_schatz</a> <br>
</td> </tr> </table>

</td>
</tr>
</table>


