---
layout: default
title: Teaching - 2012
---


<a name="2012"><table width="100%" bgcolor="#e6e6e6"> <tr><th style="font-variant:small-caps; font-size:12pt; color:#003399;" align="left">2012</th></tr></table></a><br>

<!-- ########################################################################################### -->

<a name="2012QB"><h3>Quantitative Biology - 2012</h3></a> <hr>

<a href="QB Bioinformatics 1 - Exact Match.pdf">Lecture 1: Exact Matching</a><br>
<p>
Brute-force, Binary search, quicksort, binary trees, hashing, BWT
</p>

<a href="QB Bioinformatics 2 - Dynamic Programming.pdf">Lecture 2: Dynamic Programming and Applications</a><br>
<p>
Dynamic programming, Fibonacci, Longest-increasing subsequencing, Edit Distance, Smith-Waterman, Dynamic Time Warp, BLAST
</p>

<a href="QB Bioinformatics 3 - Graphs and Genomes.pdf">Lecture 3: Graphs and Genomes</a><br>
<p>
Graph representations, network motifs, modularity, breath-first-search, depth-first-search, eulerian path, traveling salesman, genome assembly, assembly validation.
</p>

<a href="QB Bioinformatics 4 - Gene Finding and HMMs.pdf">Lecture 4: Gene Finding and HMMs</a><br>
<p>
Prokaryotic and Eukaryotic gene finding, Glimmer, GlimmerHMM, Forward algorithm, Viterbi, GHMM
</p>


<a href="BWT.m">Code Sample: BWT Construction and Searching</a><br>
<p>
Construct the BWT for a genome, and compute exact matches against it.
</p>

<a href="BWT.pdf">Handouts on the BWT</a><br>
<p>
Details on BWT Construction and searching
</p>


<br>
<br>

<!-- ########################################################################################### -->

<a name="2012QBBootcamp"><h3>Quantitative Biology Bootcamp - 2012</h3></a> <hr>

<a href="QB Bootcamp 1 - Programming 101.pdf">Lecture 1: Molecular Biology, Computers, and Unix</a><br>
<p>
Milestones in Molecular Biology and the rise of sequencing.
Overview of computer systems, introduction to unix and scripting.
</p>

<a href="QB Bootcamp 2 - Sequence Analysis.pdf">Lecture 2: Sequence Alignment and Computational Thinking</a><br>
<p>
Introduction to Alignment and Algorithms, Suffix Arrays, Binary Search, QuickSort, 
Burrows-Wheeler Transform, Bowtie.
</p>

<a href="QB Bootcamp 3 - Genome Resources.pdf">Lecture 3: Genomic Resources</a><br>
<p>
NCBI, UCSC, CSHL Meetings and Courses, Galaxy.
</p>

<a href="QB Bootcamp 4 - ORC Exercise.pdf">Lecture 4: Dicovering Origins of Replication</a><br>
<p>
Background on tracking DNA replication with next-gen sequencing,
Walk-through of analysis steps, Visualization of discovered replication sites.
</p>

<a href="tutorial.m">Lecture 5: Matlab Tutorial</a><br>
<p>
Introduction to Matlab, working with variables and matrices, ploting, working with
strings, working with structures, working with CEL arrays.
</p>

<br>
<br>

<!-- ########################################################################################### -->

<a name="2012SBUGenetics"><h3>SBU Graduate Genetics - 2012</h3></a> <hr>

<a href="SBU.Schatz.1.Sequence Alignment.pdf">Lecture 1: Sequence Alignment and Computational Thinking</a><br>
<p>
Introduction to Alignment and Algorithms, Suffix Arrays, Binary Search, QuickSort, 
Burrows-Wheeler Transform, Bowtie.
</p>

<a href="SBU.Schatz.2.Genome Assembly.pdf">Lecture 2: Whole Genome Assembly and Alignment</a>
<p>
Theory and Practice of whole genome assembly and alignment highlighting the Celera Assembler.
</p>

<br>
<br>


<!-- ########################################################################################### -->

<a name="2012CSHLSeq"><h3>CSHL Advanced Sequencing Course - 2012</h3></a> <hr>

<a href="http://bioinformatics.bc.edu/wikis/csh12/index.php/Main_Page">Course Wiki</a>
<p>
Schedule of presentations and slides.
</p>

<a href="CSHL.Sequencing/Whole Genome Assembly and Alignment.pdf">Whole Genome Assembly and Alignment</a>
<a href="CSHL.Sequencing/Whole Genome Assembly and Alignment.ppt">[ppt]</a><br>
<p>
Theory and Practice of whole genome assembly and alignment highlighting Allpaths, SOAPdenovo, and the Celera Assembler.
</p>

<a href="CSHL.Sequencing/Assembly Tutorial.pdf">Assembly Tutorial</a>
<a href="CSHL.Sequencing/Assembly Tutorial.ppt">[ppt]</a><br>
<p>
Tutorial on using Allpaths for whole genome assembly
</p>

<a href="CSHL.Sequencing/asm.tgz">Challenge Problem Data</a>
<a href="CSHL.Sequencing/AHBC01.fasta.gz">[ref genome]</a><br>
<p>
Data for challenge problem in the tutorial
</p>


<br>
<br>


<!-- ########################################################################################### -->

<a name="2012URP"><h3>Undergraduate Research Program in Bioinformatics - 2012</h3></a> <hr>

<a href="2012.URP.CS Fundamentals.pdf">Lecture 1: Sequence Alignment and Computational Thinking</a><br>
<p>
In this class we explored the problem of finding exact occurrences of a query
sequence in a large genome or database of sequences.  Under this theme, we
started by analyzing the brute force approach introducing the concepts of
algorithm, complexity analysis, and E-values.  Next we discussed suffix arrays
as an index for accelerating the search, including analyzing the performance of
binary search. We also considered two traditional algorithms for sorting
(Selection Sort versus QuickSort) and their relative performance.  In the second
half of the class we discussed finding approximate occurrences of a short query
sequence in a large genome or database of sequences. We first defined the
problem by considering various metrics of an approximate occurrence such as
hamming distance, or edit distance.  We then considered different methods for
computing inexact alignments including brute force global &amp; local
alignments, and seed-and-extend algorithms. Finally we discussed Bowtie as a
Burrows-Wheeler transform based short read mapping algorithm for discovering
alignments to reference genome.
</p>

<br>

<a href="2012.URP.Sequencing Lunch.pdf">Lecture 2: Sequencing Pitfalls</a><br>
<p>
In this session we reviewed the currently available sequencing technologies and best practices,
focusing on the widely used Illumina sequencing platform, the up and coming PacBio
sequencing platform, and the recently announced Oxford Nanopore instruments. Special
attention was placed on the complexities and biases with Illumina.
</p>

<br>

<a href="2012.URP.Graphs and Genomes.pdf">Lecture 3: Graphs and Genomes</a><br>
<p>
The theme of this class was graphs and methods for graph analysis. The emphasis
was on genome assembly but included a discussion of other biological networks
including PPI networks, regulation networks, neuron interaction networks, and
cell cycle graphs. In the class, we considered fundamental properties of graphs,
such as nodes, edges, degrees, and shortest paths. We then examined in detail
algorithms for searching graphs with a  with breadth-first-search, and then
approaches for finding minimum cost paths through weighed graphs (traveling
salesman problem), including exhaustive search, greedy algorithms, and
branch-and-bound. This lead to a discussion of the intractable nature of
NP-complete problems, and reviewed several important examples (vertex cover,
clique finding, knapsack problem, Hamiltonian cycle).
</p>

<br>
<br>

