---
layout: default
title: Schatzlab - Research
---

<span class="contenthead">Research</span><br><hr>

<table width="100%">
<tr>
<td width="75%">

<span class="intro">

<p>
My lab focuses on the development of scalable algorithms and systems to analyze biological sequence data, concentrating 
on the alignment, assembly, and analysis of high-throughput DNA sequencing reads. This work is critical to unlocking 
the potential of large-scale sequencing as a tool for genetics research. Addressing the magnitude of the challenge, 
my work includes capitalizing on the latest advances in distributed and parallel computing to advance the 
state of the art in bioinformatics and genomics.
</p>


<h3>Research Interests</h3>
<ul>
  <li>Genome Assembly &amp; Validation</li>
  <li>Sequence Alignment</li>
  <li>High Performance and Multi-Core Computing</li>
  <li>Human Genetics</li>
  <li>Environmental Sampling &amp; Metagenomics</li>
  <li>Scientific Visualization</li>
  <li>Cloud Computing</li>
</ul>

<h3>Affiliations</h3>
<ul>
  <li><a href="http://www.cshl.edu/research/quantitative-biology">Simons Center for Quantitative Biology</a></li>
  <li><a href="http://webpro.cshl.edu/cancercenter/">Cancer Center at Cold Spring Harbor Laboratory</a></li>
  <li><a href="http://www.cshl.edu/gradschool">Watson School of Biological Sciences</a></li>
  <li><a href="http://www.jhu.edu/">Johns Hopkins University</a></li>
  <li><a href="http://www.cs.sunysb.edu/">Stony Brook University</a></li>
  <li><a href="http://www.stonybrook.edu/commcms/laufer/">Laufer Center for Physical and Quantitative Biology</a></li>
  <li><a href="http://www.gigasciencejournal.com/">Gigascience</a>
  <li><a href="http://www.genomebiology.com/">Genome Biology</a>
  <li><a href="http://www.cell.com/cell-systems/home">Cell Systems</a>
</ul>

</span>

</td>
<td>
<a href="/publications/"><img border=0 src="/images/collage.png"></a>
</td>
</tr>
</table>

<br>
<br>



<span class="contenthead">Selected Software Packages</span><br><hr>

<table class="intro">
<tr><td width="128"><a href="http://amos.sourceforge.net">AMOS</a></td><td width="606">A fast and flexible API for genome assembly and manipulations</td></tr>
<tr><td><a href="http://assemblytics.com">Assemblytics</a></td><td>Interactive analysis of variants within an assembly</td></tr>
<tr><td><a href="http://bowtie-bio.sf.net/">Bowtie</a></td><td>Ultrafast, memory efficient short read aligner</td></tr>
<tr><td><a href="http://wgs-assembler.sourceforge.net">Celera Assembler</a></td><td> The program used to assemble the human genome at Celera Genomics in 2001</td></tr>
<tr><td><a href="http://cloudburst-bio.sourceforge.net">CloudBurst</a></td><td> Highly Sensitive Short Read Mapping with MapReduce</td></tr>
<tr><td><a href="http://contrail-bio.sourceforge.net/">Contrail</a></td><td>Assembly of Large Genomes using Cloud Computing</td></tr>
<tr><td><a href="http://bowtie-bio.sourceforge.net/crossbow/">Crossbow</a></td><td> Whole Genome Resequencing Analysis in the Clouds</td></tr>
<tr><td><a href="https://github.com/PacificBiosciences/FALCON">FALCON</a></td><td> Phased Diploid Genome Assembly with PacBio sequencing reads</td></tr>
<tr><td><a href="http://code.google.com/p/genome-indexing/">Genome-indexing</a></td><td>Rapid Burrows-Wheeler Transform Construction with MapReduce</td></tr>
<tr><td><a href="http://genomescope.org">GenomeScope</a></td><td>Estimate genomic properties from unassembled sequencing reads</td></tr>
<tr><td><a href="http://qb.cshl.edu/ginkgo//">Ginkgo</a></td><td>Interactive analysis and assessment of single-cell copy-number variations</td></tr>
<tr><td><a href="http://amos.sourceforge.net/hawkeye">Hawkeye</a></td><td> Genome Assembly Viewer and Analysis Tool.</td></tr>
<tr><td><a href="https://github.com/aquaskyline/LRSIM">LRSim</a></td><td> Linked Read Simulator</td></tr>
<tr><td><a href="http://mummer.sourceforge.net">MUMmer</a></td><td> A modular system for the rapid whole genome alignment of finished or draft sequence</td></tr>
<tr><td><a href="http://mummergpu.sourceforge.net">MUMmerGPU</a></td><td>High-throughput Sequence Alignment on the GPU using <a href="http://www.nvidia.com/cuda">CUDA</a> GPGPU API from <a href="http://www.nvidia.com">nVidia</a></td></tr>
<tr><td><a href="http://schatzlab.cshl.edu/data/nanocorr/">NanoCorr</a></td><td>Hybrid Error Correction of Oxford Nanopore Sequencing Reads</a></td></tr>
<tr><td><a href="https://github.com/philres/ngmlr">NGM-LR</a></td><td>Fast and Accurate Mapping of Long Reads</a></td></tr>
<tr><td><a href="http://www.phylotrac.org">PhyloTrac</a></td><td>Visualization and Analysis tool for the PhlyoChip</td></tr>
<tr><td><a href="http://www.cbcb.umd.edu/software/quake/">Quake</a></td><td>Quality guided correction and filtration of errors in short reads. </td></tr>
<tr><td><a href="http://genomeribbon.com">Ribbon</a></td><td>Visualization of Long Read Alignments</td></tr>
<tr><td><a href="http://scalpel.sourceforge.net/">Scalpel</a></td><td>Indel variant analysis of short-read sequencing data </td></tr>
<tr><td><a href="https://sourceforge.net/projects/splitmem/">SplitMem</a></td><td>Pan Genome Analysis with suffix skips </td></tr>
<tr><td><a href="https://github.com/fritzsedlazeck/Sniffles">Sniffles</a></td><td>Detection of Structural Variations from Long Reads</td></tr>
<tr><td><a href="https://github.com/fritzsedlazeck/SURVIVOR">SURVIVOR</a></td><td>Detection and Analysis of Structural Variations</td></tr>
<tr><td><a href="http://teaser.cibiv.univie.ac.at/">Teaser</a></td><td>Fast personalized benchmarks and optimization for NGS read mapping</td></tr>
<tr><td><a href="https://github.com/hanfang/Topsorter">TopSorter</a></td><td>SV Identification and Validation using 10X Genomics Linked Reads</td></tr>
</table>

