---
layout: default
title: Beyond the Genome 2013
---

<!-- ########################################################################################### -->

<span class="contenthead">2013 Beyond the Genome Informatics Challenge: Metagenomics Variant Encoding</span><br><hr>
<h3>Sven-Eric Schelhorn and Michael Schatz</h3>

<p>
The goal of this challenge is to identify a secret message we encoded as
variants into a metagenomics sample.  The sample was generated by mixing
portions of the reference sequences of several microbial species. Sequence reads
were simulated from these portions. Within each portion of a
referencesequence, a foreigninsert (i.e, not originating from any of the microbial
species) was placed. These inserts encode a message.

See the <a href="BTG13.Challenge.pdf">presentation</a> for all the details. Download the data here: <a href="btg2013.tgz">btg2013.tgz</a>
</p>

<p>
You can use the included dna-encode.pl script to decode the message.  It uses
the algorithm defined in 
<a href="https://www.sciencemag.org/content/early/2012/08/15/science.1226355.full">GM Church, Y Gao, and S Kosuri. (2012)
  Next-Generation Digital Information Storage in DNA. Science. DOI: 10.1126/science.1226355</a>
</p>

<p>
If you have Perl installed on your computer, you can get the documentation for the
dna-encode.pl script with the following command:
</p>

<pre>
  $ perldoc dna-encode.pl
</pre>

<p>
You can get a brief synopsis of the command with this command:
<p>

<pre>
  $ perl dna-encode.pl --help
</pre>

<p>
The types of reads in each FASTQ (.fq) file are described in detail below.
</p>

<pre>
dna-encode.pl           Perl script to encode/decode text to/from DNA

sh_end_{1,2}.fastq.gz   Paired end read data from the mixed references, 
                        fastq format, 2x250bp from  1000+/-50bp fragments

lo_end_{1,2}.fastq.gz   Paired end read data from the mixed references, 
                        fastq format, 2x150bp from  5300+/-500bp fragments
</pre>

<br><br>

<h3>Hints</h3>
<hr>

<ol>
  <li>Itâ€™s a metagenomic sample- Choose your tools accordingly.</li>
  <li>After you identified an insert, you need to identify the insert wildtype.There are several ways to distinguish it from the variants. BLAST, consensus, pairwise
  similarities...</li>
  <li>NCBI Blast may be unreliable due to the Government Shutdown. If yes, try to use the public BLAST server at EBI/EMBL (WU-BLAST).</li>
</ol>


<br><br> <br><br> <br><br> <br><br> <br><br> <br><br> <br><br> <br><br> <br><br> <br><br> <br><br> <br><br> <br><br> <br><br> 

<span class="contenthead">Solution Guide</span><br><hr>

<p>
 Solution Guide available <a href="BTG13.Solution.pdf">here</a>
</p>

<br><br>

