---
layout: default
title: Schatzlab - Selected Posters
---

<table cellpadding=4 border=0 bgcolor="#e6e6e6" class="intro" width=80% align=center>
  <tr>
    <td width="25%" bgcolor="#f9f9f9" align=center><a href="/publications">publications</a> </td>
    <td width="25%" bgcolor="#f9f9f9" align=center><a href="/publications/posters">posters</a> </td>
    <td width="25%" bgcolor="#f9f9f9" align=center><a href="/publications/reviews">press</a> </td>
    <td width="25%" bgcolor="#f9f9f9" align=center><a href="/presentations">presentations</a> </td>
  </tr>
</table>

<br>
<br>


<!------------------------------------------------ ------ -->
<a name="posters"><table width="100%" bgcolor="#e6e6e6"> <tr><th style="font-variant:small-caps; font-size:12pt; color:#003399;" align="left">Posters</th></tr></table></a><br>

<table class="intro">

<tr><td colspan=2><br><h3>2016</h3><hr></td></tr>
<!------------------------------------------------ ------ -->

<tr><td valign=top align=right>36.</td><td>
<a href="2016/2016.AGBT.SV with 3rdgen.pdf">Detection of Structural Variants using third generation sequencing</a>. Presented by Fritz Sedlazeck et al <br>
<a href="http://www.agbt.org">Advances in Genome Biology and Technology (AGBT)</a>, Orlando, FL. Feb 12, 2016.
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>35.</td><td>
<a href="2016/2016.AGBT.GenomeScope.pdf">GenomeScope: Fast genome analysis from unassembled short reads</a>. Presented by Greg Vurture et al <br>
<a href="http://www.agbt.org">Advances in Genome Biology and Technology (AGBT)</a>, Orlando, FL. Feb 12, 2016.
</td></tr>

<tr><td><br></td></tr>

<tr><td colspan=2><br><h3>2015</h3><hr></td></tr>
<!------------------------------------------------ ------ -->

<tr><td valign=top align=right>34.</td><td>
<a href="2015/2015.GI.Macro.pdf">Genome and transcriptome of the regeneration-competent flatworm, Macrostomum lignano</a>. Presented by Kaja Wasik et al <br>
<a href="https://meetings.cshl.edu/meetings.aspx?meet=info&year=15">Genome Informatics</a>, Cold Spring Harbor, NY. Oct 28-30, 2015.
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>33.</td><td>
<a href="2015/2015.GI.NextGenMap-LR.pdf">NextGenMap-LR: Highly accurate read mapping of third generation sequencing reads for improved structural variation analysis</a>. Presented by Phillip Rescheneder et al <br>
<a href="https://meetings.cshl.edu/meetings.aspx?meet=info&year=15">Genome Informatics</a>, Cold Spring Harbor, NY. Oct 28-30, 2015.
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>32.</td><td>
<a href="2015/2015.GI.Resurgence of Ref Quality Genomes.pdf">The Resurgence of Reference Quality Genomes</a><br>
<a href="https://meetings.cshl.edu/meetings.aspx?meet=info&year=15">Genome Informatics</a>, Cold Spring Harbor, NY. Oct 28-30, 2015.
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>31.</td><td>
<a href="2015/2015.GI.Sniffles.pdf">Detection of Structural Variants using third generation sequencing</a>. Presented by Fritz Sedlazeck et al<br>
<a href="https://meetings.cshl.edu/meetings.aspx?meet=info&year=15">Genome Informatics</a>, Cold Spring Harbor, NY. Oct 28-30, 2015.
</td></tr>

<tr><td><br></td></tr>

<tr><td colspan=2><br><h3>2014</h3><hr></td></tr>
<!------------------------------------------------ ------ -->

<tr><td valign=top align=right>30.</td><td>
<a href="2014.BoG.PacBio Assembly.pdf">Near perfect assemblies of eukaryotic genomes using PacBio long read sequencing</a><br>
<a href="http://www.cshl.edu">Biology of Genomes</a>, Cold Spring Harbor, NY. May 10, 2014.
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>29.</td><td>
<a href="2014.AGBT.Scalpel.pdf">Accurate detection of de novo and transmitted INDELs within exome-capture data using micro-assembly</a><br>
<a href="http://www.agbt.org">Advances in Genome Biology and Technology (AGBT)</a>, Marco Island, FL. Feb 14, 2014.
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>28.</td><td>
<a href="2014.AGBT.PacBio Assembly.pdf">Error correction and assembly complexity of single molecule sequencing reads: How long is long enough?</a><br>
Authored by Hayan Lee, James Gurtowski, Shinjae Yoo, Shoshana Marcus, W. Richard McCombie, and Michael Schatz<br>
<a href="http://www.agbt.org">Advances in Genome Biology and Technology (AGBT)</a>, Marco Island, FL. Feb 14, 2014.
</td></tr>

<tr><td><br></td></tr>

<tr><td colspan=2><br><h3>2013</h3><hr></td></tr>
<!------------------------------------------------ ------ -->

<tr><td valign=top align=right>27.</td><td>
<a href="2013.GenomeInformatics.PBCorrection.pdf">An improved method for hybrid correction of long-read, low-identity sequencing data</a><br>
Authored by James Gurtowski, Hayan Lee, and Michael Schatz<br>
<a href="http://meetings.cshl.edu/meetings/2013/info.shtml">Genome Informatics</a>, Cold Spring Harbor, NY. Oct 31, 2013.
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>26.</td><td>
<a href="2013.BOG.COINVGH.pdf">COIN-VGH: Sensitive and specific virtual genomic hybridization to pinpoint denovo variations</a><br>
Authored by Laura Gomez-Romero et al.<br>
<a href="http://meetings.cshl.edu/meetings/2013/genome13.shtml">Biology of Genomes</a>, Cold Spring Harbor, NY. May 7-11, 2013.
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>25.</td><td>
<a href="2013.BOG.Metassembly.pdf">De novo genome metassembly</a><br>
Authored by Alejandro H. Wences, Paul Baranay, Michael C Schatz<br>
<a href="http://meetings.cshl.edu/meetings/2013/genome13.shtml">Biology of Genomes</a>, Cold Spring Harbor, NY. May 7-11, 2013.
</td></tr>

<tr><td colspan=2><br><h3>2012</h3><hr></td></tr>
<!------------------------------------------------ ------ -->

<tr><td valign=top align=right>24.</td><td>
<a href="">FASTG: Representing the true information content of a genome assembly</a><br>
Authored by Iain MacCallum et al.<br>
<a href="http://meetings.cshl.edu/meetings/genome12.shtml">Biology of Genomes</a>, Cold Spring Harbor, NY. May 8-12, 2012.
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>23.</td><td>
<a href="">Assessing the role of de novo gene-killers in the incidence of autism</a><br>
Authored by Ivan Iossifov et al.<br>
<a href="http://meetings.cshl.edu/meetings/genome12.shtml">Biology of Genomes</a>, Cold Spring Harbor, NY. May 8-12, 2012.
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>22.</td><td>
<a href="">Detection and validation of de novo mutations in exome-capture data using micro-assembly</a><br>
Authored by Giuseppe Narzisi et al.<br>
<a href="http://meetings.cshl.edu/meetings/genome12.shtml">Biology of Genomes</a>, Cold Spring Harbor, NY. May 8-12, 2012.
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>21.</td><td>
<a href="">Hybrid Error Correction and De Novo Assembly of Single-Molecule Sequencing Read </a><br>
Authored by Adam Phillippy et al.<br>
<a href="http://agbt.org">AGBT</a>, Marco Island, FL, Feb 15-18, 2012
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>20.</td><td>
<a href="">Combining Sequences from Different Sequencing Platforms (Hiseq, Miseq, PacBio) to Improve de novo Genome Assembly.</a><br>
Authored by Eric Antoniou et al.<br>
<a href="http://agbt.org">AGBT</a>, Marco Island, FL, Feb 15-18, 2012<br>
</td></tr>

<tr><td><br></td></tr>

<tr><td colspan=2><br><h3>2011</h3><hr></td></tr>
<!------------------------------------------------ ------ -->

<tr><td></td></tr>

<tr><td valign=top align=right>19.</td><td>
<a href="2011.GenomeInformatics.Jnomics.pdf">Jnomics-A cloud-scale sequence analysis suite.</a><br>
Authored by Matt Titmus, Sneh Lata, Eric Antonious, James Gurtowski, W. Richard McCombie, and Michael Schatz<br>
<a href="http://meetings.cshl.edu/meetings/info11.shtml">Genome Informatics</a>, Cold Spring Harbor NY, Nov 3, 2011.
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>18.</td><td>
<a href="2011.Informatics.Metassembler.pdf">Metassembler: A pipeline for improving de novo genome assembly.</a><br>
Authored by Paul Baranay, Scott Emrich, and Michael Schatz.<br>
<a href="http://meetings.cshl.edu/meetings/info11.shtml">Genome Informatics</a>, Cold Spring Harbor NY, Nov 3, 2011.
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>17.</td><td>
<a href="">Rare de novo and transmitted mutations in autistic spectrum disorders.</a><br>
Authored by Michael Ronemus et al.<br>
<a href="http://www.ichg2011.org/">ASHG</a>, Montreal, CA. Oct 11-15. 
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>16.</td><td>
<a href="">Combining short (Illumina) and long (PacBio) NGS reads to improve de novo genome assemblies</a><br>
Authored by Michael Schatz, Melissa delaBastide, Stephanie Muller, Laura Gelley, Eric Antoniou, and W. Richard McCombie<br>
<a href="http://www.ichg2011.org/">ASHG</a>, Montreal, CA. Oct 11-15. 
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>15.</td><td>
<a href="2011.PersonalGenomes.Dark_Matter.pdf">Genome Mappability Analyzer:Characterizing the dark matter of the human genome</a><br>
Authored by <a href="http://schatzlab.cshl.edu/people/hlee/">Hayan Lee</a>, and Michael Schatz<br>
<a href="http://meetings.cshl.edu/meetings/person11.shtml">Personal Genomes</a>, CSHL, Cold Spring Harbor, NY. Sept 30 - Oct 2, 2011.
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>14.</td><td>
<a href="">MicroSeq:High-throughput, genome-wide microsatellite genotyping in individual genomes</a><br>
Authored by <a href="http://schatzlab.cshl.edu/people/mbekritsky/">Mitch Bekritsky</a>, Jennifer Troge, Dan Levy, Michael Wigler, and Michael Schatz<br>
<a href="http://meetings.cshl.edu/meetings/person11.shtml">Personal Genomes</a>, CSHL, Cold Spring Harbor, NY. Sept 30 - Oct 2, 2011.
</td></tr>

<tr><td><br></td></tr>

<tr><td colspan=2><br><h3>2010</h3><hr></td></tr>
<!------------------------------------------------ ------ -->

<tr><td></td></tr>


<tr><td valign=top align=right>13.</td> <td>
<a href="2010.ISMB.Quake.pdf">Quality guided correction and filtration of errors in short reads</a>. <br>
Authored by <a href="http://www.cbcb.umd.edu/~dakelley/">David Kelley</a>, Michael Schatz, and Steven Salzberg<br>
<a href="http://www.iscb.org/ismb2010">ISMB 2010</a>, Boston, MA. July 11-13, 2010.<br>
<p>High-throughput sequencing technologies such as that offered by Illumina have permeated nearly all areas of biological research.  Illuminas technology produces sequencing reads of 35-125 bp that may have base calling errors at rates as high as 1-2%.  These errors create difficulties for downstream sequence analysis tasks, such as detecting overlaps for genome assembly or aligning reads to a reference genome for SNP detection.</p>
<p>Due to its lower cost, deep coverage of the genome is generally possible using Illumina sequencing.  Past work has shown that errors can be identified in a set of reads with deep coverage by first counting k-mers, and then considering k-mers with coverage less than some threshold to be artifacts of sequencing errors.  Previous methods to correct reads with errors have searched for a minimal set of edits to the read that ensure all k-mers have sufficient coverage.  We demonstrate that due to biases with respect to where errors occur in the read and the likelihood of specific nucleotide to nucleotide errors, such approaches are prone to mistakes that introduce corrected sequencing reads that fail to represent any true genomic fragment.</p>
<p>We introduce a program named Quake that corrects errors in reads using a more robust model of sequencing errors than previous approaches.  By using read quality values and learning the rates at which nucleotides are called as errors to different nucleotides, Quake achieves near perfect accuracy on simulated data.  We also demonstrate the role of error correction with Quake in improving assembly and SNP detection.</p> 
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>12.</td> <td>
<a href="BOG.2010 - Contrail.pdf">De Novo Assembly of Large Genomes with Cloud Computing</a>. <br>
Co-authored with Dan Sommer, David Kelley, and Mihai Pop.<br>
<a href="http://meetings.cshl.edu/meetings/genome10.shtml">Biology of Genomes 10</a>, Cold Spring Harbor, NY. May 14, 2010.<br>
<p>The first step towards analyzing a previously unsequenced organism is to assemble the genome by merging together the sequencing reads into progressively longer contig sequences. New assemblers such as Velvet, Euler-USR, and SOAPdenovo attempt to reconstruct the genome by constructing, simplifying, and traversing the de Bruijn graph of the reads. These assemblers have successfully assembled small genomes from short reads, but have had limited success scaling to larger mammalian-sized genomes, mainly because they require memory and compute resources that are unobtainable for most users. </p>
<p>Addressing this limitation, we are developing a new assembly program Contrail (http://contrail-bio.sf.net), which uses the Hadoop/MapReduce distributed computing framework to enable de novo assembly of large genomes. MapReduce was developed by Google to simplify their large data processing needs by scaling computation across many computers, and the open-source version called Hadoop (http://hadoop.apache.org) is becoming a de facto standard for large data analysis, especially in so called cloud computing environments where compute resources are rented on demand. For example, we have also successfully leveraged Hadoop and the Amazon Elastic Compute Cloud for Crossbow (http://bowtie-bio.sf.net/crossbow) to accelerate short read mapping and genotyping, allowing quick (< 4 hours), cheap (< $100), and accurate (> 99% accuracy) genotyping of an entire human genome from 38-fold short read coverage.<p> 
<p>Similar to other leading short read assemblers, Contrail relies on the graph-theoretic framework of de Bruijn graphs. However, unlike these programs Contrail uses Hadoop to parallelize the assembly across many tens or hundreds of computers, effectively removing memory concerns and making assembly feasible for even the largest genomes.  Preliminary results show contigs produced by Contrail are of similar size and quality to those generated by other leading assemblers when applied to small (bacterial) genomes, which scales far better to large genomes. We are also developing extensions to Contrail to efficiently compute a traditional overlap-graph based assembly of large genomes within Hadoop, a strategy that will be especially valuable as read lengths increase to 100bp and beyond. </p> 
</td></tr>

<tr><td><br></td></tr>

<tr><td colspan=2><br><h3>2009</h3><hr></td></tr>
<!------------------------------------------------ ------ -->
<tr><td></td></tr>


<tr><td valign=top align=right>11.</td> <td>
<a href="SC09-Crossbow.pdf">Whole Genome Resequencing Analysis in the Clouds</a>. <br>
Co-authored with Ben Langmead, Jimmy Lin, Mihai Pop, and Steven Salzberg.<br>
<a href="http://sc09.supercomputing.org">SC09</a>. Portland, OR. Nov. 17, 2009.<br>
<p> Biological researchers have a critical need for highly efficient methods for analyzing vast quantities of DNA resequencing data. For example, the 1000 Genomes Project aims to characterize the variations within 1000 human genomes by aligning and analyzing billions of short DNA sequences from each individual. Each genome will utilize ~100GB of compressed sequence data and require ~400 hours computation. Crossbow is our new high-throughout pipeline for whole genome resequencing analysis. It combines Bowtie, an ultrafast and memory efficient short read aligner, and SoapSNP, a highly accurate Bayesian genotyping algorithm, within the distributed processing framework Hadoop to accelerate the computation using many compute nodes. Our results show the pipeline is extremely efficient, and can accurately analyze an entire genome in one day on a small 10-node local cluster, or in one afternoon and for less than $250 in the Amazon EC2 cloud. Crossbow is available open-source at <a href="http://bowtie-bio.sf.net/crossbow">http://bowtie-bio.sourceforge.net/crossbow</a>.  </p> 
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>10.</td> <td>
<a href="2009-10-05.CLuE.Poster.pdf">Commodity Computing in Genomics Research</a>. <br>
Co-authored with Mihai Pop and Dan Sommer.<br>
<a href="https://wiki.umiacs.umd.edu/ccc/index.php/CLuE_PI_Meeting_2009">NSF CLuE PI Meeting</a>. Mountain View CA. Oct 5, 2009. <br> 
<p> In the next few years the data generated by DNA sequencing instruments around the world will exceed petabytes, surpassing the amounts of data generated by large-scale physics experiments such as the Hadron Collider. Even today, the terabytes of data generated every few days by each sequencing instrument test the limits of existing network and computational infrastructures. Our project is aimed at evaluating whether cloud computing technologies and the MapReduce/Hadoop infrastructure can enable the analysis of the large data-sets being generated. We will report on initial results in two specific applications: human genotyping and genome assembly using next generation sequencing data.  </p>
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>9.</td> <td>
<a href="Crossbow_WABI_Sept2009.pdf">Human SNPs from short reads in hours using cloud computing</a>.<br>
Co-authored with Ben Langmead, Jimmy Lin, Mihai Pop, and Steven Salzberg. <br>
Awarded Best Poster at <a href="http://www.genomics.jhu.edu/symposium.html">3rd Annual Young Investigator Symposium on Genomics & Bioinformatics</a>. <a href="http://www.jhu.edu">Johns Hopkins University</a>. Baltimore, MD. Sept 25, 2009. <br>
<p> As growth in short read sequencing throughput vastly outpaces improvements in microprocessor speed, there is a critical need to accelerate common tasks, such as short read alignment and SNP calling, via large-scale parallelization.  </p>
<p> Crossbow is a software tool that combines the speed of the short read aligner <a href="http://bowtie-bio.sf.net">Bowtie</a> and the accuracy of the <a href="http://soap.genomics.org.cn/soapsnp.html">SOAPsnp</a> consensus and SNP caller within a cloud computing environment. Crossbow aligns reads and makes highly accurate SNP calls from a dataset comprising 38-fold coverage of the human genome in under 1 day on a local 40 core cluster, and under 3 hours using a 320-core cluster rented from Amazon’s Elastic Compute Cloud (EC2) service.  Crossbow’s ability to run on EC2 means that users need not own or operate an expensive computer cluster in order to run Crossbow.  Crossbow is available at <a href="http://bowtie-bio.sf.net/crossbow">http://bowtie-bio.sf.net/crossbow</a> under the Artistic license.  </p>
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>8.</td> <td>
<a href="Crossbow_WABI_Sept2009.pdf">Human SNPs from short reads in hours using cloud computing</a>.<br>
Co-authored with Ben Langmead, Jimmy Lin, Mihai Pop, and Steven Salzberg.<br>
<a href="http://www.wabi09.org">WABI 2009</a>. Philadelphia, PA. Sept 12, 2009.<br>
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>7.</td> <td>
<a href="BOG-ParallelAssembler.pdf">Towards a de novo short read assembler for large genomes using cloud computing</a>.<br>
<a href="http://meetings.cshl.edu/meetings/genome09.shtml">Biology of Genomes 09</a>, Cold Spring Harbor, NY. May 7, 2009.<br> 
<p> The massive volume of data and short read lengths from next generation DNA sequencing machines has spurred development of a new class of short read genome assemblers. Several of the new assemblers, such as Velvet and Euler-USR, model the assembly problem as constructing, simplifying, and traversing the de Bruijn graph of the read sequences, where nodes in the graph represent k-mers in the reads, with edges between nodes for consecutive k-mers. This approach has many advantages for these data, such as efficient computation of overlapping reads and robust handling of sequencing errors, and has demonstrated success for assembling small to moderately sized genomes. However, this approach is computationally challenging to scale to mammalian-sized genomes because it requires constructing and manipulating a graph far larger than can fit into memory.  </p> 
<p> MapReduce was developed at Google for parallel computation on their extremely large data sets, including their database of more than 1 trillion web pages.  Computation in MapReduce is structured into 2 main phases: the map phase and the reduce phase, which act together to construct a large distributed hash table of key-value pairs in a map phase, and then evaluate a function on each bucket of the hash table in the reduce phase. The power of MapReduce is dozens or hundreds of map and reduce instances can execute in parallel, enabling efficient computation even on terabyte and petabyte sized data sets.  </p> 
<p> Drawing on the success of CloudBurst, a MapReduce-based short read mapping algorithm capable of mapping millions of reads to the human genome with high sensitivity, we have developed a MapReduce-based short read assembler that shows tremendous potential for enabling de novo assembly of mammalian-sized genomes.  The deBruijn graph is constructed with MapReduce by emitting and then grouping key-value pairs (ki,ki+1) between successive k-mers in the read sequences.  After construction, MapReduce is used again to remove spurious nodes and edges from the graph caused by sequencing error in the reads, and to compress simple chains of nodes into long sequence nodes representing the unambiguous regions of the genome between repeat boundaries. The resulting graph is a small fraction of the size of the original deBruijn graph, and is output in a format compatible with other short read assemblers for additional analysis.  </p>
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>6.</td> <td>
<a href="">Improving the genome sequence of D. simulans via co-assembly of multiple strains</a>.<br>
Co-authored with Adam Phillippy, Timur Chabuk, Bo Liu and Steven Salzberg.<br>
<a href="http://meetings.cshl.edu/meetings/genome09.shtml">Biology of Genomes 09</a>, Cold Spring Harbor, NY. May 7, 2009.<br> 
<p>Using the closely related D. melanogaster as a reference, we aggressively co-assembled the seven sequenced strains of D. simulans using our comparative assembly program, AMOScmp.  The AMOScmp contigs were then passed to Celera Assembler to assemble and scaffold alongside reads that failed to match the reference genome. Our improved co-assembly increases depth of coverage threefold over the original assembly and contains thousands of additional genes. These results show that comparative assembly is a promising means for assembling diverse population samples and outperforms traditional assembly in quality and the number of genes it is able to successfully recover. In addition, when combined with overlap-based assembly, comparative assembly can succeed even for reference genomes of a different species.</p>
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>5.</td> <td>
<a href="">A whole-genome assembly of the domestic cow, B. taurus</a>.<br>
Co-authored with Aleksey Zimmin, Steven Salzberg, et al. <br>
<a href="http://meetings.cshl.edu/meetings/genome09.shtml">Biology of Genomes 09</a>, Cold Spring Harbor, NY. May 7, 2009.<br> 
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>4.</td> <td>
<a href="PSB2009-Summarization.pdf">Better Modules in Protein-Protein Interaction Networks</a>.<br>
Co-authored with Saket Navlakha and Carl Kingsford.<br>
<a href="http://psb.stanford.edu/">Pacific Symposium on Biocomputing</a>, Hawaii. Jan. 10, 2009.<br>
</td></tr>

<tr><td><br></td></tr>

<tr><td colspan=2><br><h3>2008 and earlier</h3><hr></td></tr>
<!------------------------------------------------ ------ -->
<tr><td></td></tr>

<tr><td valign=top align=right>3.</td> <td>
<a href="Forensics-BOG2008.pdf">Genome Assembly Forensics: Finding the Elusive Mis-assembly</a>.<br>
Co-authored with Adam Phillippy and Mihai Pop.<br>
<a href="http://meetings.cshl.edu/meetings/genome08.shtml">Biology of Genomes 08</a>, Cold Spring Harbor, NY. May 9, 2008.<br>
<p> Since the initial "draft" sequence of the human genome was released in 2001, it has become clear that it was not an entirely accurate reconstruction of the genome. Despite significant advances in sequencing and assembly since then, genome sequencing continues to be an inexact process. Genome finishing and validation have remained a largely manual and expensive process, and consequently, many genomes are presented as draft assemblies. Draft assemblies are of unknown quality and potentially contain significant mis-assemblies, such as collapsed repeats, sequence excision, or artificial rearrangements. Too often these assemblies are judged only by contig size, with larger contigs preferred without regard to quality, because it has been difficult to gauge large scale assembly quality.  </p>
<p> Our new automated software pipeline, amosvalidate, addresses this deficiency and automatically detects mis-assemblies using a battery of known and novel assembly quality metrics. Instead of focusing on a single assembly characteristic as other validation approaches have tried, the power of our approach comes from leveraging multiple sources of evidence. amosvalidate statistically analyzes mate-pair orientations and separations, repeat content, depth-of-coverage, correlated polymorphisms in the read alignments, and read alignment breakpoints to identify structurally suspicious regions of the assembly. The suspicious regions identified by individual metrics are then clustered and combined to identify (with high confidence) regions that are mis-assembled. This approach is necessary for accurately detecting mis-assemblies because each of the individual characteristics has unavoidable natural variation, but, when considered together, have greatly increased analysis power. Furthermore, our pipeline can easily be adjusted to analyze assemblies utilizing new sequencing technologies where some metrics are unreliable or not available, such as base pair quality or mate pairs.  </p>
<p> Our validation pipeline provides a robust measure of assembly quality that goes beyond the simple measures commonly reported. Evaluation of the pipeline has shown it to be highly sensitive for mis-assembly detection, and has revealed mis-assemblies in both draft and finished genomes. This is particularly troubling as scientists move away from the "gene by gene" paradigm and attempt to understand the global organization of genomes. Without a correct genome sequence or even a clear understanding of the errors present, such studies may draw incorrect conclusions. Our goals are to help scientists locate mis-assembled regions of an assembly and help them correct those regions by focusing their efforts where it is needed most. amosvalidate is compatible with many common assembly formats and is released open-source at http://amos.sourceforge.net.  </p>
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>2.</td> <td>
<a href="MUMmerGPUPoster-MicrobialGenomics.pdf">High-throughput sequence alignment using Graphics Processing Units</a>.<br> 
Co-authored with <a href="http://www.cbcb.umd.edu/~cole">Cole Trapnell</a>.<br>
15th Annual Microbial Genomes Conference. College Park, MD. Sept 17, 2007.
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>1.</td> <td>
<a href="RECOMB-ImprovingGenomeAssembly.pdf">Improving Genome Assembly without Sequencing</a>.<br>
Co-authored with Steven Salzberg, Arthur Delcher, and Pawel Gajer. <br>
<a href="http://www.broadinstitute.org/events/recomb2005/">RECOMB 2005</a>. Cambridge, MA. May 5, 2005.
</td></tr>

</table>

<br>

