---
layout: default
title: Schatzlab - Teaching
---

<li><a href="">2017</a></li>
<ul>
  <li><a href="https://github.com/schatzlab/appliedgenomics">JHU Applied Comparative Genomics</a></li>
</ul>

<li><a href="">2016</a></li>
<ul>
  <li><a href="http://gaming.jhu.edu/~phf/2016/fall/cs226/schedule.html">JHU CS 600.226 Data Structures</a></li>
</ul>

<li><a href="/teaching/2015/">2015</a></li>
<ul>
  <li><a href="/teaching/guest.html">CSHL Genome Access</a></li>
</ul>

<li><a href="/teaching/2014/">2014</a></li>
<ul>
  <li><a href="/teaching/2014/index.html#2014AdvSeq">CSHL Advanced Sequencing Course</a></li>
  <li><a href="/teaching/2014/index.html#2014WSBS">CSHL WSBS Genomics and Quantitative Biology</a></li>
  <li><a href="/teaching/2014/index.html#2014Plants">CSHL Frontiers and Techniques in Plant Science</a></li>
  <li><a href="/teaching/2014/index.html#2014URP">CSHL Undergraduate Research Program in Bioinformatics</a></li>
  <li><a href="/teaching/guest.html">DNA Learning Center: Bringing next-gen sequence analysis into undergraduate education</a></li>
  <li><a href="/teaching/guest.html">CSHL Genome Access</a></li>
</ul>

<li><a href="/teaching/2013/">2013</a></li>
<ul>
  <li><a href="/teaching/2013/index.html#2013QB">CSHL Quantitative Biology</a></li>
  <li><a href="/teaching/2013/index.html#2013QBBootcamp">CSHL Quantitative Biology Bootcamp</a></li>
  <li><a href="/teaching/2013/index.html#2013AdvSeq">CSHL Advanced Sequencing Course</a></li>
  <li><a href="/teaching/2013/index.html#2013P4B">CSHL Programming for Biology</a></li>
  <li><a href="/teaching/guest.html">CSHL Genome Access</a></li>
  <li><a href="/teaching/2013/index.html#2013URP">CSHL Undergraduate Research Program in Bioinformatics</a></li>
  <li><a href="/teaching/2013/index.html#2013SBUGG">SBU Gradudate Genetics</a></li>
  <li><a href="/teaching/2013/index.html#2013SBUCS">SBU Intro to Computational Biology</a></li>
  <li><a href="/teaching/2013/index.html#2013iPQB">SBU Introduction to Physical and Quantitative Biology</a></li>
</ul>
 
<li><a href="/teaching/2012/">2012</a></li>
<ul>
  <li><a href="/teaching/2012/index.html#2012QB">CSHL Quantitative Biology</a></li>
  <li><a href="/teaching/2012/index.html#2012QBBootcamp">CSHL Quantitative Biology Bootcamp</a></li>
  <li><a href="/teaching/2012/index.html#2012CSHLSeq">CSHL Advanced Sequencing Course</a></li>
  <li><a href="/teaching/2012/index.html#2012URP">CSHL Undergraduate Research Program in Bioinformatics</a></li>
  <li><a href="/teaching/2012/index.html#2012SBUGenetics">SBU Graduate Genetics</a></li>
</ul>

<li><a href="/teaching/2011/">2011</a></li>
<ul>
  <li><a href="/teaching/2011/index.html#2011QB">Quantitative Biology &amp; Bioinformatics</a></li>
  <li><a href="/teaching/2011/index.html#2011FIG">Frontiers in Genomics, National University of Mexico</a></li>
  <li><a href="/teaching/2011/index.html#2011URP">Undergraduate Research Program in Bioinformatics</a></li>
</ul>

<li><a href="/teaching/2010/">2010</a></li>
<ul>
  <li><a href="/teaching/2010/index.html#2010QB">Quantitative Biology &amp; Bioinformatics</a></li>
</ul>

<li><b>Special Topics</b></li>
<ul>
  <li><a href="/teaching/AssemblyClass/">genome assembly class</a></li>
  <li><a href="/teaching/guest.html">guest lectures</a></li>
</ul>


<br><br><br>

<!-- ########################################################################################### -->

<span class="contenthead">Exercises</span><br><hr>

<table class="intro" width=100% border=0>

<tr><td colspan=3><b>Beyond the Genome Challenges</b></td></tr>
<tr> <td valign=top width=175><a href="/teaching/exercises/btg11">BTG11: Viral Insertion</a></td><td>&nbsp;</td> <td valign=top>Identify the virus in a cancer exome sequencing project</td> </tr>
<tr> <td valign=top width=175><a href="/teaching/exercises/btg12">BTG12: Digital Encoding</a></td><td>&nbsp;</td> <td valign=top>Identify the secret message inserted into a microbial genome</td> </tr>
<tr> <td valign=top width=175><a href="/teaching/exercises/btg13">BTG13: Metagenomics Variants</a></td><td>&nbsp;</td> <td valign=top>Identify the secret message encoded as variants within a metagenomics sample</td> </tr>
<tr><td colspan=3>&nbsp;</td></tr>

<tr><td colspan=3><b>Genomics</b></td></tr>
<tr> <td valign=top width=175><a href="/teaching/exercises/mapping">Read Mapping</a></td><td>&nbsp;</td> <td valign=top>Read mapping, identifying variants, running BWA/SAMTools</td> </tr>
<tr> <td valign=top width=175><a href="/teaching/exercises/rnaseq">RNA-seq Analysis</a></td><td>&nbsp;</td> <td valign=top>Read mapping, computing expression, running Bowtie/Tophat/Cufflinks</td> </tr>
<tr> <td valign=top width=175><a href="/teaching/exercises/assembly">Genome assembly</a></td><td>&nbsp;</td> <td valign=top>De novo assembly and error correction, whole genome alignment</td> </tr>
<tr> <td valign=top width=175><a href="http://nbviewer.ipython.org/url/schatzlab.cshl.edu/teaching/exercises/coverage/planesweepcoverage.ipynb">Coverage analysis</a></td><td>&nbsp;</td> <td valign=top>Coverage analysis using plane-sweep algorithms</td> </tr>
<tr><td colspan=3>&nbsp;</td></tr>

<tr><td colspan=3><b>Probability and Statistics</b></td></tr>
<tr> <td valign=top width=175><a href="http://nbviewer.ipython.org/url/schatzlab.cshl.edu/teaching/exercises/stats/1.Rolling%20a%20die.ipynb">Rolling a die</a></td><td>&nbsp;</td> <td valign=top>Uniform Random Probability</td> </tr>
<tr> <td valign=top width=175><a href="http://nbviewer.ipython.org/url/schatzlab.cshl.edu/teaching/exercises/stats/2.Flipping%20a%20coin.ipynb">Flipping a coin</a></td><td>&nbsp;</td> <td valign=top>Binimal and Normal Distributions</td> </tr>
<tr> <td valign=top width=175><a href="http://nbviewer.ipython.org/url/schatzlab.cshl.edu/teaching/exercises/stats/3.Throwing%20Marbles%20into%20Jars.ipynb">Marbles in jars</a></td><td>&nbsp;</td> <td valign=top>Poisson Distribution</td> </tr>
<tr> <td valign=top width=175><a href="http://nbviewer.ipython.org/url/schatzlab.cshl.edu/teaching/exercises/stats/4.Throwing%20darts.ipynb">Throwing darts</a></td><td>&nbsp;</td> <td valign=top>Exponential Distribution</td> </tr>
<tr><td colspan=3>&nbsp;</td></tr>

<tr><td colspan=3><b>Computing</b></td></tr>
<tr> <td valign=top width=175><a href="/teaching/exercises/python/python1.zip">Python Basics 1</a></td><td>&nbsp;</td> <td valign=top>Introduction to python, variables, lists, conditions, loops</td> </tr> 
<tr> <td valign=top width=175><a href="/teaching/exercises/python/python2.zip">Python Basics 2</a></td><td>&nbsp;</td> <td valign=top>Brute force search, dictionaries, motif finding</td> </tr> 
<tr> <td valign=top width=175><a href="/teaching/exercises/hadoop">Hadoop</a></td><td>&nbsp;</td> <td valign=top>Getting started with Hadoop, kmer counting and word analysis</td> </tr> 
<tr><td colspan=3>&nbsp;</td></tr>

</table>

<br><br><br>

<!-- ########################################################################################### -->

<span class="contenthead">Essential Resoures</span><br><hr>

<table class="intro" width=100% border=0>

<tr><td colspan=3><b>Online Resources</b></td></tr>
<tr><td valign=top width=175><a href="http://www.ncbi.nlm.nih.gov/">NCBI</a></td>           <td>&nbsp;</td> <td valign=top>PubMed, Genomes, Genbank, SRA</td> </tr>
<tr><td valign=top width=175><a href="http://genome.ucsc.edu/">UCSC Genome Browser</a></td> <td>&nbsp;</td> <td valign=top>Interactive genome plots, coverage, annotations</td> </tr>
<tr><td valign=top width=175><a href="http://usegalaxy.org">Galaxy</a></td>                 <td>&nbsp;</td> <td valign=top>Interactive genome analysis (<a href="http://genomics.cshl.edu">CSHL version</a>)</td> </tr>
<tr><td valign=top width=175><a href="http://seqanswers.com">SeqAnswers</a></td>            <td>&nbsp;</td> <td valign=top>Forums and tutorials for genomics analysis</td> </tr>
<tr><td valign=top width=175><a href="http://www.khanacademy.org/">Khan Academy</a></td>    <td>&nbsp;</td> <td valign=top>Videos and Tutorials on many science and mathematics related topics</td> </tr>
<tr><td valign=top width=175><a href="http://citeulike.org">CiteULike</a></td>              <td>&nbsp;</td> <td valign=top>Social networking for citations and papers</td> </tr>
<tr><td valign=top width=175><a href="http://rosalind.info">Rosalind</a></td>               <td>&nbsp;</td> <td valign=top>Programming and Bioinformatics exercises</td> </tr>
<tr><td valign=top width=175><a href="https://github.com/BenLangmead/comp-genomics-class/blob/master/README.md">JHU Comp Genomics</a></td>              <td>&nbsp;</td> <td valign=top>iPython Notebooks of basic string algorithms</td> </tr>
<tr><td colspan=3>&nbsp;</td></tr>


<tr><td colspan=3><b>FASTQ trimming and error correction</b></td></tr>
<tr><td valign=top width=175><a href="http://hannonlab.cshl.edu/fastx_toolkit/">FASTX-Tools</a></td>    <td>&nbsp;</td> <td valign=top>Filter and trim fastq files</td> </tr>
<tr><td valign=top width=175><a href="http://www.cbcb.umd.edu/software/quake/index.html">Quake</a></td> <td>&nbsp;</td> <td valign=top>Error Correction of Short Reads</td> </tr>
<tr><td colspan=3>&nbsp;</td></tr>

<tr><td colspan=3><b>Aligners and variant callers</b></td></tr>
<tr><td valign=top width=175><a href="http://bowtie-bio.sf.net">Bowtie</a>/<a href="http://bowtie-bio.sf.net/bowtie2">Bowtie2</a></td>                         <td>&nbsp;</td> <td valign=top>Ultra-fast short read mapping with the BWT</td> </tr>
<tr><td valign=top width=175><a href="http://bio-bwa.sourceforge.net/">BWA</a></td>                     <td>&nbsp;</td> <td valign=top>Feature rich short read alignment</td> </tr>
<tr><td valign=top width=175><a href="http://samtools.sourceforge.net/">SAMTools</a></td>               <td>&nbsp;</td> <td valign=top>SAM/BAM alignment filtering and scoring</td> </tr>
<tr><td valign=top width=175><a href="http://picard.sourceforge.net/">Picard</a></td>                   <td>&nbsp;</td> <td valign=top>SAM/BAM alignment filtering</td> </tr>
<tr><td valign=top width=175><a href="http://code.google.com/p/bedtools/">BedTools</a></td>             <td>&nbsp;</td> <td valign=top>Slice and dice genomics intervals</td> </tr>
<tr><td valign=top width=175><a href="http://www.broadinstitute.org/gsa/wiki/index.php/Home_Page">GATK</a></td>  <td>&nbsp;</td> <td valign=top>Variant calling</td> </tr>
<tr><td valign=top width=175><a href="http://code.google.com/p/hydra-sv/">Hydra</a></td>                <td>&nbsp;</td> <td valign=top>Structural Variations calling with discordant pair analysis</td> </tr>
<tr><td colspan=3>&nbsp;</td></tr>

<tr><td colspan=3><b>RNA-seq Analysis</b></td></tr>
<tr><td valign=top width=175><a href="http://tophat.cbcb.umd.edu">TopHat</a></td>                <td>&nbsp;</td> <td valign=top>Spliced alignment for RNA-seq</td> </tr>
<tr><td valign=top width=175><a href="http://cufflinks.cbcb.umd.edu">Cufflinks</a></td>          <td>&nbsp;</td> <td valign=top>Transcript assembly and quantification</td> </tr>
<tr><td valign=top width=175><a href="http://trinityrnaseq.sourceforge.net/">Trinity</a></td>    <td>&nbsp;</td> <td valign=top>De novo transcript assembly and quantification</td> </tr>
<tr><td colspan=3>&nbsp;</td></tr>

<tr><td colspan=3><b>Assembly and Whole Genome Alignment</b></td></tr>
<tr><td valign=top width=175><a href="http://www.broadinstitute.org/software/allpaths-lg/blog/">ALLPATHS-LG</a></td> <td>&nbsp;</td> <td valign=top>Assembly of short reads</td> </tr>
<tr><td valign=top width=175><a href="http://soap.genomics.org.cn/soapdenovo.html">SOAPdenovo</a></td>               <td>&nbsp;</td> <td valign=top>Assembly of short reads</td> </tr>
<tr><td valign=top width=175><a href="http://wgs-assembler.sf.net">Celera Assembler</a></td>                         <td>&nbsp;</td> <td valign=top>Assembly of long reads</td> </tr>
<tr><td valign=top width=175><a href="http://mummer.sourceforge.net/">MUMmer</a></td>                                <td>&nbsp;</td> <td valign=top>Whole Genome Alignment</td> </tr>
<tr><td valign=top width=175><a href="http://amos.sourceforge.net/">AMOS</a></td>                                    <td>&nbsp;</td> <td valign=top>Assembly and assembly related tools </td> </tr>
<tr><td colspan=3>&nbsp;</td></tr>

<tr><td colspan=3><b>Statistics and Visualization</b></td></tr>
<tr><td valign=top width=175><a href="https://www.enthought.com/products/canopy/">Canopy</a></td> <td>&nbsp;</td> <td valign=top>Python Integrated Development Environment</td> </tr>
<tr><td valign=top width=175><a href="http://matplotlib.org/">matplotlib</a></td>               <td>&nbsp;</td> <td valign=top>Visualization in python</td> </tr>
<tr><td valign=top width=175><a href="http://www.r-project.org/">R</a></td>               <td>&nbsp;</td> <td valign=top>Statistics and visualization</td> </tr>
<tr><td valign=top width=175><a href="http://circos.ca/">Circos</a></td>                  <td>&nbsp;</td> <td valign=top>Circular plots of genomes and other data </td> </tr>
<tr><td valign=top width=175><a href="http://www.graphviz.org/">Graphviz</a></td>         <td>&nbsp;</td> <td valign=top>Layout and render graphs</td> </tr>
<tr><td valign=top width=175><a href="http://www.cs.waikato.ac.nz/ml/weka/">Weka</a></td> <td>&nbsp;</td> <td valign=top>Data Mining and classification</td> </tr>
<tr><td valign=top width=175><a href="http://www.broadinstitute.org/igv/">IGV</a></td>    <td>&nbsp;</td> <td valign=top>Interactive Genomics Viewer for short reads</td> </tr>
<tr><td colspan=3>&nbsp;</td></tr>

</table>

<br><br>

