---
layout: default
title: Schatzlab Presentations
---

<table cellpadding=4 border=0 bgcolor="#e6e6e6" class="intro" width=80% align=center>
  <tr>
    <td width="25%" bgcolor="#f9f9f9" align=center><a href="/publications">publications</a> </td>
    <td width="25%" bgcolor="#f9f9f9" align=center><a href="/publications/posters">posters</a> </td>
    <td width="25%" bgcolor="#f9f9f9" align=center><a href="/publications/reviews">reviews</a> </td>
    <td width="25%" bgcolor="#f9f9f9" align=center><a href="/presentations">presentations</a> </td>
  </tr>
</table>

<br>
<br>

<a name="reviews"><table width="100%" bgcolor="#e6e6e6"> <tr><th style="font-variant:small-caps; font-size:12pt; color:#003399;" align="left">Presentations</th></tr></table></a>

<table class="intro">

<tr><td colspan=2><br><h3>2017</h3><hr></td></tr>

<tr><td valign=top align=right>119.</td> <td>
<a href="/presentations/2017/2017.05.22.JIMB.PerfectGenomeSequencing.pdf">In pursuit of perfect genome sequencing</a>.<br>
<a href="http://jimb.stanford.edu/">Joint Institute for Metrology in Biology</a> Stanford, Ca. May 22, 2017</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>118.</td> <td>
<a href="/presentations/2017/2017.02.15.ENTexAGBT.pdf">Personalized Phased Diploid Genomes of the EN-Tex Samples</a>.<br>
<a href="https://www.agbt.org">Advances in Genome Biology and Technology (AGBT)</a> Hollywood, FL. Feb 15, 2017</td></tr>

<tr><td><br></td></tr>

<tr><td colspan=2><br><h3>2016</h3><hr></td></tr>

<tr><td valign=top align=right>117.</td> <td>
<a href="/presentations/2016/2016.10.28.BIODATA.PacBioSV.pdf">Accurate and fast detection of complex and nested structural variations using long read technologies</a>. Presented by Fritz Sedlazeck<br>
<a href="https://meetings.cshl.edu/meetings.aspx?meet=DATA&year=16">Biological Data Science</a> Cold Spring Harbor, NY. Oct 27, 2016</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>116.</td> <td>
<a href="/presentations/2016/2016.10.27.BIODATA16.scikitribo.pdf">Scikit-ribo reveals precise codon-level translational control by dissecting ribosome pausing and codon elongation</a>. Presented by Han Fang<br>
<a href="https://meetings.cshl.edu/meetings.aspx?meet=DATA&year=16">Biological Data Science</a> Cold Spring Harbor, NY. Oct 27, 2016</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>115.</td> <td>
<a href="/presentations/2016/2016.02.12.AGBT.scikit-ribo.Robust Modeling of translational control.pdf">Scikit-ribo - Accurate A-site prediction and robust modeling of translational control</a>. Presented by Han Fang<br>
<a href="http://www.agbt.org">Advances in Genome Biology and Technology (AGBT)</a> Orlando, FL. Feb 10-13, 2016</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>114.</td> <td>
<a href="/presentations/2016/2016.02.12.AGBT.SplitThreader.Analysis of Highly Rearranged Genomes.pdf">SplitThreader: A graphical algorithm for analysis of highly rearranged and amplified cancer genomes</a>. Presented by Maria Nattestad<br>
<a href="http://www.agbt.org">Advances in Genome Biology and Technology (AGBT)</a> Orlando, FL. Feb 10-13, 2016</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>113.</td> <td>
<a href="/presentations/2016/2016.02.12.AGBT.GECCO.Recurrent noncoding mutations in PDAC.pdf">Recurrent noncoding regulatory mutations in pancreatic ductal adenocarcinoma</a>. Presented by Tyler Garvin<br>
<a href="http://www.agbt.org">Advances in Genome Biology and Technology (AGBT)</a> Orlando, FL. Feb 10-13, 2016</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>112.</td> <td>
<a href="/presentations/2016/2016.01.12.PAG.Resurgence of Ref Quality Genomes.pdf">The Resurgence of Reference Quality Genomes</a>.<br>
<a href="http://www.intlpag.org/">Plant and Animal Genomes Conference (PAG XXIV)</a> San Diego, CA. Jan 12, 2016</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>111.</td> <td>
<a href="/presentations/2016/2016.01.12.PAG.Structural Variations.pdf">Analysis of Structural Variations using 3rd gen Sequencing</a>.<br>
<a href="http://www.intlpag.org/">Plant and Animal Genomes Conference (PAG XXIV)</a> San Diego, CA. Jan 12, 2016</td></tr>

<tr><td><br></td></tr>

<tr><td colspan=2><br><h3>2015</h3><hr></td></tr>

<tr><td valign=top align=right>110.</td> <td>
<a href="/presentations/2015/2015.10.29.GI.SKBR3.pdf">Comprehensive Genome and Transcriptome Structural Analysis of a Breast Cancer Cell Line using PacBio Long Read Sequencing</a>. Presented by Maria Nattestad<br>
<a href="https://meetings.cshl.edu/meetings.aspx?meet=info&year=15">Genome Informatics</a> Cold Spring Harbor, NY. Oct 28-31, 2015</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>109.</td> <td>
<a href="/presentations/2015/2015.10.29.GI.Ginkgo.pdf">Ginkgo—Interactive analysis and quality assessment of single-cell CNV data</a>. Presented by Robert Aboukhalil <br>
<a href="https://meetings.cshl.edu/meetings.aspx?meet=info&year=15">Genome Informatics</a> Cold Spring Harbor, NY. Oct 28-31, 2015</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>108.</td> <td>
<a href="/presentations/2015/2015.10.29.GI.Scikit-ribo.pdf">Scikit-ribo - Accurate A-site prediction and robust modeling of translational control</a>. Presented by Han Fang<br>
<a href="https://meetings.cshl.edu/meetings.aspx?meet=info&year=15">Genome Informatics</a> Cold Spring Harbor, NY. Oct 28-31, 2015</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>107.</td> <td>
<a href="/presentations/2015/2015.10.22.JHU Genomics Symposium.pdf">Single Cell and Single Molecule Approaches to Studying Cancer</a>.<br>
<a href="http://genomics.jhu.edu/symposium.html">JHU Genomics Symposium</a> Baltimore, MD. Oct 22, 2015</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>106.</td> <td>
<a href="/presentations/2015/2015.10.05.ASHG.SKBR3.pdf">Comprehensive Genome and Transcriptome Structural Analysis of a Breast Cancer Cell Line using PacBio Long Read Sequencing</a>. Presented by Maria Nattestad<br>
<a href="http://www.ashg.org/2015meeting/">American Society of Human Genetics (ASHG)</a> Baltimore, MD. Oct 7, 2015</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>105.</td> <td>
<a href="/presentations/2015/2015.05.22.NYU Symposium.pdf">Single cell and single molecule approaches for studying cancer</a>. <br>
<a href="http://biology.as.nyu.edu/object/2015.cgsb.symposium">NYU Genomics Symposium</a> New York, NY. May 22, 2015</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>104.</td> <td>
<a href="/presentations/2015/2015.04.20.Laufer Center Cancer Genomics.pdf">The resurgence of reference quality genomes</a>.<br>
<a href="http://laufercenter.stonybrook.edu/">Laufer Center for Quantitative Biology</a> Stony Brook University, Stony Brook, NY. April 20, 2015</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>103.</td> <td>
<a href="/presentations/2015/2015.04.09.UMinn Resurgence of Ref Quality Genomes.pdf">The resurgence of reference quality genomes</a>.<br>
<a href="http://twin-cities.umn.edu/">University of Minnesota</a> St. Paul, MN. April 9, 2015</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>102.</td> <td>
<a href="/presentations/2015/2015.04.07.LIIGH.Schatzlab.pdf">Algorithms for studying the structure and function of genomes</a>.<br>
<a href="http://www.campusjuriquilla.unam.mx/">UNAM LIIGH</a> Queretaro, Mexico. April 7, 2015</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>101.</td> <td>
<a href="/presentations/2015/2015.03.27.Simons.Single Cell Single Molecule.pdf">Algorithms for Single Cell and Single Molecule Biology</a>.<br>
<a href="https://www.simonsfoundation.org/lecture/genomics-in-single-cells-and-microbiomes/">Simons Foundation</a> New York, NY. March 27, 2015</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>100.</td> <td>
<a href="/presentations/2015/2015.03.24.VIZBI2015 Ginkgo.pdf">Single Cell Copy Number Analysis</a>. Presented by Robert Aboukhalil<br>
<a href="http://www.vizbi.org/2015/">VIZBI 2015</a> Cambridge, MA. March 24, 2015</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>99.</td> <td>
<a href="/presentations/2015/2015.02.28.AGBT Nanocorr Assembly.pdf">Error Correction and Assembly of Oxford Nanopore Sequencing</a>. Presented by James Gurtowski<br>
<a href="http://www.agbt.org/">AGBT</a> Marco Island, Fl. Feb 27, 2015</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>98.</td> <td>
<a href="/presentations/2015/2015.02.27.AGBT PacBio SKBR3.pdf">PacBio Long Read Sequencing and Structural Analysis of a Breast Cancer Cell Line</a>. Presented by W. Richard McCombie<br>
<a href="http://www.agbt.org/">AGBT</a> Marco Island, Fl. Feb 27, 2015</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>97.</td> <td>
<a href="/presentations/2015/2015.02.06.JHU CS.pdf">Part II: Algorithms for studying the structure and function of genomes (CS)</a><br>
<a href="http://www.jhu.edu/">Johns Hopkins University</a> Baltimore, MD. Feb 6, 2015</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>96.</td> <td>
<a href="/presentations/2015/2015.02.05.JHU Bio.pdf">Part I: Algorithms for studying the structure and function of genomes (Biology)</a><br>
<a href="http://www.jhu.edu/">Johns Hopkins University</a> Baltimore, MD. Feb 5, 2015</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>95.</td> <td>
<a href="/presentations/2015/2015.01.13.PAG.Resurgence of Ref Quality Genomes.pdf">The resurgence of reference quality genomes.</a><br>
<a href="http://www.intlpag.org/">Plant and Animal Genome XXIII (PAG)</a> San Diego, CA. Jan 13, 2015</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>94.</td> <td>
<a href="/presentations/2015/2015.01.13.PAG.Bioinformatics.Oxford Nanopore Assembly.pdf">Hybrid Error Correction and De Novo Assembly with Oxford Nanopore.</a><br>
<a href="http://www.intlpag.org/">Plant and Animal Genome XXIII (PAG)</a> San Diego, CA. Jan 13, 2015</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>93.</td> <td>
<a href="/presentations/2015/2015.01.11.PAG.Sugarcane_Assembly_Hayan_Lee.pdf">Sugarcane Genome De Novo Assembly Challenges</a> Presented by Hayan Lee<br>
<a href="http://www.intlpag.org/">Plant and Animal Genome XXIII (PAG)</a> San Diego, CA. Jan 13, 2015</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>92.</td> <td>
<a href="/presentations/2015/2015.01.06.Penn State.Resurgence of Ref Qual Sequencing.pdf">The resurgence of reference quality assemblies using 3rd generation sequencing.</a><br>
<a href="http://www.psu.edu">Penn State</a> State College, PA. Jan 6, 2015</td></tr>

<tr><td><br></td></tr>

<tr><td colspan=2><br><h3>2014</h3><hr></td></tr>

<tr><td valign=top align=right>91.</td> <td>
<a href="/presentations/2014/2014.12.09 Resurgence of Ref Qual Sequencing.pdf">The resurgence of reference quality assemblies using 3rd generation sequecning.</a><br>
<a href="http://www.amnh.org/">American Museum of Natural History</a> New York, NY. Dec 9, 2014</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>90.</td> <td>
<a href="/presentations/2014/2014.12.08.iPlant Assembly Quality.pdf">Assembly Quality</a><br>
<a href="http://www.cshl.edu">USDA/ARS Workshop</a> Cold Spring Harbor, NY. Dec 8, 2014</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>89.</td> <td>
<a href="/presentations/2014/2014.11.08.Indel Errors.pdf">Reducing INDEL calling errors in whole genome and exome sequencing data.</a>. Presented by Han Fang<br>
<a href="http://meetings.cshl.edu/meetings/2014/data14.shtml">Biological Data Sciences</a> Cold Spring Harbor, NY. Nov 5-8, 2014</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>88.</td> <td>
<a href="/presentations/2014/2014.11.05.BIODATA14 Intro.pdf">Biological Data Sciences Introduction</a><br>
<a href="http://meetings.cshl.edu/meetings/2014/data14.shtml">Biological Data Sciences</a> Cold Spring Harbor, NY. Nov 5-8, 2014</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>87.</td> <td>
<a href="/presentations/2014/2014.09.20.Genome Informatics.NanoCorr2.pdf">Error Correction and Assembly of Single Molecule Sequencing Data</a>. Presented by James Gurtowski<br>
<a href="https://registration.hinxton.wellcome.ac.uk/display_info.asp?id=406">Genome Informatics</a> Cambridge, UK. Sept 23, 2014</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>86.</td> <td>
<a href="/presentations/2014/2014.09.20.GRC Assembly Workshop Roundtable 1.pdf">Assembly Workshop Roundtable</a><br>
<a href="http://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/">Genome Reference Consortium</a> Cambridge, UK. Sept 20, 2014</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>85.</td> <td>
<a href="/presentations/2014/2014.09.20.GRC Assembly Workshop.PanGenomics.pdf">Pan-genomics: theory and practice</a><br>
<a href="http://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/">Genome Reference Consortium</a> Cambridge, UK. Sept 20, 2014</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>84.</td> <td>
<a href="/presentations/2014/2014.09.09.CSHL Promotion talk2.pdf">Special Seminar: Algorithms for Genome Sequencing and Disease Analytics</a><br>
<a href="http://www.cshl.edu/">CSHL</a> Cold Spring Harbor, NY. Sept 9, 2014</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>83.</td> <td>
<a href="/presentations/2014/2014.07.11.HitSeq.SplitMEM.pdf">SplitMEM: Graphical Pan-Genome Analysis with Suffix Skips</a>. Presented by Shoshana Marcus<br>
<a href="http://www.hitseq.org">HITSeq/ISMB</a> Boston, MA. July 11, 2014</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>82.</td> <td>
<a href="http://blog.pacificbiosciences.com/2014/07/optimizing-eukaryotic-de-novo-genome.html">Optimizing Eukaryotic De Novo Genome Assembly</a>. Presented with James Gurtowski and Sergey Koren<br>
<a href="http://www.pacificbiosciences.com/">PacBio Webinar</a> Menlo Park, CA. June 26, 2014</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>81.</td> <td>
<a href="/presentations/2014/2014.06.18.CSHL Big Data.pdf">Big Data Meets DNA: How Biological Data Sciences is improving our health, foods, and energy needs</a> [<a href="2014.06.18.CSHL Big Data.Flyer.png">Flyer</a>]<br>
<a href="http://www.cshl.edu">CSHL Public Lectures</a> Cold Spring Harbor, NY. June 18, 2014</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>80.</td> <td>
<a href="/presentations/2014/2014.05.29.SFAF.EukaryoticAssembly.pdf">Near perfect de novo assemblies of eukaryotic genomes using PacBio long read sequencing</a>. Presented by James Gurtowski<br>
<a href="http://www.lanl.gov/conferences/sequencing-finishing-analysis-future/index.php">Sequencing, Finishing, and Analysis in the Future Meeting</a> Santa Fe, NM. May 29, 2014</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>79.</td> <td>
<a href="/presentations/2014/2014.05.22.Procter and Gamble.pdf">Big Data Meets DNA</a><br>
<a href="http://www.pg.com">Procter and Gamble</a> Mason, OH. May 22, 2014</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>78.</td> <td>
<a href="/presentations/2014/2014.05.05.SplitMEM.pdf">SplitMEM: Graphical Pan Genome Analysis with Suffix Skips</a>. Presented by Shoshana Marcus<br>
<a href="http://www.cshl.edu">CSHL Quantitative Biology Seminar Series</a> Cold Spring Harbor, NY. May 5, 2014</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>77.</td> <td>
<a href="/presentations/2014/2014.04.08.IEEE Fellows Syracuse.pdf">Big Data Meets DNA</a><br>
<a href="http://ewh.ieee.org/r1/syracuse/">IEEE Fellows Night Dinner</a> [<a href="/presentations/2014.04.08.IEEE.Fellows.Flyer.pdf">Flyer</a>] Syracuse, NY. April 8, 2014</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>76.</td> <td>
<a href="/presentations/2014/2014.04.07.Hamilton.Assembly and Disease Analytics.pdf">Genome Assembly and Disease Analytics</a><br>
<a href="https://www.hamilton.edu/academics/departments/faculty?dept=biology">Department of Biology</a>. Hamilton College. Clinton, NY. April 7, 2014</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>75.</td> <td>
<a href="/presentations/2014/2014.03.24.Keystone BigData.pdf">The next 10 years of quantitative biology</a><br>
<a href="http://www.keystonesymposia.org/index.cfm?e=web.Meeting.Program&meetingid=1274">Keystone Symposia: Big Data in Biology</a>. San Francisco, CA. March 25, 2014</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>74.</td> <td>
<a href="/presentations/2014/2014.03.11.UFlorida.Assembly and Disease Analytics.pdf">Genome Assembly and Disease Analytics</a><br>
<a href="http://ufgi.ufl.edu/">University of Florida Genetics Institute</a>. Gainseville, FL. March 11, 2014</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>73.</td> <td>
<a href="/presentations/2014/2014-02-19.Brown.Assembly and Disease Analytics.pdf">Genome Assembly and Disease Analytics</a><br>
<a href="http://www.brown.edu/research/projects/computational-molecular-biology/">Center for Computational and Molecular Biology</a> Brown University. Providence RI. Feb 19, 2014</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>72.</td> <td>
<a href="/presentations/2014/2014-02-14.AGBT.PacBioAssembly.pdf">A near perfect de novo assembly of a eukaryotic genome.</a> by W. Richard McCombie<br>
<a href="http://www.agbt.org/">Advances in Genome Biology and Technology (AGBT)</a> Marco Island, FL. Feb 14, 2014</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>71.</td> <td>
<a href="/presentations/2014/2014-02-11.JHU.Assembly and Disease Analytics.pdf">Genome Assembly and Disease Analytics</a><br>
<a href="http://idies.jhu.edu/">Institute for Data Intensive Engineering and Science</a> Johns Hopkins University. Baltimore MD. Feb 11, 2014</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>70.</td> <td>
<a href="/presentations/2014/2014-01-14.PAG.Jnomics Services.pdf">KBase: Variation and RNA-seq services</a>.<br>
<a href="http://www.intlpag.org/web/">Plant and Animal Genomes (PAGXXII)</a> San Diego, CA. Jan 14, 2014</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>69.</td> <td>
<a href="/presentations/2014/2014-01-14.PAG.Single Molecule Assembly.pdf">De novo assembly of complex genomes using single molecule sequencing</a>.<br>
<a href="http://www.intlpag.org/web/">Plant and Animal Genomes (PAGXXII)</a> San Diego, CA. Jan 14, 2014</td></tr>

<tr><td><br></td></tr>

<tr><td colspan=2><br><h3>2013</h3><hr></td></tr>

<tr><td valign=top align=right>68.</td> <td>
<a href="/presentations/2013/2013-10-31.Scalpel.GI2013.pdf">SCALPEL: Micro-assembly approach to detect indels within exome-capture data</a>.<br>
<a href="http://www.cshl.edu">Genome Informatics</a> Cold Spring Harbor, NY. Oct 31, 2013</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>67.</td> <td>
<a href="/presentations/2013/2013-10-18.CSHL Inhouse.pdf">Algorithms for the analysis of complex genomes</a>.<br>
<a href="http://www.cshl.edu">Cold Spring Harbor In-house seminar series</a> Cold Spring Harbor, NY. Oct 10, 2013</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>66.</td> <td>
<a href="/presentations/2013/2013-10-03.BtG.De novo assembly of complex genomes.pdf">De novo assembly of complex genomes</a>.<br>
<a href="http://www.beyond-the-genome.com/">Beyond the Genome</a> Mission Bay Conference Center, San Francisco, CA. Oct 1-3, 2013</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>65.</td> <td>
<a href="/presentations/2013/2013-09-18.Cornell.De novo assembly of complex genomes.pdf">De novo assembly of complex genomes</a>.<br>
<a href="http://weill.cornell.edu/">Institute for Computational Biomedicine</a> Weill Cornell Medical College, New York, NY. Sept 18, 2013</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>64.</td> <td>
<a href="/presentations/2013/2013-09-10.UIUC.De novo assembly of complex genomes.pdf">De novo assembly of complex genomes</a>.<br>
<a href="http://www.igb.illinois.edu/">Institute for Genomic Biology Seminar</a> UIUC, Urbana, IL. Sept 10, 2013</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>63.</td> <td>
<a href="/presentations/2013/2013-06-28.KBase Variation Services.pdf">KBase Variation Services</a> with James Gurtowski.<br>
<a href="http://kbase.us">KBase Webinar</a>. CSHL, Cold Spring Harbor, NY. June 28, 2013</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>62.</td> <td>
<a href="/presentations/2013/2013-06-18.PBUserMeeting.pdf">Hybrid De Novo Assembly of Eukaryotic Genomes</a> by James Gurtowski.<br>
<a href="http://programs.pacificbiosciences.com/l/1652/2013-04-11/2sty6r">PacBio Users Meeting</a>. University of Maryland School of Medicine, Baltimore, MD. June 18, 2013</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>61.</td> <td>
<a href="/presentations/2013/2013-05-02.Microbial Assembly.pdf">Genome Sequencing and Assembly</a>.<br>
<a href="http://hmpdacc.org/events/microbiome-assembly-webinar.html">Human Microbiome Consortium Virtual Meeting: Approaches in Microbiome Assembly</a>. University of Maryland School of Medicine, Baltimore, MD. May 2, 2013</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>60.</td> <td>
<a href="/presentations/2013/2013-04-29.Banbury.Neuroscience Consortium.pdf">IT Considerations: Hurdles and Solutions</a>.<br>
<a href="http://www.cshl.edu/banbury/">Developing a Neuroscience Consortium</a>. Banbury Center, Cold Spring Harbor, NY. April 29, 2013</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>59.</td> <td>
<a href="/presentations/2013/2013-04-10.UVA.De novo assembly of complex genomes.pdf">De novo assembly of complex genomes</a>.<br>
<a href="http://cphg.virginia.edu/seminar-series/">University of Virginia</a>. Charlottesville, VA. April 10, 2013</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>58.</td> <td>
<a href="/presentations/2013/2013-03-18.NYGC.AWS.CloudScaleSolutions.pdf">Cloud-scale Sequence Analyis</a>.<br>
<a href="http://nygenome.org">New York Genome Center / AWS</a>. New York, NY. Mar 18, 2013.</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>57.</td> <td>
<a href="/presentations/2013/2013-02-20.AGBT.Assembling Crop Genomes.pdf">Assembling Crop Genomes with Single Molecule Sequencing</a>.<br>
<a href="http://www.agbt.edu/">AGBT 2013</a>. Marco Island, FL. Feb 22, 2013.</td></tr>

<tr><td><br></td></tr>

<tr><td colspan=2><br><h3>2012</h3><hr></td></tr>

<tr><td valign=top align=right>56.</td> <td>
<a href="/presentations/2012/2012-11-20.Schatz CSHL Inhouse.pdf">Human Genetics and Plant Genomics: The long and the short of it</a>.<br>
<a href="http://www.cshl.edu/">CSHL In House Symposium XXVI</a>. Cold Spring Harbor, NY. Nov 20, 2012.</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>55.</td> <td>
<a href="/presentations/2012/2012-10-18.PacBio.Assembling Crop Genomes.pdf">De novo assembly of complex crop genomes</a>.<br>
<a href="http://www.pacificbiosciences.com/">PacBio Users Meeting</a>. Menlo Park, CA. Oct 18, 2012.</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>54.</td> <td>
<a href="/presentations/2012/2012-10-08.TGAC.Assembling Crop Genomes.pdf">Assembling crop genomes with 2nd and 3rd generation sequencing</a>.<br>
<a href="http://www.tgac.ac.uk/outreach/workshop/">Strategies for de novo assemblies of complex crop genomes</a>.  The Genome Analysis Centre, Norwich, England. Oct 8, 2012</td> </tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>53.</td> <td>
<a href="/presentations/2012/2012-09-27.BtG.Assembly Primer.pdf">Genome Assembly and Alignment Primer</a>.<br>
<a href="http://www.beyond-the-genome.com/">Beyond the Genome</a>.  Harvard Medica School, Boston, MA. Sept 27, 2012</td> </tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>53.</td> <td>
<a href="/presentations/2012/2012-09-27.BtG.Illuminating the genetics of complex human diseases.pdf">Illuminating the genetics of complex human diseases</a>.<br>
<a href="http://www.beyond-the-genome.com/">Beyond the Genome</a>.  Harvard Medica School, Boston, MA. Sept 27, 2012</td> </tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>52.</td> <td>
<a href="/presentations/2012/2012-09-18.Purdue.De novo assembly of complex genomes.pdf">De novo assembly of complex genomes</a>.<br>
<a href="http://www.stat.purdue.edu/calendar/seminars/bioinformatics/">Purdue Statistical Bioinformatics Seminar Series</a>.  Purdue University, West Lafayette, IN. Sept 18, 2012</td> </tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>51.</td> <td>
<a href="/presentations/2012/2012-05-10.CSHL BOG.PacBio.pdf">SMRT-assembly: Error correction and de novo assembly of complex genomes using single molecule, real time sequencing</a>.<br>
<a href="http://meetings.cshl.edu/meetings/genome12.shtml">Biology of Genomes</a>. Cold Spring Harbor, NY. May 10, 2012</td> </tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>50.</td> <td>
<a href="/presentations/2012/2012-03-29.NYU.Scalable Solutions for 2nd and 3rd gen sequencing.pdf">Scalable Solutions for 2nd and 3rd generation sequencing</a>.<br>
<a href="http://cgsb.as.nyu.edu/page/events">NYU HiTS Seminar</a>.  New York, NY. March 29, 2012</td> </tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>49.</td> <td>
<a href="/presentations/2012/2012-03-20.JGI.Entering the era of megagenomics.pdf">Entering the era of mega-genomics</a>.<br>
<a href="http://www.jgi.doe.gov/meetings/usermeeting/index.html">JGI Users Meeting</a>.  Walnut Creek, CA. March 20, 2012</td> </tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>48.</td> <td>
<a href="/presentations/2012/2012-03-13.iPlant.Entering the era of megagenomics.pdf">Entering the era of mega-genomics</a>.<br>
<a href="https://pods.iplantcollaborative.org/wiki/display/TT/Tech+Talks">iPlant Tech Talk</a>. Webinar. March 13, 2012</td> </tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>47.</td> <td>
<a href="/presentations/2012/2012-03-02.UNC Charlotte.Entering the era of megagenomics.pdf">Entering the era of mega-genomics</a>.<br>
<a href="http://bioinformatics.uncc.edu/events/entering-era-mega-genomics-presented-dr-michael-c-schatz-assistant-professor-simons-center-qu">UNC Charlotte</a>.  Charlotte, NC. March 2, 2012</td> </tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>46.</td> <td>
<a href="/presentations/2012/2012-02-17.AGBT.Metassembler.pdf">Metassembler: Improving de novo genome assembly</a>.<br>
<a href="http://agbt.org">AGBT</a>.  Marco Island, FL. Feb 12, 2012</td> </tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>45.</td> <td>
<a href="/presentations/2012/2012-02-07.Pioneer.Entering the era of megagenomics.pdf">Entering the era of mega-genomics</a>.<br>
<a href="http://pioneer.com">Pioneer</a>.  Des Moines, IA. Feb 7, 2012</td> </tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>44.</td> <td>
<a href="/presentations/2012/2012-01-17.PAG.SMRTassembly.pdf">SMRT-assembly: Error correction and de novo assembly of complex genomes using SMRT sequencing</a>.<br>
<a href="http://www.intlpag.org/web/">PAG-XX: PacBio Workshop</a>.  San Diego, CA. Jan 17, 2012</td> </tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>43.</td> <td>
<a href="/presentations/2012/2012-01-15.PAG.De novo assembly of complex genomes.pdf">De novo assembly of complex genomes using 3rd generation sequencing</a>.<br>
<a href="http://www.intlpag.org/web/">PAG-XX: Sequencing Complex Genomes</a>.  San Diego, CA. Jan 15, 2012</td> </tr>

<tr><td><br></td></tr>


<tr><td colspan=2><br><h3>2011</h3><hr></td></tr>

<tr><td valign=top align=right>42.</td> <td>
<a href="/presentations/2011/2011-12-09.CSHL.Micro.Mega.Meta.assembly.pdf">Applications of micro-, mega-, and meta-assembly</a>.<br>
<a href="http://www.cshl.edu">CSHL In house seminar series</a>.  Cold Spring Harbor, NY. Dec 9, 2011.  </td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>41.</td> <td>
<a href="/presentations/2011/2011-11-03.Genome Informatics.pdf">Applications of micro-, mega-, and meta-assembly</a>.<br>
<a href="http://meetings.cshl.edu/meetings/info11.shtml">CSHL Genome Informatics</a>.  Cold Spring Harbor, NY. Nov 3, 2011.  </td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>40.</td> <td>
<a href="/teaching/2011/2011.Frontiers-Answering%20the%20demands%20of%20digitial%20genomics.pdf">Frontiers in Genomics: Answering the demands of digital genomics</a>.  <br>
<a href="http://www.lcg.unam.mx/frontiers/">Frontiers of Genomics</a>, <a href="http://www.ccg.unam.mx/en">Center for Genomic Sciences</a>, University of Mexico.  Cuernavaca Mexico. Oct 3-4, 2011.  </td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>39.</td> <td>
<a href="/presentations/2011/2011-09-19.BtG.Answering the demands of digital genomics.pdf">Beyond the genome: Answering the demands of digital genomics</a>.
<a href="/presentations/2011/2011-09-19.BtG.Demo.pdf">[Demo]</a>
<a href="/presentations/2011/2011-09-19.BtG.Panel.pdf">[Panel]</a>
<br>
<a href="http://www.beyond-the-genome.com/">Beyond the Genome</a>. Rockville MD. Sept 20, 2011.  </td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>38.</td> <td>
<a href="/presentations/2011/2011-09-07.PacBio Users Meeting.pdf">SMRT-assembly approaches</a>.<br>
<a href="http://www.pacificbiosciences.com/">PacBio Users Meeting</a>. Menlo Park, CA. Sept 7, 2011.
</td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>38.</td> <td>
<a href="/presentations/2011/2011-06-08.HPDC.MapReduce.Genome Indexing.pdf">Rapid Parallel Genome Indexing using MapReduce</a>.<br>
<a href="http://graal.ens-lyon.fr/mapreduce/">2nd Annual Workshop on MapReduce</a>. HPDC 2011, San Jose, CA. June 8, 2011.
</td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>37.</td> <td>
<a href="/presentations/2011/2011-06-08.HDPC.3DAPAS.pdf">Cloud Computing and the DNA Data Race</a>.<br>
<a href="http://salsahpc.indiana.edu/ECMLS2011/">Emerging Computational Methods for the Life Sciences</a>. HPDC 2011, San Jose, CA. June 8, 2011.
</td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>36.</td> <td>
<a href="/presentations/2011/2011-04-14.PSC.pdf">Cloud Computing and the DNA Data Race</a>.<br>
<a href="http://www.psc.edu/data-analytics/">Data-Intensive Analysis, Analytics, and Informatics</a>.  Pittsburgh Supercomputing Center, Pittsburgh, PA. Apr 14-15, 2011.
</td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>35.</td> <td>
<a href="/presentations/2011/2011-03-28.Columbia.pdf">Cloud Computing and the DNA Data Race</a>.<br>
<a href="http://www.cumc.columbia.edu/">Columbia University Medical Center</a>. New York, NY. Mar 28, 2011.
</td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>34.</td> <td>
<a href="/presentations/2011/2011-03-16.Assemblathon.pdf">Assembly and Validation of Large Genomes from Short Reads</a>.<br>
<a href="http://www.genome10k.org/events/event/5">2011 Genome Assembly Workshop &amp; Genome 10k Project Meeting</a>. Chaminade, Santa Cruz, CA. Mar 14-16, 2011.
</td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>33.</td> <td>
<a href="/presentations/2011/2011-02-15.LauferCenter.pdf">Cloud Computing and the DNA Data Race</a>.<br>
<a href="http://laufercenter.stonybrook.edu/">Laufer Center for Physical and Quantitative Biology</a>. Stony Brook University, Stony Brook, NY. Feb. 15, 2011.
</td></tr> 

<tr><td colspan=2><br><h3>2010</h3><hr></td></tr>

<tr><td valign=top align=right>32.</td> <td>
<a href="/presentations/2010/2009/2009-11-23.CSHL In House.pdf">Ultra Large DNA Sequence Analysis</a>.<br>
<a href="http://www.cshl.edu/">2010 CSHL In House Symposium</a>. CSHL. Cold Spring Harbor NY, Nov. 23, 2010.
</td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>31.</td> <td>
<a href="/presentations/2010/2010-11-12.GENSIPS.Assembly in the Clouds.pdf">Assembly in the Clouds</a>.<br>
<a href="http://meetings.cshl.edu/meetings/gensips10.shtml">9th IEEE International Workshop on Genomic Signal Processing and Statistics (GENSIPS)</a>. CSHL. Cold Spring Harbor NY, Nov. 12, 2010.
</td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>30.</td> <td>
<a href="/presentations/2010/2010-10-22.CSHL.Sequence.Cloud.Computing.pdf">Cloud Computing and the DNA Data Race: Theory and Practice</a>.<br>
<a href="http://meetings.cshl.edu/courses/c-seqtech10.shtml">Advanced Sequencing Technologies &amp; Applications</a>. CSHL. Cold Spring Harbor NY, Oct. 25, 2010.
</td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>29.</td> <td>
<a href="/presentations/2010/2010-10-13.Beyond the Genome - Assembly in the Clouds.pdf">Assembly in the Clouds</a>.<br>
<a href="http://biomedcentral.cvent.com/EVENTS/Info/Fees.aspx?e=89d8be73-d072-43f8-8e35-ec75c44b3a03">Beyond the Genome</a>. Harvard Medical School. Boston MA, Oct 11-13, 2010.
</td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>28.</td> <td>
<a href="/presentations/2010/2010-10-13.Beyond the Genome - Cloud Technical Challenge.pdf">Cloud Technical Challenges</a>.<br>
<a href="http://biomedcentral.cvent.com/EVENTS/Info/Fees.aspx?e=89d8be73-d072-43f8-8e35-ec75c44b3a03">Beyond the Genome</a>. Harvard Medical School. Boston MA, Oct 11-13, 2010.
</td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>27.</td> <td>
<a href="/presentations/2010/2010-07-23.Illumina.pdf">Assembly of Large Genomes using Cloud Computing / How to compute with 1000s of cores</a>.<br>
Illumina Sequencing Panel. Toronto, ON Canada. July 23, 2010.
</td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>26.</td> <td>
<a href="/presentations/2010/2010-06-29.Schimmy.pdf">Design Patterns for Efficient Graph Algorihms in MapReduce</a>.<br>
Presented by <a href="http://www.umiacs.umd.edu/~jimmylin/">Jimmy Lin</a><br>
<a href="http://developer.yahoo.com/events/hadoopsummit2010/">Hadoop Summit 2010</a>.  Santa Clara, CA. June 29, 2010.
</td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>25.</td> <td>
<a href="/presentations/2010/2010-06-16.MayoClinic.pdf"> Cloud Computing and the DNA Data Race</a>.<br>
<a href="http://www.mayoclinic.org/">Mayo Clinic Genomics Interest Group</a>.
Rochester, MN. June 16, 2010.
</td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>24.</td> <td>
<a href="/presentations/2010/2010-05-18.BioHPCStoneRidge.pdf">High
Performance Computing for DNA Sequence Alignment and Assembly</a>.<br>
<a href="http://www.stoneridgetechnology.com/">Stone Ridge Technology</a>. Bel Air, MD. May 18, 2010.
</td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>23.</td> <td>
<a href="/presentations/2010/2010-04-01.NHGRI.ComputationalArchitecturesofTheCloud.pdf">Computational Architecture of Cloud Environments</a>.<br>
NHGRI Cloud Computing Meeting. <a href="http://www.genome.gov">NHGRI</a>. Bethesda, MD. April 1, 2010.
</td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>22.</td> <td>
<a href="/presentations/2010/2010-03-23.CSHL.ScalableSolutions.pdf">Scalable Solutions for DNA Sequence Analysis</a>.<br>
<a href="http://www.cshl.edu/">Cold Spring Harbor Laboratory</a>. Cold Spring Harbor, NY. March 23, 2010.
</td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>21.</td> <td>
<a href="/presentations/2010/2010-03-15.XGen-Scalable Solutions.pdf">CloudBurst, Crossbow, and Contrail: Scaling Up Bioinformatics with Cloud Computing</a>.<br>
Sequencing Data Analysis and Storage, <a href="http://www.healthtech.com/xgn/sdw">XGen Conference</a>. San Diego, CA. March 15, 2010.<br>
<p>One of the main challenges for computational biologists is creating efficient algorithms to match improvements in high throughput sequencing. Here we describe how CloudBurst and Crossbow use cloud computing for mapping and genotyping whole human genomes at deep coverage in an afternoon. We'll also describe how our new program, Contrail, uses cloud computing to scale up de Bruijn graph construction and analysis for the assembly of large genomes from short reads. </p>
</td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>20.</td> <td>
<a href="/presentations/2010/2010-03-11.Argonne-ScalableSolutions.pdf">Scalable Solutions for DNA Sequence Analysis</a>.<br>
Argonne Leadership Computing Seminar, <a href="http://www.anl.gov/">Argonne National Lab</a>. Argonne, IL. March 11, 2010.
</td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>19.</td> <td>
<a href="/presentations/2010/2010-01-29.NHGRI.ScalableSolutions.pdf">Scalable Solutions for DNA Sequence Analysis</a>.<br>
NHGRI/UMD Computational Workshop, <a href="http://www.genome.gov/">NHGRI</a>. Rockville, MD. Jan. 29, 2010.
</td></tr> 

<tr><td colspan=2><br><h3>2009</h3><hr></td></tr>

<tr><td valign=top align=right>18.</td> <td>
<a href="/presentations/2009/2009-12-15.HadoopMeetup.pdf">Scalable Solutions for DNA Sequence Analysis</a>.<br>
<a href="http://www.meetup.com/Hadoop-DC/">DC Hadoop Users Group</a>. College Park, MD. Dec. 16, 2009.
</td></tr> 

<tr><td><br></td></tr>


<tr><td valign=top align=right>17.</td> <td>
<a href="/presentations/2009/2009-12-04.JHU.ScalableSolutions.pdf">Scalable Solutions for DNA Sequence Analysis</a>.<br>
JHU/UMD Joint Sequencing Meeting, <a href="http://www.biostat.jhsph.edu/">JHU Biostatistics Department</a>. Baltimore, MD. Dec. 4, 2009.
</td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>16.</td> <td>
<a href="/presentations/2009/SC09-DoctoralResearch.pdf">GPGPU and Cloud Computating for DNA Sequence Analysis</a>.<br>
Doctoral Showcase at <a href="http://sc09.supercomputing.org">SC09</a>. Portland, OR. Nov. 19, 2009.<br>
<p> Recent advances in DNA sequencing technology have dramatically increased in the scale and scope of DNA sequence analysis, but these analyses are complicated by the volume and complexity of the data involved. For example, genome assembly computes the complete sequence of a genome from billions of short fragments read by a DNA sequencer, and read mapping computes how short sequences match a reference genome to discover conserved and polymorphic regions. Both of these important computations benefit from recent advances in high performance computing, such as in <a href="http://mummergpu.sf.net">MUMmerGPU</a> which uses highly parallel graphics processing units as high performance parallel processors, and in <a href="http://cloudburst-bio.sf.net">CloudBurst</a> and <a href="http://bowtie-bio.sourceforge.net/crossbow">Crossbow</a> which use the MapReduce framework coupled with cloud computing to parallelize read mapping across large remote compute grids. These techniques have demonstrated orders of magnitude improvements in computation time for these problems, and have the potential to make otherwise infeasible studies practical.  </p>
</td></tr> 

<tr><td><br></td></tr>

<tr><td valign=top align=right>15.</td> <td>
<a href="/presentations/2009/SC09-SysBio.pdf">Commodity Computing in Genomics Research</a>.<br> 
<a href="http://www.mcs.anl.gov/events/workshops/sc09-sysbio/index.php">Workshop on Using Clouds for parallel computations in systems biology</a> in <a href="http://sc09.supercomputing.org/index.php">SC09</a>. Portland, OR. Nov. 16, 2009. <br>
<p> In the next few years the data generated by DNA sequencing instruments around the world will exceed petabytes, surpassing the amounts of data generated by large-scale physics experiments such as the Hadron Collider. Even today, the terabytes of data generated every few days by each sequencing instrument test the limits of existing network and computational infrastructures. Our project is aimed at evaluating whether cloud computing technologies and the MapReduce/Hadoop infrastructure can enable the analysis of the large data-sets being generated. We will report on initial results in two specific applications: human genotyping and genome assembly using next generation sequencing data.  </p>
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>14.</td> <td>
<a href="/presentations/2009/2009-10-05.CLuE.Presentation.pdf">Commodity Computing in Genomics Research</a>.<br>
Presented by Mihai Pop. Authored by Mihai Pop, Michael Schatz, and Dan Sommer.<br>
<a href="https://wiki.umiacs.umd.edu/ccc/index.php/CLuE_PI_Meeting_2009">NSF CLuE PI Meeting</a>. Mountain View CA. Oct 5, 2009.
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>13.</td> <td>
<i>Upper bounds on the ability to reconstruct prokaryotic genomes with next generation sequencing technologies.</i><br>
Presented by <a href="/~mpop">Mihai Pop</a>. Authored by Joshua Wetzel, Michael Schatz, Carl Kingsford, and Mihai Pop.<br>
<a href="http://www.wabi09.org/">WABI 2009</a>. Philadelphia, PA. Sept 12 2009.
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>12.</td> <td>
<a href="/presentations/2009/2009-06-26 SOM Assembly Bootcamp.pdf">Assembly Boot Camp</a>.<br>
<a href="http://www.igs.umaryland.edu/">UMD Institute for Genome Sciences</a>.  Baltimore, MD. June 26 2009.<br> 
<p> The theory and practice of genome assembly using the Celera Assembler. Special emphasis is given to tuning the parameters and settings to get the best results for your data.</p>
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>11.</td> <td>
<a href="/presentations/2009/2009-06-18.JCVI.pdf">High Throughput Sequence Analysis with MapReduce</a>.<br>
<a href="http://www.jcvi.org">J. Craig Venter Institute</a> Informatics Seminar.  Rockville, MD.  June 18, 2009. <br>
<p> MapReduce is the parallel distributed computing framework developed by Google for large data computations, including analyzing their collection of more than 1 trillion web pages on clusters with 10s of thousands of nodes. This system enables rapid development of highly scalable applications, because developers write just a few application specific functions, and the system automatically and intelligently provides the scheduling, monitoring, and partitioning necessary to scale to this size.  Furthermore, MapReduce is becoming a de facto standard for executing large computations within the cloud, where remote compute resources are used generically under a pay-as-you-go pricing model.  </p>
<p> In this presentation, I will describe the leading open-source implementation of MapReduce called Hadoop, the cloud computing capabilities of Amazon, and outline MapReduce-based sequence analysis algorithms for read alignment, SNP discovery, and genome assembly. Scalable algorithms for these problems are essential given that current sequencing technologies routinely generate tens or hundreds of gigabytes of data for a single experiment, and can require hundreds or thousands of hours of computation. The results show MapReduce is an extremely effective system for analyzing these datasets, with near linear speedups as the size of the cluster grows. Furthermore, the Amazon compute cloud can be an efficient and cost-effective resource, especially for periodic or unusually large compute tasks.  </p> 
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>10.</td> <td>
<a href="http://www.umiacs.umd.edu/~jimmylin/cloud-computing/2009-06-10/bioinformatics-in-the-clouds.pdf">Genetic Sequence Analysis in the Clouds</a>.<br>
Presented by <a href="http://www.umiacs.umd.edu/~jimmylin/">Jimmy Lin</a> at the <a href="http://developer.yahoo.com/events/hadoopsummit09/">Hadoop Summit 2009.</a> Santa Clara, CA. June 10, 2009. 
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>9.</td> <td>
<a href="http://www.slideshare.net/tracylaxdal/aws-startup-event-washington-dc-2009-schatz-cloud-burst">CloudBurst: Highly Sensitive Read Mapping with MapReduce</a>.<br>
<a href="">Amazon Web Services</a> Start Up Event. Washington, DC. May 27, 2009.
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>8.</td> <td>
<a href="/presentations/2009/2009-05-21 MUMmerGPU-CCOE.pdf">High Throughput Sequence Alignment using Graphics Processing Units</a>.<br>
UMD nVidia Partnership Workshop. College Park, MD. May 21, 2009.
</td></tr>

<tr><td colspan=2><br><h3>2008 and earlier</h3><hr></td></tr>

<tr><td valign=top align=right>7.</td> <td>
<a href="http://www.google.com/url?sa=t&source=web&ct=res&cd=1&url=http%3A%2F%2Fvideo.google.com%2Fvideoplay%3Fdocid%3D6602835232432608222&ei=_BDISZqwHc2rtgeKoq3ICg&usg=AFQjCNG2cbExr2sU72TiEAHoTUrQ2-uzKg&sig2=GEwnaLtK9-_vsaQBMEVcDw">Revealing Biological Modules via Graph Summarization.</a>.<br>
Presented by <a href="http://www.cs.umd.edu/~saket/">Saket Navlakha</a> at the
<a href="http://compbio.mit.edu/recombsat/">RECOMB-SB/RG/DREAM3 2008 satelite conference.</a> Boston, MA. Oct. 30, 2008.<br>
<p> A technique called Graph Summarization can be used to partition protein-protein interaction networks to reveal modules that are more biologically relevant than the clusters produced by other graph partitioning techniques. We apply GS to predict Gene Ontology annotations of biological process for proteins of unknown annotations. We also apply it to detecting membership in protein complexes, as annotated in the MIPS catalog.  GS outperforms other approaches such MCODE, MCL and modularity. </p> 
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>6.</td> <td>
<a href="/presentations/2008/PAG16-PapayaTransgenes.pdf">Hunting Down the Papaya Transgenes</a>.<br>
<a href="http://www.intl-pag.org/16/16-pag.html">Plant and Animal Genomes Conference-XVI</a>. San Diego, CA.  Jan. 16, 2008. <br> 
<p> In the middle of the last century, the Papaya ringspot potyvirus (PRSV) devastated the papaya industry on the island of Oahu in Hawaii and in other fields throughout the world. With the eminent threat of the disease spreading to the fields in the Puna district of Hawaii island, researchers in the mid 1980s developed PRSV-resistant transgenic lines of papaya using the pathogen-derived resistance approach, in which genes from PRSV were inserted into the papaya genome using a gene gun. The commercialization of these transgenic lines in the late 1990s virtually saved the Hawaiian papaya industry, but without a full genome sequence, there was lingering concern as to the exact nature of the transgenic insertions.  </p> 
<p> In my presentation, I will report on the draft genome sequence of the virus-resistant ‘SunUp’ papaya, created in collaboration with the University of Hawaii, the University of Illinois at Urbana-Champaign, and other institutions. I will focus on the computational methods used for assembling the genome, validating its correctness, and the subsequent search for transgenic inserts. Our genome wide analysis, combined with Southern blot analysis and directed PCR, confirms the efficiency of the gene gun technology, with only 3 conclusive transgenic insertions. In addition, even though the papaya genome is nearly twice the size of the Arabidopsis genome, it contains fewer genes, and thus makes it an excellent candidate for further study of biosynthetic pathways and networks. </p>
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>5.</td> <td>
<a href="/presentations/2007/MUMmerGPU-CBCB.pdf">High-throughput sequence alignment using Graphics Processing Units</a>.<br>
Co-presented with <a href="http://www.cbcb.umd.edu/~cole">Cole Trapnell</a> at the
<a href="http://www.cbcb.umd.edu/seminars/">CBCB Seminar</a>. Univ of Maryland. Sept. 20, 2007<br> 
<p> The recent availability of new, less expensive high-throughput DNA sequencing technologies has yielded a dramatic increase in the volume of sequence data that must be analyzed. Sequence alignment programs such as MUMmer have proven essential for analysis of these data, but researchers will need ever faster, high-throughput alignment tools running on inexpensive hardware to keep up with new sequence technologies. We present <a href="http://mummergpu.sourceforge.net">MUMmerGPU</a>, a high-throughput parallel sequence alignment program that runs on commodity Graphics Processing Units (GPUs) in common workstations. MUMmerGPU uses the new Compute Unified Device Architecture (CUDA) from nVidia to align multiple query sequences against a single reference sequence stored as a suffix tree. By processing the queries in parallel on the highly parallel graphics card, MUMmerGPU achieves more than a 10-fold speedup over a serial CPU version of the sequence alignment kernel, and outperforms MUMmer by more than 3-fold in total application time when aligning reads from recent sequencing projects using Solexa/Illumina, 454, and Sanger sequencing technologies.  </p>
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>4.</td> <td>
<a href="/presentations/2006/CompGenomicsHawkeye.pdf">Interactive visual analytic tools for genome assembly</a>.<br>
<a href="http://www.cpe.vt.edu/genomics/">9th Annual Computational Genomics Conference</a>. Baltimore, MD. Oct. 29, 2006.<br>
<p> Genome assembly remains an inexact science. Even when accomplished with the best software available, the assembly of a genome often contains numerous errors, both small and large. <a href="http://amos.sourceforge.net/hawkeye/">Hawkeye</a> is a visual analytics tool for genome assembly analysis and validation, designed to aid in identifying and correcting assembly errors. Hawkeye blends the best practices from information and scientific visualization to facilitate inspection of large-scale assembly data while minimizing the time needed to detect mis-assemblies and make accurate judgments of assembly quality.  </p>
<p> All levels of the assembly data hierarchy are made accessible to users, along with summary statistics and common assembly metrics. A ranking component guides investigation towards likely mis-assemblies or interesting features to support the task at hand. Wherever possible, high-level overviews, dynamic filtering, and automated clustering are leveraged to focus attention and highlight anomalies in the data. Hawkeyes effectiveness has been proven on several genome projects, where it has been used both to improve quality and to validate the correctness of complex genomes.  </p>
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>3.</td> <td>
<a href="/presentations/2006/AMOS_Validation_Visualization.pdf">AMOS Assembly Validation and Visualization</a>.<br>
<a href="http://www.tigr.org">The Institute for Genomic Research</a>. Rockville, MD. April 7, 2006.<br>
<p> During my talk, I will discuss the techniques and tools to discover and correct misassemblies in genome assemblies. The three primary sources of information  used to detect misassemblies are the "happiness" of the mate-pairs, the base call agreement within the multiple alignment of reads, and the depth of coverage of those reads. </p>
<p> The open source <a href="http://amos.sourceforge.net">AMOS</a> assembly package provides tools for systematically analyzing these qualities to discover regions with potential misassemblies. The AMOS Assembly Investigator is a powerful genome assembly visualizer with semantic zooming capabilities. It allows one to navigate and visually inspect these potential misassemblies in a systematic fashion at all levels of detail. Once regions with misassemblies have been identified, users can correct the misassemblies with the AMOS contig patching tools.  </p>
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>2.</td> <td>
<a href="/presentations/2005/CBCB-ImprovingAssembly.pdf">Improving Genome Assemblies without Sequencing</a>.<br>
<a href="http://www.cbcb.umd.edu/seminars/">CBCB Seminar</a>. Univ. of Maryland. College Park, MD.  Sept. 28, 2005.<br>
<p> Genome assembly is the problem of reconstructing the genome sequence of an organism from a collection of short sequenced reads. An assembly takes the form of contiguous stretches of DNA sequence (contigs) linked together in scaffolds by mate-pair and other information.  Genome assembly is scientifically one of the most important areas of bioinformatics research as an accurate genome sequence is needed for addressing several fundamental biological questions. Unfortunately, it is also one of the most complex computationally, having been proved NP-hard under various formalisms and a typical problem size of thousands or millions of inputs.  </p>
<p> During my talk, I will discuss some of the algorithmic challenges and trade-offs in genome assembly. I will also discuss some computational methods for improving an assembly, which can be applied generally but without requiring additional laboratory results. One method was implemented in AutoEditor, which acts as a second generation base-caller to find and correct base-calling errors in reads using the original chromatogram trace and the multiple alignment of reads. A second was implemented in AutoJoiner, which attempts to automatically close gaps between linked contigs, and generally enhance contig quality, by extending the usable portion of reads within an assembly.  </p>
</td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>1.</td> <td>
<a href="/presentations/2005/TIGR-ImprovingAssembly.pdf">Improving Genome Assemblies without Sequencing</a>.<br>
<a href="http://www.tigr.org">The Institute for Genomics Research</a>. Rockville, MD. April 25, 2005.
</td></tr>


</table>



<br>

<!--#include virtual="/template/footer.shtml" -->

