---
layout: default
title: Schatzlab - SKBR3
---


<span class="contenthead">Complex rearrangements and oncogene amplifications revealed by long-read DNA and RNA sequencing of the SKBR3 breast cancer cell line</span><br>
<hr>

Maria Nattestad, Sara Goodwin, Karen Ng, Timour Baslan, Fritz J. Sedlazeck, Philipp Rescheneder, Tyler Garvin, Han Fang, James Gurtowski, Elizabeth Hutton, Elizabeth Tseng, Chen-Shan Chin, Timothy Beck, Yogi Sundaravadanam, Melissa Kramer, Eric Antoniou, John D. McPherson, James Hicks, William Richard McCombie, Michael C. Schatz <br><br>

<table class="intro">
<tr>
<td valign=top>
The SK-BR-3 cell line is one of the most important models for HER2+ breast cancers, which affect one in five breast cancer patients. SK-BR-3 is known to be highly rearranged although much of the variation is in complex and repetitive regions that may be underreported. Addressing this, we sequenced SK-BR-3 using long-read single molecule sequencing from Pacific Biosciences, and develop one of the most detailed maps of structural variations (SVs) in a cancer genome available with nearly 20,000 variants present, most of which were missed by prior efforts. Surrounding the important HER2 locus, we discover a complex sequence of nested duplications and translocations, suggesting a punctuated progression. Full-length transcriptome sequencing further revealed several novel gene fusions within the nested genomic variants. Combining long-read genome and transcriptome sequencing enables an in-depth analysis of how SVs disrupt the transcriptome and sheds new light on the complexity of cancer progression.
</td>
<td width=150 valign="top">
<img src="skbr3.png" width=150>
</td></tr></table>
  
  
<br><br><br>

<span class="contenthead">Analysis Manuscripts</span> <br>

<table class="intro" width=100%>
  
<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top><a href="http://www.biorxiv.org/content/early/2017/08/10/174938">Complex rearrangements and oncogene amplifications revealed by long-read DNA and RNA sequencing of a breast cancer cell line</a><br>
  Nattestad et al. (2017) bioRxiv. doi: https://doi.org/10.1101/174938<td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top><a href="http://www.biorxiv.org/content/early/2017/07/28/169557">Accurate detection of complex structural variations using single molecule sequencing</a><br>
  Sedlazeck, Rescheneder et al (2017) bioRxiv. doi: https://doi.org/10.1101/169557<td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top><a href="http://www.biorxiv.org/content/early/2016/11/15/087981">SplitThreader: Exploration and analysis of rearrangements in cancer genomes</a><br>
  Nattestad et al (2016) bioRxiv. doi: https://doi.org/10.1101/087981<td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top><a href="http://www.biorxiv.org/content/early/2016/10/20/082123">Ribbon: Visualizing complex genome alignments and structural variation</a><br>
  Nattestad et al (2016) bioRxiv. doi: https://doi.org/10.1101/082123<td>
</tr> <tr><td height=8 colspan=2></td></tr>

</table>

<br>

<span class="contenthead">Raw Reads</span> <br>
<table class="intro" width=100%>
<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top>Raw PacBio and Illumina reads are available in the NCBI SRA under under BioProject <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA476239/">PRJNA476239</a></td>
</tr> <tr><td height=8 colspan=2></td></tr>
</table>

<br>

<span class="contenthead">PacBio-based Analysis</span> <br>
<table class="intro" width=100%>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top>Reads aligned with NGMLR:<br>
  <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/reads_lr_skbr3.fa_ngmlr-0.2.3_mapped.bam">reads_lr_skbr3.fa_ngmlr-0.2.3_mapped.bam</a></td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top>Variants discovered with Sniffles:<br>
  <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/reads_lr_skbr3.fa_ngmlr-0.2.3_mapped.bam.sniffles1kb_auto_l8_s5_noalt.vcf.gz">reads_lr_skbr3.fa_ngmlr-0.2.3_mapped.bam.sniffles1kb_auto_l8_s5_noalt.vcf.gz</a></td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top>SplitThreader analysis of Copy number variants + long range Sniffles variants:<br>
  <a href="http://splitthreader.com/vis.php?code=example1&nickname=SK_BR_3_breast_cancer_cell_line____PacBio_sequencing">http://splitthreader.com</a></td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top>Ribbon display of SKBR3 structural variations:<br>
  <a href="http://genomeribbon.com/?perma=E_PacBio_SKBR3_TRA_chr16_to_chr19_(5MB)">http://genomeribbon.com</a></td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top>FALCON assembly of PacBio reads:<br>
  <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/skbr3.falcon.pa_ctg.fa.gz">skbr3.falcon.pa_ctg.fa.gz</a></td>
</tr> <tr><td height=8 colspan=2></td></tr>
  
<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top>Assemblytics analysis of FALCON assembly:<br>
  <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/SKBR3_PacBio_Falcon_Assembly.Assemblytics_structural_variants.bed.gz">SKBR3_PacBio_Falcon_Assembly.Assemblytics_structural_variants.bed.gz</a></td>
</tr> <tr><td height=8 colspan=2></td></tr>

</table>  

<br>

<span class="contenthead">Illumina-based Analysis</span> <br>
<table class="intro" width=100%>
<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top>Illumina PE550 reads aligned with BWA-MEM:<br>
    <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/SKBR3_550bp_pcrFREE_S1_L001_AND_L002_R1_001.101bp.bwamem.ill.mapped.sort.bam">SKBR3_550bp_pcrFREE_S1_L001_AND_L002_R1_001.101bp.bwamem.ill.mapped.sort.bam</a><td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top>Delly SVs:<br>
    <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/SKBR3_550bp_pcrFREE_S1_L001_AND_L002_R1_001.101bp.bwamem.ill.mapped.sort.bam.delly_noalt.vcf.gz">SKBR3_550bp_pcrFREE_S1_L001_AND_L002_R1_001.101bp.bwamem.ill.mapped.sort.bam.delly_noalt.vcf.gz</a><td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top>Lumpy SVs:<br>
    <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/SKBR3_550bp_pcrFREE_S1_L001_AND_L002_R1_001.101bp.bwamem.ill.mapped.sort.bam.lumpy_noalt.vcf.gz">SKBR3_550bp_pcrFREE_S1_L001_AND_L002_R1_001.101bp.bwamem.ill.mapped.sort.bam.lumpy_noalt.vcf.gz</a><td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top>Manta SVs:<br>
    <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/SKBR3_550bp_pcrFREE_S1_L001_AND_L002_R1_001.101bp.bwamem.ill.mapped.sort.bam.manta_noalt.vcf.gz">SKBR3_550bp_pcrFREE_S1_L001_AND_L002_R1_001.101bp.bwamem.ill.mapped.sort.bam.manta_noalt.vcf.gz</a><td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top>PopIns SVs:<br>
  <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/skbr3.popins.vcf.gz">skbr3.popins.vcf.gz</a><br>  
</tr> <tr><td height=8 colspan=2></td></tr>
  
<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top>SURVIVOR consensus SVs:<br>
    <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/SURVIVOR_1k_min2_min50bp.vcf">SURVIVOR_1k_min2_min50bp.vcf.gz</a><td>
</tr> <tr><td height=8 colspan=2></td></tr>
  
<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top>Allpaths-LG assembly of the Illumina reads:<br>
  <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/skbr3.allpaths.assembly.fa.gz">skbr3.allpaths.assembly.fa.gz</a><br>
  <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/skbr3.allpaths.contigs.fa.gz">skbr3.allpaths.contigs.fa.gz</a><br>    
</tr> <tr><td height=8 colspan=2></td></tr>
  
<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top>Assemblytics analysis of ALLPATHS-LG assembly:<br>
  <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/SKBR3_Illumina_AllpathsLG_Assembly.Assemblytics_structural_variants.bed.gz">SKBR3_Illumina_AllpathsLG_Assembly.Assemblytics_structural_variants.bed.gz</a></td>
</tr> <tr><td height=8 colspan=2></td></tr>
  
</table>


