---
layout: default
title: 2014 URP
---


<a name="2014"><table width="100%" bgcolor="#e6e6e6"> <tr><th style="font-variant:small-caps; font-size:12pt; color:#003399;" align="left">2014 CSHL Undergraduate Research Program in Bioinformatics </th></tr></table></a><br>
<!-- ########################################################################################### -->


<a href="2014.URP.Searching for GATTACA.pdf">Searching for GATTACA</a><br>
<p>
In this class we explored the problem of finding exact occurrences of a query
sequence in a large genome or database of sequences.  Under this theme, we
started by analyzing the brute force approach introducing the concepts of
algorithm, complexity analysis, and E-values.  Next we discussed suffix arrays
as an index for accelerating the search, including analyzing the performance of
binary search. We also considered two traditional algorithms for sorting
(Selection Sort versus QuickSort) and their relative performance.  In the second
half of the class we discussed finding approximate occurrences of a short query
sequence in a large genome or database of sequences. We first defined the
problem by considering various metrics of an approximate occurrence such as
hamming distance, or edit distance.  We then considered different methods for
computing inexact alignments including brute force global &amp; local
alignments, and seed-and-extend algorithms. Finally we discussed Bowtie as a
Burrows-Wheeler transform based short read mapping algorithm for discovering
alignments to reference genome.
</p>

<br>

<h3>Python &amp; Bioinformatics</h3>

<a href="python1.zip">Python Class 1</a><br>
<p>
Introduction to python, variables, lists, conditions, loops
</p>

<br>

<a href="python2.zip">Python Class 2</a><br>
<p>
Brute force search, dictionaries, motif finding
</p>

<br>

<h3>iPython Notebooks for Probability &amp; Statistics</h3>
<ol>
  <li> <a href="http://nbviewer.ipython.org/url/schatzlab.cshl.edu/teaching/2014/urp/1.Rolling%20a%20die.ipynb">Rolling a die (Uniform Random Probability)</a></li>
  <li> <a href="http://nbviewer.ipython.org/url/schatzlab.cshl.edu/teaching/2014/urp/2.Flipping%20a%20coin.ipynb">Flipping a coin (Binomial &amp; Normal Distributions)</a></li>
  <li> <a href="http://nbviewer.ipython.org/url/schatzlab.cshl.edu/teaching/2014/urp/3.Throwing%20Marbles%20into%20Jars.ipynb">Throwing Marbles into Jars (Poisson Distribution)</a></li>
  <li> <a href="http://nbviewer.ipython.org/url/schatzlab.cshl.edu/teaching/2014/urp/4.Throwing%20darts.ipynb">Throwing Darts (Exponential Distribution)</a></li>
</ol>
<br>

<p>
We also used the exercises at <a href="http://rosalind.info/problems/locations/">Rosalind</a> throughout the course.
</p>

<br>


<h3>Special topics</h3>
<p>
Talk by <a href="http://churchlandlab.cshl.edu/">Anne Churchland</a> on <a href="CareerDevelopmentWorkLifeBalanceTalk.pdf">balancing work and life.</a>
</p>



<!--#include virtual="/template/footer.shtml" -->
