---
layout: default
title: Schatzlab - Publications
---

<table cellpadding=4 border=0 bgcolor="#e6e6e6" class="intro" width=80% align=center>
  <tr>
    <td width="25%" bgcolor="#f9f9f9" align=center><a href="/publications">publications</a> </td>
    <td width="25%" bgcolor="#f9f9f9" align=center><a href="/publications/posters">posters</a> </td>
    <td width="25%" bgcolor="#f9f9f9" align=center><a href="/publications/reviews">press</a> </td>
    <td width="25%" bgcolor="#f9f9f9" align=center><a href="/presentations">presentations</a> </td>
  </tr>
</table>

<br>
<br>

<a name="publications"><table width="100%" bgcolor="#e6e6e6"> <tr><th style="font-variant:small-caps; font-size:12pt; color:#003399;" align="left">Publications</th></tr></table></a><br>


<table class="intro">

<tr><td colspan=3><h3>Preprints</h3><hr></td></tr>

<tr><td valign=top align=right>P7.</td>
<td valign=top> <a href="https://www.biorxiv.org/content/early/2017/10/13/202903">Addressing confounding artifacts in reconstruction of gene co-expression networks</a><br>
Parsana, P, Ruberman, C, Jaffe, AE, <b>Schatz, MC</b>, Battle, A, Leek, JT</b> (2017) <i>bioRxiv</i> https://doi.org/10.1101/202903</td>
<td valign=top> <a href="https://www.biorxiv.org/content/early/2017/10/13/202903"><img width=50 height=70 src="thumb/2017-confounders.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>P6.</td>
<td valign=top> <a href="http://www.biorxiv.org/content/early/2017/08/10/174938">Complex rearrangements and oncogene amplifications revealed by long-read DNA and RNA sequencing of a breast cancer cell line</a><br>
Nattestad, M, Goodwin, S, Ng, K, Baslan, T, Sedlazeck, FJ, Rescheneder, P, Garvin, T, Fang, H, Gurtowski, J, Hutton, E, Tseng, E, Chin, C, Beck, T, Sundaravadanam, Y, Kramer, M, Antoniou, E, McPherson, JD, Hicks, J, McCombie, WR, <b>Schatz, MC</b> (2017) <i>bioRxiv</i> doi: https://doi.org/10.1101/174938</td>
<td valign=top> <a href="http://www.biorxiv.org/content/early/2017/08/10/174938"><img width=50 height=70 src="thumb/2017-skbr3.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>P5.</td>
<td valign=top> <a href="http://www.biorxiv.org/content/early/2017/07/28/169557">Accurate detection of complex structural variations using single molecule sequencing</a><br>
Sedlazeck, FJ, Reschender, P, Smolka, M, Fang, H, Nattestad, M, von Haeseler, A, <b>Schatz, MC</b> (2017) <i>bioRxiv</i> https://doi.org/10.1101/169557</td>
<td valign=top> <a href="http://www.biorxiv.org/content/early/2017/07/28/169557"><img width=50 height=70 src="thumb/2017-sniffles.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>P4.</td>
<td valign=top> <a href="http://biorxiv.org/content/early/2016/11/15/087981">SplitThreader: Exploration and analysis of rearrangements in cancer genomes</a><br>
Nattestad, M, Alford, MC, Sedlazeck, FJ, <b>Schatz, MC </b> (2016) <i>bioRxiv</i> doi: https://doi.org/10.1101/087981</td>
<td valign=top> <a href="http://biorxiv.org/content/early/2016/11/15/087981"><img width=50 height=70 src="thumb/2016-splitthreader.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>P3.</td>
<td valign=top> <a href="http://biorxiv.org/content/early/2016/10/20/082123">Ribbon: Visualizing complex genome alignments and structural variation</a><br>
Nattestad, M, Chin, CS, <b>Schatz, MC </b> (2016) <i>bioRxiv</i> doi: http://dx.doi.org/10.1101/082123</td>
<td valign=top> <a href="http://biorxiv.org/content/early/2016/10/20/082123"><img width=50 height=70 src="thumb/2016-ribbon.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>P2.</td>
<td valign=top> <a href="http://biorxiv.org/content/early/2016/12/22/096354">The DOE Systems Biology Knowledgebase (KBase)</a><br>
Arkin, AP, Steven, RL, Cottingham, RW, Maslov, S, Henry, CS, et al. (2016) <i>bioRxiv</i> doi: https://doi.org/10.1101/096354</td>
<td valign=top> <a href="http://biorxiv.org/content/early/2016/12/22/096354"><img width=50 height=70 src="thumb/2016-kbase.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>P1.</td>
<td valign=top> <a href="http://www.biorxiv.org/content/early/2016/04/13/048603">Third-generation sequencing and the future of genomics</a><br>
Lee, H, Gurtowski, J, Yoo, S, Nattestad, M, Marcus, S, Goodwin, S, McCombie, WR  <b>Schatz, MC </b> (2016) <i>bioRxiv</i> doi: http://dx.doi.org/10.1101/048603 </td>
<td valign=top> <a href="http://www.biorxiv.org/content/early/2016/04/13/048603"><img width=50 height=70 src="thumb/2016-resurgence.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td></td></tr>
<tr><td></td></tr>

<tr><td colspan=3><h3>2018</h3><hr></td></tr>

<tr><td valign=top align=right>92.</td>
<td valign=top> <a href="http://www.cell.com/pb-assets/journals/research/cell-systems/cels_387.pdf">Scikit-ribo: Accurate estimation and robust modeling of translation dynamics at codon resolution</a><br>
Fang, H, Radhakrishnan, A, Siepel, A, Lyon, GJ, <b>Schatz, MC</b> (2017) <i>Cell Systems</i> https://doi.org/10.1016/j.cels.2017.12.007 </td>
<td valign=top> <a href="http://www.cell.com/pb-assets/journals/research/cell-systems/cels_387.pdf"><img width=50 height=70 src="thumb/2017-scikitribo.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>91.</td>
<td valign=top> <a href="https://www.nature.com/articles/s41438-017-0011-0">Reference Quality Assembly of the 3.5 Gb genome of Capsicum annuum from a Single Linked-Read Library</a><br> 
Hulse-Kemp, AM, Maheshwari, S, Stoffel, K, Hill, TA, Jaffe, D, Williams, S, Weisenfeld, N, Ramakrishnan, S, Kumar, V, Shah, P, <b>Schatz, MC</b>, Church, DM, Van Deynze, A (2017) <i>Horticulture Research</i> doi:10.1038/s41438-017-0011-0</td>
<td valign=top> <a href="https://www.nature.com/articles/s41438-017-0011-0"><img width=50 height=70 src="thumb/2017-pepper.jpg"></a> </td> </tr>
<tr><td></td></tr>

<tr><td></td></tr>
<tr><td></td></tr>

<tr><td colspan=3><h3>2017</h3><hr></td></tr>

<tr><td valign=top align=right>90.</td>
<td valign=top> <a href="http://www.sciencedirect.com/science/article/pii/S2001037017300855">LRSim: a Linked Reads Simulator generating insights for better genome partitioning</a><br>
Luo, R, Sedlazeck, FJ, Darby, CA, Kelly, SM, <b>Schatz, MC </b> (2017) <i>Computational and Structural Biotechnology Journal</i> https://doi.org/10.1016/j.csbj.2017.10.002</td>
<td valign=top> <a href="http://www.sciencedirect.com/science/article/pii/S2001037017300855"><img width=50 height=70 src="thumb/2017-lrsim.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>89.</td>
<td valign=top> <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-017-3927-8">Hybrid assembly with long and short reads improves discovery of gene family expansions</a><br>
Miller, JR, Zhou, P, Mudge, J, Gurtowski, J, Lee, H, Ramaraj, T, Walenz, BP, Liu, J, Stupar, RM, Denny, R, Song, L, Singh, N, Maron, LR, McCouch, SR, McCombie, WR,
<b>Schatz, MC,</b> Tiffin, P, Young, KD, Silverstein, KAT (2017) <i>BMC Genomics</i> doi: 10.1186/s12864-017-3927-8. 
(Please cite this for <a href="https://github.com/jgurtowski/ectools">ECTools hybrid error correction</a>)</td>
<td valign=top> <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-017-3927-8"><img width=50 height=70 src="thumb/2017-alpaca.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>88.</td>
<td valign=top> <a href="https://academic.oup.com/gigascience/article/doi/10.1093/gigascience/gix045/3868354/16GT-a-fast-and-sensitive-variant-caller-using-a">16GT: a fast and sensitive variant caller using a 16-genotype probabilistic model</a><br>
Luo, R, <b>Schatz, MC</b>, Salzberg, SL (2017) <i>GigaScience</i> doi: https://doi.org/10.1093/gigascience/gix045</td>
<td valign=top> <a href="https://academic.oup.com/gigascience/article/doi/10.1093/gigascience/gix045/3868354/16GT-a-fast-and-sensitive-variant-caller-using-a"><img width=50 height=70 src="thumb/2017-16gt.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>87.</td>
<td valign=top> <a href="https://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3861.html">Recurrent noncoding regulatory mutations in pancreatic ductal adenocarcinoma</a><br>
Feigin, ME, Garvin, T, Bailey, P, Waddell, N, Chang, DK, Kelley, DR, Shuai, S, Gallinger, S, McPherson, JD,
Grimmond, SM, Khurana, E, Stein, LD, Biankin, AV, <b>Schatz, MC</b>, Tuveson, DA (2017) <i>Nature Genetics</i> doi:10.1038/ng.3861</td>
<td valign=top> <a href="https://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3861.html"><img width=50 height=70 src="thumb/2017-gecco.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>86.</td>
<td valign=top> <a href="https://academic.oup.com/bioinformatics/article/3089939/GenomeScope-Fast-reference-free-genome-profiling">GenomeScope: Fast reference-free genome profiling from short reads</a><br>
Vurture, GW, Sedlazeck, FJ, Nattestad, M, Underwood, CJ, Fang, H, Gurtowski, J, <b>Schatz, MC </b> (2017) <i>Bioinformatics</i>  doi: https://doi.org/10.1093/bioinformatics/btx153</td>
<td valign=top> <a href="https://academic.oup.com/bioinformatics/article/3089939/GenomeScope-Fast-reference-free-genome-profiling"><img width=50 height=70 src="thumb/2016-genomescope.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>85.</td>
<td valign=top> <a href="http://www.nature.com/nmeth/journal/v14/n4/full/nmeth.4240.html">Nanopore sequencing meets epigenetics</a><br>
<b>Schatz, MC</b> (2017) <i>Nature Methods</i> 14, 347–348 (2017) doi:10.1038/nmeth.4240</td>
<td valign=top> <a href="http://www.nature.com/nmeth/journal/v14/n4/full/nmeth.4240.html"><img width=50 height=70 src="thumb/2017-nanoporemeth.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>84.</td>
<td valign=top> <a href="http://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=7858013">Bioinformatics of DNA</a><br>
Heath, LS, Bravo, HC, Caccamo, M, <b>Schatz, MC</b> (2017) <i>Proceedings of the IEEE</i> doi: 10.1109/JPROC.2017.2652958<br>
See entire issue <a href="http://ieeexplore.ieee.org/xpl/tocresult.jsp?isnumber=7857822">here</a></td>
<td valign=top> <a href="http://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=7858013"><img width=50 height=70 src="thumb/2017-IEEEDNA.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td><br></td></tr>

<tr><td valign=top align=right>83.</td>
<td valign=top> <a href="https://elifesciences.org/content/6/e21615">TGF-β reduces DNA ds-break repair mechanisms to heighten genetic diversity and adaptability of CD44+/CD24- cancer cells</a><br>
Pal, D, Pertot, A, Shirole, NH, Yao, Z, Anaparthy, N, Garvin, T, Cox, H, Chang, K, Rollins, F, Kendall, J, Edwards, L, Singh, VA,
Stone, GC, <b>Schatz, MC</b>, Hicks, J, Hannon, G, Sordella, R (2017) <i>eLife</i> doi: http://dx.doi.org/10.7554/eLife.21615</td>
<td valign=top> <a href="https://elifesciences.org/content/6/e21615"><img width=50 height=70 src="thumb/2017-tgfbeta.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td><br></td></tr>

<tr><td colspan=3><h3>2016</h3><hr></td></tr>

<tr><td valign=top align=right>82.</td>
<td valign=top> <a href="http://ieeexplore.ieee.org/document/7802776/">NanoBLASTer: Fast alignment and characterization of Oxford Nanopore single molecule sequencing reads</a><br>
Amin, M.R., Skiena, S, <b>Schatz, MC</b> (2016) <i>Proceedings of the Computational Advances in Bio and Medical Sciences (ICCABS) Conference.</i> doi: 10.1109/ICCABS.2016.7802776</td>
<td valign=top> <a href="http://ieeexplore.ieee.org/document/7802776/"><img width=50 height=70 src="thumb/2016-nanoblaster.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>81.</td>
<td valign=top> <a href="http://www.nature.com/nprot/journal/v11/n12/full/nprot.2016.150.html">Indel variant analysis of short-read sequencing data with Scalpel</a><br>
Fang, H, Grabowska, EA, Arora, K, Vacic, V, Zody, MC, Iossifov, I, ORawe, JA, Wu, Y, Jimenez-Barron, LT, Rosenbaum, J, Ronemus, M, Lee, Y, Wang, Z, Lyon, GJ, Wigler, M, <b>Schatz, MC </b>, Narzisi G (2015) <i>Nature Protocols</i> doi:10.1038/nprot.2016.150 </td>
<td valign=top> <a href="http://www.nature.com/nprot/journal/v11/n12/full/nprot.2016.150.html"><img width=50 height=70 src="thumb/2015-scalpelprotocol.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>80.</td>
<td valign=top> <a href="http://genome.cshlp.org/content/early/2016/11/07/gr.207837.116.abstract">The evolution of inflorescence diversity in the nightshades and heterochrony during meristem maturation</a><br>
Lemmon, ZH, Park, SJ, Jiang, K, Van Eck, J, <b>Schatz, MC</b>, Lippman ZB (2016) <i>Genome Research</i> doi: 10.1101/gr.207837.116</td>
<td valign=top> <a href="http://genome.cshlp.org/content/early/2016/11/07/gr.207837.116.abstract"><img width=50 height=70 src="thumb/2016-infloresence.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>79.</td>
<td valign=top> <a href="http://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.4035.html">Phased Diploid Genome Assembly with Single Molecule Real-Time Sequencing</a><br>
Chin, CS, Peluso, P, Sedlazeck, FJ, Nattestad, M, Concepcion, GT, Clum, A, Dunn, C, O'Malley, R, Figueroa-Balderas, R, Morales-Cruz, A, Cramer, GR, Delledonne, M, Luo, C, Ecker, JR, Cantu, D, Rank, DR., <b>Schatz, MC </b> (2016) <i>Nature Methods</i> doi:10.1038/nmeth.4035</td>
<td valign=top> <a href="http://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.4035.html"><img width=50 height=70 src="thumb/2016-falconunzip.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>78.</td>
<td valign=top> <a href="http://dnaresearch.oxfordjournals.org/cgi/content/full/dsw022?ijkey=dUGUjXfyDPK8Z9y&keytype=ref">Complete telomere-to-telomere de novo assembly of the Plasmodium falciparum genome through long-read (>11 kb), single molecule, real-time sequencing</a><br>
Vembar, SS, Seetin, M, Lambert, C, Nattestad, M, <b>Schatz, MC</b>, Baybayan, P, Scherf, A, Smith, ML. (2016) <i>DNA Research</i> doi: 10.1093/dnares/dsw022 </td>
<td valign=top> <a href="http://dnaresearch.oxfordjournals.org/cgi/content/full/dsw022?ijkey=dUGUjXfyDPK8Z9y&keytype=ref"><img width=50 height=70 src="thumb/2016-plasmodium.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>77.</td>
<td valign=top> <a href="http://bioinformatics.oxfordjournals.org/content/early/2016/06/16/bioinformatics.btw369.short">Assemblytics: a web analytics tool for the detection of variants from an assembly</a><br>
Nattestad, M, <b>Schatz, MC </b> (2016) <i>Bioinformatics</i> doi: 10.1093/bioinformatics/btw369</td>
<td valign=top> <a href="http://bioinformatics.oxfordjournals.org/content/early/2016/06/16/bioinformatics.btw369.short"><img width=50 height=70 src="thumb/2016-assemblytics.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>76.</td>
<td valign=top> <a href="http://www.nature.com/articles/nplants201674">Insight into the evolution of the Solanaceae from the parental genomes of Petunia hybrida</a><br>
Bombarely, A et al. (2016) <i>Nature Plants</i> doi:10.1038/nplants.2016.74</td>
<td valign=top> <a href="http://www.nature.com/articles/nplants201674"><img width=50 height=70 src="thumb/2016-petunia.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>75.</td>
<td valign=top> <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1005954">Chromosomal-Level Assembly of the Asian Seabass Genome Using Long Sequence Reads and Multi-layered Scaffolding</a><br>
Vij, S et al. (2016) <i>PLOS Genetics</i> http://dx.doi.org/10.1371/journal.pgen.1005954</td>
<td valign=top> <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1005954"><img width=50 height=70 src="thumb/2016-seabass.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>74.</td>
<td valign=top> <a href="http://www.nature.com/ncomms/2016/160202/ncomms10164/full/ncomms10164.html">Genome assembly and geospatial phylogenomics of the bed bug Cimex lectularius</a><br>
Rosenfeld, JA et al. (2016) <i>Nature Communication</i> doi:10.1038/ncomms10164</td>
<td valign=top> <a href="http://www.nature.com/ncomms/2016/160202/ncomms10164/full/ncomms10164.html"><img width=50 height=70 src="thumb/2016-bedbug.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td colspan=3><h3>2015</h3><hr></td></tr>

<tr><td valign=top align=right>73.</td>
<td valign=top> <a href="http://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3435.html">The pineapple genome and the evolution of CAM photosynthesis</a><br>
Ming, R, VanBuren, R, Man Wai, C, Tang, H, <b>Schatz MC</b>, et al. (2015) <i>Nature Genetics</i> doi:10.1038/ng.3435 </td>
<td valign=top> <a href="http://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3435.html"><img width=50 height=70 src="thumb/2015-pineapple.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>72.</td>
<td valign=top> <a href="http://www.genomebiology.com/2015/16/1/235">Teaser: Individualized benchmarking and optimization of read mapping results for NGS data</a><br>
Smolka M, Rescheneder P, <b>Schatz MC</b>, Haeseler AV, Sedlazeck FJ. (2015) <i>Genome Biology</i> 16:235 doi:10.1186/s13059-015-0803-1 </td>
<td valign=top> <a href="http://www.genomebiology.com/2015/16/1/235"><img width=50 height=70 src="thumb/2015-teaser.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>71.</td>
<td valign=top> <a href="http://genome.cshlp.org/content/early/2015/10/07/gr.191395.115.abstract">Oxford Nanopore sequencing, hybrid error correction, and de novo assembly of a eukaryotic genome</a><br> 
Goodwin, S, Gurtowski, J, Ethe-Sayers, S, Deshpande, P, <b>Schatz MC*</b>, McCombie, WR* (2015) <i>Genome Research</i> doi: 10.1101/gr.191395.115</td>
<td valign=top> <a href="http://genome.cshlp.org/content/early/2015/10/07/gr.191395.115.abstract"><img width=50 height=70 src="thumb/2015-Nanocorr.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>70.</td>
<td valign=top> <a href="http://genome.cshlp.org/content/25/10/1417.full">Biological data sciences in genome research</a><br>
<b>Schatz MC</b> (2015) <i>Genome Research</i> 25: 1417-1422. doi: 10.1101/gr.191684.115</td>
<td valign=top> <a href="http://genome.cshlp.org/content/25/10/1417.full"><img width=50 height=70 src="thumb/2015-next20years.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>69.</td>
<td valign=top> <a href="http://www.genomebiology.com/2015/16/1/207">Metassembler: merging and optimizing de novo genome assemblies</a><br>
Wences, AH <b>Schatz MC</b> (2015) <i>Genome Biology</i> 16:207. doi:10.1186/s13059-015-0764-4 </td>
<td valign=top> <a href="http://www.genomebiology.com/2015/16/1/207"><img width=50 height=70 src="thumb/2015-metassembler.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>68.</td>
<td valign=top> <a href="http://www.pnas.org/content/early/2015/09/15/1516718112.abstract">Genome and transcriptome of the regeneration-competent flatworm, Macrostomum lignano</a><br> 
Wasik, KA, Gurtowski J, Zhou, X, Mendivil Ramos O, Delás MJ, Battistoni, G, El Demerdash, O, Faciatori, I, Vizoso, DV,
Ladurner, P, Schärer, L, McCombie, WR, Hannon, GJ*, <b>Schatz MC*</b> (2015) <i>PNAS</i> doi: 10.1073/pnas.1516718112 </td>
<td valign=top> <a href="http://www.pnas.org/content/early/2015/09/15/1516718112.abstract"><img width=50 height=70 src="thumb/2015-Macro.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>67.</td>
<td valign=top> <a href="http://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.3578.html">Interactive analysis and assessment of single-cell copy-number variations.</a><br> 
Garvin, T, Aboukhalil, R, Kendall, J, Baslan, T, Atwal, GS, Hicks, J, Wigler, M, <b>Schatz MC</b> (2015) <i>Nature Methods</i> doi:10.1038/nmeth.3578 </td>
<td valign=top> <a href="http://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.3578.html"><img width=50 height=70 src="thumb/2014-Ginkgo.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>66.</td>
<td valign=top> <a href="http://rnajournal.cshlp.org/content/early/2015/08/31/rna.052456.115?top=1">Dual functions of Macpiwi1 in transposon silencing and stem cell maintenance in the flatworm Macrostomum lignano</a><br> 
Zhou X, Battistoni G, Demerdash OE, Gurtowski J, Wunderer J, Falciatori I, Ladurner P, <b>Schatz MC</b>, Hannon GJ, Wasik KA (2015) <i>RNA</i> doi: 10.1261/rna.052456.115</td>
<td valign=top> <a href="http://rnajournal.cshlp.org/content/early/2015/08/31/rna.052456.115?top=1"><img width=50 height=70 src="thumb/2015-Macpiwi.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>65.</td>
<td valign=top> <a href="http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002195">Big Data: Astronomical or Genomical?</a><br> 
Stephens, Z, et al. (2015) <i>PLOS Biology</i> DOI: 10.1371/journal.pbio.1002195</td>
<td valign=top> <a href="http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002195"><img width=50 height=70 src="thumb/2015-Genomical.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>64.</td>
  <td valign=top> <a href="http://www.sciencedirect.com/science/article/pii/S1055790315000470">Molecular genetic diversity and characterization of conjugation genes in the fish parasite Ichthyophthirius multifiliis</a><br> 
MacColl E, et al. (2015) <i>Molecular Phylogenetics and Evolution</i> 86:1-7. doi:10.1016/j.ympev.2015.02.017</td>
<td valign=top> <a href="http://www.sciencedirect.com/science/article/pii/S1055790315000470"><img width=50 height=70 src="thumb/2015-Ich.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>63.</td>
<td valign=top> <a href="http://genomebiology.com/2015/16/1/13">Extending reference assembly models</a><br> 
Church, DM, et al. (2015) <i>Genome Biology</i> 16:13  doi:10.1186/s13059-015-0587-3</td>
<td valign=top> <a href="http://genomebiology.com/2015/16/1/13"><img width=50 height=70 src="thumb/2015-HumanGraphGenome.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>62.</td>
<td valign=top> <a href="http://journal.frontiersin.org/Journal/10.3389/fbioe.2015.00008/abstract">The challenge of small-scale repeats for indel discovery</a><br> 
Narzisi, G, <b>Schatz MC</b> (2015) <i>Frontiers in Bioengineering and Biotechnology</i> 3:8. doi: 10.3389/fbioe.2015.00008</td>
<td valign=top> <a href="http://journal.frontiersin.org/Journal/10.3389/fbioe.2015.00008/abstract"><img width=50 height=70 src="thumb/2015-RepeatIndel.png"></a> </td> </tr>
<tr><td></td></tr>



<tr><td></td></tr>
<tr><td></td></tr>

<tr><td colspan=3><h3>2014</h3><hr></td></tr>

<tr><td valign=top align=right>61.</td>
<td valign=top> <a href="http://genomebiology.com/2014/15/11/506/abstract">Whole genome de novo assemblies of three divergent strains of rice (O. sativa) documents novel gene space of aus and indica</a><br> 
<b>Schatz MC</b>, Maron, LG, Stein, JC, Wences, AH, Gurtowski, J, Biggers, E, Lee, H, Kramer, M, Antoniou, E, Ghiban, E, Wright, MH, Chia, JM, Ware, D, McCouch, S, McCombie, WR (2014) <i>Genome Biology</i> 15:506  doi:10.1186/s13059-014-0506-z</td>
<td valign=top> <a href="http://genomebiology.com/2014/15/11/506/abstract"><img width=50 height=70 src="thumb/2014-PanRice.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>60.</td>
<td valign=top> <a href="http://bioinformatics.oxfordjournals.org/content/early/2014/11/13/bioinformatics.btu756.abstract">SplitMEM: A graphical algorithm for pan-genome analysis with suffix skips</a><br> 
Marcus, S, Lee, H, <b>Schatz MC</b> (2014) <i>Bioinformatics</i> doi: 10.1093/bioinformatics/btu756</td>
<td valign=top> <a href="http://bioinformatics.oxfordjournals.org/content/early/2014/11/13/bioinformatics.btu756.abstract"><img width=50 height=70 src="thumb/2014-SplitMEM.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>59.</td>
<td valign=top> <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature13908.html">The contribution of de novo coding mutations to autism spectrum disorder</a><br> 
Iossifov, I et al. (2014) <i>Nature</i> doi:10.1038/nature13908 </td>
<td valign=top> <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature13908.html"><img width=50 height=70 src="thumb/2014-AutismCoding.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>58.</td>
<td valign=top> <a href="http://genomemedicine.com/content/6/10/89/abstract">Reducing INDEL calling errors in whole-genome and exome sequencing data</a><br> 
Fang, H, Narzisi, G, O'Rawe, J, Wu, Y, Rosenbaum, J, Ronemus, M, Iossifov, I, <b>Schatz MC</b>, Lyon, GJ (2014) <i>Genome Medicine</i> 6:89. doi:10.1186/s13073-014-0089-z </td>
<td valign=top> <a href="http://genomemedicine.com/content/6/10/89/abstract"><img width=50 height=70 src="thumb/2014-IndelErrors.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>57.</td>
<td valign=top> <a href="http://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.3069.html">Accurate de novo and transmitted indel detection in exome-capture data using microassembly.</a><br> 
Narzisi, G, O'Rawe, JA, Iossifov, I, Fang, H, Lee, YH, Wang, Z, Wu, Y, Lyon, G, Wigler, M, <b>Schatz MC</b> (2014) <i>Nature Methods</i> doi:10.1038/nmeth.3069 </td>
<td valign=top> <a href="http://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.3069.html"><img width=50 height=70 src="thumb/2014-Scalpel.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>56.</td>
<td valign=top> <a href="http://www.gigasciencejournal.com/content/3/1/11/abstract">High-coverage sequencing and annotated assemblies of the budgerigar genome</a><br> 
Ganapathy, G et al. (2014) <i>GigaScience</i> 3:11  doi:10.1186/2047-217X-3-11 </td>
<td valign=top> <a href="http://www.gigasciencejournal.com/content/3/1/11/abstract"><img width=50 height=70 src="thumb/2014-Budgerigar.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>55.</td>
<td valign=top> <a href="http://dl.acm.org/citation.cfm?id=2616502&CFID=390393143&CFTOKEN=57975528">Large-scale sequencing and assembly of cereal genomes using Blacklight</a><br> 
Blood, P, Marcus, S, <b>Schatz MC</b> (2014) <i>XSEDE14</i> Atlanta GA, July 13-18. 2014 <a href="preprint/2014.BlacklightAssembly.pdf">[preprint]</a> </td>
<td valign=top> <a href="http://dl.acm.org/citation.cfm?id=2616502&CFID=390393143&CFTOKEN=57975528"><img width=50 height=70 src="thumb/2014-BlacklightAssembly.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>54.</td>
<td valign=top> <a href="http://www.springer.com/computer/bioinformatics/book/978-3-319-07952-3">On Algorithmic Complexity of Biomolecular Sequence Assembly Problem</a><br> 
 Narzisi, G, Mishra, B, <b>Schatz MC</b> (2014) <i>Algorithms for Computational Biology</i> Lecture Notes in Computer Science, Vol. 8542 <a href="preprint/2014.AssemblyComplexity.pdf">[preprint]</a> </td>
<td valign=top> <a href="http://www.springer.com/computer/bioinformatics/book/978-3-319-07952-3"><img width=50 height=70 src="thumb/2014-AssemblyComplexity.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td colspan=3><h3>2013</h3><hr></td></tr>

<tr><td valign=top align=right>53.</td>
<td valign=top> <a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0076376">Cultivation and Complete Genome Sequencing of Gloeobacter kilaueensis sp. nov., from a Lava Cave in Kīlauea Caldera, Hawai'i</a><br> 
Saw JHW, <b>Schatz M</b>, Brown MV, Kunkel DD, Foster JS, et al. (2013) <i>PLoS One</i> 8(10): e76376. doi:10.1371/journal.pone.0076376</td>
<td valign=top> <a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0076376"><img width=50 height=70 src="thumb/2013-Gloeobacter.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>52.</td>
<td valign=top> <a href="http://www.gigasciencejournal.com/content/2/1/10/abstract">Assemblathon 2: evaluating de novo methods of genome assembly in three vertebrate species</a><br> 
Bradnam, KR. et al (2013) <i>GigaScience</i> 2:10 doi:10.1186/2047-217X-2-10</td>
<td valign=top> <a href="http://www.gigasciencejournal.com/content/2/1/10/abstract"><img width=50 height=70 src="thumb/2013-Assemblathon2.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>51.</td>
<td valign=top> <a href="http://genomebiology.com/2013/14/6/405">The advantages of SMRT sequencing</a><br> 
Roberts, RJ, Carneiro, MO, and <b>Schatz, MC</b> (2013) <i>Genome Biology</i> 14:405 doi:10.1186/gb-2013-14-6-405</td>
<td valign=top> <a href="http://genomebiology.com/2013/14/6/405"><img width=50 height=70 src="thumb/2013-SMRT.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>50.</td>
<td valign=top> <a href="http://genomebiology.com/2013/14/6/124">The DNA60IFX Contest</a><br> 
<b>Schatz, MC</b>, Taylor, J and Schelhorn, S (2013) <i>Genome Biology</i> 14:124 doi:10.1186/gb-2013-14-6-124</td>
<td valign=top> <a href="http://genomebiology.com/2013/14/6/124"><img width=50 height=70 src="thumb/2013-DNA60IFX.jpg"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>49.</td>
<td valign=top> <a href="http://spectrum.ieee.org/biomedical/devices/the-dna-data-deluge">The DNA Data Deluge</a><br> 
<b>Schatz, MC</b> and Langmead, B (2013) <i>IEEE Spectrum</i> July, 2013</td>
<td valign=top> <a href="http://spectrum.ieee.org/biomedical/devices/the-dna-data-deluge"><img width=50 height=70 src="thumb/2013-IEEESpectrum.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>48.</td>
<td valign=top> <a href="http://genomebiology.com/2013/14/5/R41/">Genome of the long-living sacred lotus (Nelumbo nucifera Gaertn.)</a><br> 
Ming, R et al. (2013) <i>Genome Biology</i> 14:R41</td>
<td valign=top> <a href="http://genomebiology.com/2013/14/5/R41/abstract"><img width=50 height=70 src="thumb/2013-Lotus.jpg"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>47.</td>
<td valign=top> <a href="http://genomebiology.com/2013/14/4/113">Sixty years of genome biology</a><br> 
Doolittle, WF et al. (2013) <i>Genome Biology</i> 14:113</td>
<td valign=top> <a href="http://genomebiology.com/2013/14/4/113"><img width=50 height=70 src="thumb/2013-DNA60.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>46.</td>
<td valign=top> <a href="http://www.pnas.org/content/early/2013/03/07/1220766110.abstract">Aluminum tolerance in maize is associated with higher MATE1 gene copy number</a><br> 
Maron, LG et al. (2013) <i>PNAS</i> doi: 10.1073/pnas.1220766110</td>
<td valign=top> <a href="http://www.pnas.org/content/early/2013/03/07/1220766110.abstract"><img width=50 height=70 src="thumb/2013-MaizeCNV.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td colspan=3><h3>2012</h3><hr></td></tr>

<tr><td valign=top align=right>45.</td>
<td valign=top> <a href="http://genomebiology.com/2012/13/11/177">Computational thinking in the era of big data biology</a><br> 
<b>Schatz, MC</b> (2012) <i>Genome Biology</i> 13:177</td>
<td valign=top> <a href="http://genomebiology.com/2012/13/11/177"><img width=50 height=70 src="thumb/2012-CompThinking.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>44.</td>
<td valign=top> <a href="http://onlinelibrary.wiley.com/doi/10.1002/cpe.2925/abstract">Answering the demands of digital genomics</a><br> 
Titmus, MA, Gurtowski, J,  <b>Schatz, MC</b> (2012) <i>Concurrency and Computation: Practice and Experience</i> DOI: 10.1002/cpe.2925</td>
<td valign=top> <a href="http://onlinelibrary.wiley.com/doi/10.1002/cpe.2925/abstract"><img width=50 height=70 src="thumb/2012-DigitalGenomics.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>43.</td>
<td valign=top> <a href="http://www.world-academy-of-science.org/worldcomp12/ws/program/bic19">De novo identification of 'heterotigs' towards accurate and in-phase assembly 
of complex plant genomes</a><br> 
Price J, et al. (2012) <i>Proceedings of BIOCOMP'12</i> Las Vegas, NV. [<a href="preprint/2012.Heterotig.pdf">preprint</a>]</td>
<td valign=top> <a href="http://www.world-academy-of-science.org/worldcomp12/ws/program/bic19"><img width=50 height=70 src="thumb/2012-Heterotig.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>42.</td>
<td valign=top> <a href="http://onlinelibrary.wiley.com/doi/10.1002/0471250953.bi1503s39/full">Genotyping in the Cloud with Crossbow</a><br> 
Gurtowski, J,  <b>Schatz, MC</b>, Langmead, B (2012) <i>Current Protocols in Bioinformatics</i> 39:15.3.1-15.3.15</td>
<td valign=top> <a href="http://onlinelibrary.wiley.com/doi/10.1002/0471250953.bi1503s39/full"><img width=50 height=70 src="thumb/2012-CrossbowProtocol.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>41.</td>
<td valign=top> <a href="http://www.gigasciencejournal.com/content/1/1/4">The rise of a digital immune system</a><br> 
<b>Schatz, MC</b>, Phillippy, AM (2012) <i>GigaScience</i> 1:4</td>
<td valign=top> <a href="http://www.gigasciencejournal.com/content/1/1/4"><img width=50 height=70 src="thumb/2012-DigitalImmune.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>40.</td>
<td valign=top> <a href="http://www.nature.com/nbt/journal/vaop/ncurrent/full/nbt.2280.html">Hybrid error correction and de novo assembly of single-molecule sequencing reads</a><br> 
Koren, S, <b>Schatz, MC</b>, Walenz, BP, Martin, J, Howard, JT, Ganapathy, G, Wang, Z, Rasko, DA, McCombie, WR, Jarvis, ED, Phillippy, AM (2012) <i>Nature Biotechnology.</i> doi:10.1038/nbt.2280</td>
<td valign=top> <a href="http://www.nature.com/nbt/journal/vaop/ncurrent/full/nbt.2280.html"><img width=50 height=70 src="thumb/2012-PacBioEC.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>39.</td>
<td valign=top> <a href="http://bioinformatics.oxfordjournals.org/content/28/16/2097">Genomic Dark Matter: The reliability of short read mapping illustrated by the Genome Mappability Score</a><br> 
Lee, H, <b>Schatz, MC</b> (2012) <i>Bioinformatics.</i> 10.1093/bioinformatics/bts330 </td>
<td valign=top> <a href="http://bioinformatics.oxfordjournals.org/content/28/16/2097"><img width=50 height=70 src="thumb/2012-GMS.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>38.</td>
<td valign=top> <a href="http://genomebiology.com/2012/13/4/243">Current challenges in de novo plant genome sequencing and assembly</a><br> 
<b>Schatz, MC</b>, Witkowski, J, McCombie, WR (2012) <i>Genome Biology.</i> 12:243</td>
<td valign=top> <a href="http://genomebiology.com/2012/13/4/243"><img width=50 height=70 src="thumb/2012-PlantGenome.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>37.</td>
<td valign=top> <a href="http://www.cell.com/neuron/abstract/S0896-6273(12)00340-6">De novo gene disruptions in children on the autism spectrum</a><br> 
Iossifov, I. et al. (2012) <i>Neuron.</i> 74:2 285-299</td>
<td valign=top> <a href="http://www.cell.com/neuron/abstract/S0896-6273(12)00340-6"><img width=50 height=70 src="thumb/2012-DenovoAutism.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td colspan=3><br><h3>2011</h3><hr></td></tr>

<tr><td valign=top align=right>36.</td>
<td valign=top> <a href="http://www.pnas.org/content/109/2/639">Rate of meristem maturation determines inflorescence architecture in tomato</a><br> 
Park, SJ, Jiang, K,  <b>Schatz, MC</b>, Lippman, ZB (2011) <i>PNAS.</i> doi: 10.1073/pnas.1114963109 </td>
<td valign=top> <a href="http://www.pnas.org/content/109/2/639"><img width=50 height=70 src="thumb/2011-Tomato.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>35.</td>
<td valign=top> <a href="http://bib.oxfordjournals.org/content/early/2011/12/23/bib.bbr074.full">Hawkeye and AMOS: visualizing and assessing the quality of genome assemblies</a><br> 
  <b>Schatz, MC</b>, et al. (2011) <i>Briefings in Bioinformatics.</i> doi: 10.1093/bib/bbr074 </td>
<td valign=top> <a href="http://bib.oxfordjournals.org/content/early/2011/12/23/bib.bbr074.full"><img width=50 height=70 src="thumb/2011-Hawkeye2.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>34.</td>
<td valign=top> <a href="http://genome.cshlp.org/content/22/3/557">GAGE: A critical evaluation of genome assemblies and assembly algorithms</a><br> 
  Salzberg, SL, et al. (2011) <i>Genome Research.</i> doi: 10.1101/gr.131383.111 </td>
<td valign=top> <a href="http://genome.cshlp.org/content/22/3/557"><img width=50 height=70 src="thumb/2011-GAGE.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>33.</td>
<td valign=top> <a href="http://www.pnas.org/content/108/51/E1423.short">Complex microbiome underlying secondary and primary metabolism in the tunicate-Prochloron symbiosis.</a><br> 
  Donia, MS, et al. (2011) <i>PNAS.</i> doi: 10.1073/pnas.1111712108 </td>
<td valign=top> <a href="http://www.pnas.org/content/108/51/E1423.short"><img width=50 height=70 src="thumb/2011-Procholoron.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>32.</td>
<td valign=top> <a href="http://genome.cshlp.org/content/21/12/2224">Assemblathon 1: A competitive assessment of de novo short read assembly methods.</a><br> 
  Earl, DA, et al. (2011) <i>Genome Research.</i> doi: 10.1101/gr.126599.111 </td>
<td valign=top> <a href="http://genome.cshlp.org/content/21/12/2224"><img width=50 height=70 src="thumb/2011-Assemblathon1.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>31.</td>
<td valign=top> <a href="http://jb.asm.org/cgi/content/abstract/JB.05262-11v1">Two new complete genome sequences offer insight into host and tissue specificity of plant pathogenic Xanthomonas spp.</a><br> 
  Bogdanove, AJ, et al. (2011) <i>Journal of Bacteriology.</i> doi:10.1128/JB.05262-11 </td>
<td valign=top> <a href="http://jb.asm.org/cgi/content/abstract/JB.05262-11v1"><img width=50 height=70 src="thumb/2011-Xanthamonas.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>30.</td>
<td valign=top> <a href="http://portal.acm.org/citation.cfm?id=1996104">Rapid Parallel Genome Indexing with MapReduce.</a><br> 
  Menon, RK, Bhat, GP, <b>Schatz, MC</b> (2011) <i>Proceedings of the 2nd International Workshop on MapReduce and its applications.</i> HPDC'11, San Jose, CA.  [<a href="preprint/2011-GenomeIndexingMapReduce.pdf">preprint</a>]</td>
<td valign=top> <a href="preprint/2011-GenomeIndexingMapReduce.pdf"><img width=50 height=70 src="thumb/2011-GenomeIndexingMapReduce.jpg"></a> </td> </tr>
<tr><td></td></tr>

<tr><td colspan=3><br><h3>2010</h3><hr></td></tr>

<tr><td valign=top align=right>29.</td>
<td valign=top> <a
href="http://genomebiology.com/2010/11/11/R116/abstract">Quake: quality-aware detection and correction of sequencing reads.</a><br>
Kelley, DR, <b>Schatz, MC</b>, Salzberg, SL (2010) <i>Genome Biology.</i> 11:R116 </td>
<td valign=top> <a href="http://genomebiology.com/2010/11/11/R116/abstract"><img width=50 height=70 src="/images/quake.png"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>28.</td>
<td valign=top> <a
href="http://www.biomedcentral.com/1471-2164/11/602">Genomic survey of the ectoparasitic mite Varroa destructor, a major pest of the honey bee Apis mellifera.</a><br>
Cornman, SR, <b>Schatz, MC</b>, et al. (2010) <i>BMC Genomics.</i> 11:602 </td>
<td valign=top> <a href="http://www.biomedcentral.com/1471-2164/11/602"><img width=50 height=70 src="/images/varroa.jpg"></a> </td> </tr>
<tr><td></td></tr>


<tr><td valign=top align=right>27.</td>
<td valign=top> <a
href="http://www.plosbiology.org/article/info%3Adoi%2F10.1371%2Fjournal.pbio.1000475">Multi-Platform Next-Generation Sequencing of the Domestic Turkey (Meleagris gallopavo): Genome Assembly and Analysis.</a><br>
Dalloul, RA, et al. (2010) <i>PLoS Biology.</i> 8(9): e1000475. </td>
<td valign=top> <a href="http://www.plosbiology.org/article/info%3Adoi%2F10.1371%2Fjournal.pbio.1000475"><img width=50 height=70 src="/images/turkey.jpg"></a> </td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>26.</td>
<td valign=top> <a
href="http://genomebiology.com/2010/11/8/128">The missing graphical user interface for genomics.</a><br>
<b>Schatz, MC</b>. (2010) <i>Genome Biology</i>. 11:128</td>
<td valign=top><a href="http://genomebiology.com/2010/11/8/128"><img width=50 height=70 src="/images/galaxyreview.jpg"></a></td> </tr>
<tr><td></td></tr>

<tr><td valign=top align=right>25.</td>
<td valign=top> <a href="http://portal.acm.org/citation.cfm?id=1830252.1830263">Design patterns for efficient graph algorithms in MapReduce.</a><br>
Lin, J., <b>Schatz, MC</b>. (2010) <i>Proceedings of the Eighth Workshop on Mining and Learning with Graphs Workshop (MLG-2010) </i>. </td>
<td valign=top><a href="http://portal.acm.org/citation.cfm?id=1830252.1830263"><img width=50 height=70 src="/images/mrgraph.jpg"></a></td></tr>
<tr><td></td></tr>

<tr><td valign=top align=right>24.</td>
<td valign=top> <a href="http://www.nature.com/nbt/journal/v28/n7/full/nbt0710-691.html">Cloud Computing and the DNA Data Race.</a><br>
<b>Schatz, MC</b>, Landmead, B, Salzberg, SL. (2010) <i>Nature Biotechnology</i>. 28:691-693</td>
<td valign=top><a href="http://www.nature.com/nbt/journal/v28/n7/full/nbt0710-691.html"><img width=50 height=70 src="/images/datarace.png"></a></td></tr>
<tr><td></td></tr>

<tr><td valign=top align=right>23.</td>
<td valign=top> <a href="http://aem.asm.org/cgi/content/abstract/AEM.00303-10v1">Integrated microbial survey analysis of prokaryotic communities for the PhyloChip microarray.</a><br>
<b>Schatz, MC</b>, Phillippy, AM, Gajer, P, DeSantis, TZ, Anderson, GL, Ravel, J. (2010) <i>Applied and Environmental Microbiology</i>.</td>
<td valign=top><a href="http://aem.asm.org/cgi/content/abstract/AEM.00303-10v1"><img width=50 height=70 src="/images/phylotrac.png"></a></td></tr>
<tr><td></td></tr>

<tr><td valign=top align=right>22.</td>
<td valign=top> <a href="http://genome.cshlp.org/content/20/9/1165"> Assembly of large genomes using second-generation sequencing.</a><br>
  <b>Schatz, MC</b>, Delcher, AL, Salzberg, SL. (2010) <i>Genome Research</i>, 20: 1165-1173. </td>
<td valign=top><a href="http://genome.cshlp.org/content/20/9/1165"><img width=50 height=70 src="/images/OverlapGraph.png"></a></td></tr>
<tr><td></td></tr>

<tr><td valign=top align=right>21.</td>
<td valign=top> <a href="http://www.biomedcentral.com/1471-2105/11/21"> Assembly complexity of prokaryotic genomes using short reads.  </a><br>
Kingsford, C, <b>Schatz, MC</b>, Pop, M (2010) <i>BMC Bioinformatics</i> 11:21.</td>
<td valign=top><a href="http://www.biomedcentral.com/1471-2105/11/21"><img width=50 height=70 src="/images/Complexity.png"></a></td></tr>
<tr><td></td></tr>

<tr><td colspan=3><br><h3>2009</h3><hr></td></tr>

<tr><td valign=top align=right>20.</td>
<td valign=top> <a href="http://genomebiology.com/2009/10/11/R134"> Searching for SNPs with cloud computing.  </a><br>
Langmead, B, <b>Schatz, MC</b>, Lin, J, Pop, M, Salzberg, SL (2009) <i>Genome Biology</i> 10:R134.</td>
<td valign=top><a href="http://genomebiology.com/2009/10/11/R134"><img width=50 height=70 src="/images/crossbow.png"></a></td></tr>
<tr><td></td></tr>


<tr><td valign=top align=right>19.</td>
<td valign=top> <a href="http://dx.doi.org/10.1016/j.parco.2009.05.002"> Optimizing data intensive GPGPU computations for DNA sequence alignment.  </a><br>
Trapnell, C, <b>Schatz, MC</b> (2009) <i>Parallel Computing</i> 35(8-9):429-440.</td>
<td valign=top><a href="http://dx.doi.org/10.1016/j.parco.2009.05.002"><img width=50 height=70 src="/images/mummergpu2.jpg"></a></td></tr>
<tr><td></td></tr>


<tr><td valign=top align=right>18.</td>
<td valign=top> <a href="http://www.plospathogens.org/article/info%3Adoi%2F10.1371%2Fjournal.ppat.1000466">Genomic Analyses of the Microsporidian Nosema ceranae, an Emergent Pathogen of Honey Bees.</a><br>
Cornman, RS, Chen, YP, <b>Schatz, MC</b>, et al. (2009) <i>PLoS Pathogens</i> 5(6):e1000466.</td>
<td valign=top><a href="http://www.plospathogens.org/article/info%3Adoi%2F10.1371%2Fjournal.ppat.1000466"><img width=50 height=70 src="/images/nosema.jpg"></a></td></tr>
<tr><td></td></tr>

<tr><td valign=top align=right>17.</td>
<td valign=top> <a href="http://genomebiology.com/2009/10/4/R42">A whole-genome assembly of the domestic cow, Bos taurus.</a><br> 
Zimin, AV, Delcher, AL, Florea, L, Kelley, DR, <b>Schatz, MC</b>, et al. (2009) <i>Genome Biology</i> 10:R42.</td>
<td valign=top><a href="http://genomebiology.com/2009/10/4/R42"><img width=50 height=70 src="/images/cow.gif"></a></td></tr>
<tr><td></td></tr>

<tr><td valign=top align=right>16.</td>
<td valign=top> <a href="http://bioinformatics.oxfordjournals.org/cgi/content/abstract/btp236">CloudBurst: Highly Sensitive Read Mapping with MapReduce.</a> <br>
<b>Schatz, MC</b> (2009) <i>Bioinformatics</i> 25:1363-1369.</td>
<td valign=top><a href="http://bioinformatics.oxfordjournals.org/cgi/content/abstract/btp236"><img width=50 height=70 src="/images/CloudBurst.png"></a></td></tr>
<tr><td></td></tr>

<tr><td valign=top align=right>15.</td>
<td valign=top> <a href="http://genomebiology.com/2008/9/12/R183">Comparative genomics of mutualistic viruses of Glyptapanteles parasitic wasps.</a> <br>
Desjardins, CA, et al. (2009) <i>Genome Biology</i> 9:R183.</td>
<td valign=top><a href="http://genomebiology.com/2008/9/12/R183"><img width=50 height=70 src="/images/proviral2.png"></a></td></tr>
<tr><td></td></tr>

<tr><td colspan=3><br><h3>2008</h3><hr></td></tr>

<tr><td valign=top align=right>14.</td>
<td valign=top> <a href="http://www.springerlink.com/content/136q7771qt107p41/"> Characterization of Insertion Sites in Rainbow Papaya, the First Commercialized Transgenic Fruit Crop.</a> <br>
Suzuki, JY, et al. (2008) <i>Tropical Plant Biology</i> 1:293-309.</td>
<td valign=top><a href="http://www.springerlink.com/content/136q7771qt107p41/"><img width=50 height=70 src="/images/tpb.papaya.jpg"></a></td></tr>
<tr><td></td></tr>

<tr><td valign=top align=right>13.</td>
<td valign=top> <a href="http://www.liebertonline.com/doi/abs/10.1089/cmb.2008.11TT">Revealing Biological Modules via Graph Summarization.</a><br>
Navlakha, S, <b>Schatz, M</b>, Kingsford, C. (2008) <i>Journal of Computational Biology</i> 16(2): 253-264.</td>
<td valign=top><a href="http://www.cbcb.umd.edu/~carlk/papers/GSmodules-preprint.pdf"><img width=50 height=70 src="/images/graph-summarization.png"></a></td></tr>
<tr><td></td></tr>

<tr> <td valign=top align=right>12.</td>
<td valign=top> <a href="http://www.biomedcentral.com/1471-2164/9/204">Genome sequence and rapid evolution of the rice pathogen Xanthomonas oryzae pv. oryzae PXO99A.</a><br>
Salzberg, SL, Sommer DS, <b>Schatz, MC</b>, et al. (2008)  <i>BMC Genomics</i> 9:204.</td>
<td valign=top><a href="http://www.biomedcentral.com/1471-2164/9/204"><img width=50 height=70 src="/images/xoc.png"></a></td></tr>
<tr><td></td></tr>

<tr><td valign=top align=right>11.</td>
<td valign=top> <a href="http://www.nature.com/nature/journal/v452/n7190/full/nature06856.html">The draft genome of the transgenic tropical fruit tree papaya (Carica papaya Linnaeus).</a> <br>
Ming, R, et al. (2008)  <i>Nature</i> 452, 991-996.</td>
<td valign=top><a href="http://www.nature.com/nature/journal/v452/n7190/full/nature06856.html"><img width=50 height=70 src="/images/papaya.jpg"></a></td></tr>
<tr><td></td></tr>

<tr><td valign=top align=right>10.</td>
<td valign=top> <a href="http://genomebiology.com/2008/9/3/R55/abstract">Genome Assembly forensics: finding the elusive mis-assembly.</a><br>
Phillippy, AM, <b>Schatz, MC,</b> Pop, M. (2008)  <i>Genome Biology</i> 9:R55.</td>
<td valign=top><a href="http://genomebiology.com/2008/9/3/R55/abstract"><img width=50 height=70 src="/images/forensics.png"></a></td></tr>
<tr><td></td></tr>

<tr><td colspan=3><br><h3>2007</h3><hr></td></tr>

<tr><td valign=top align=right>9.</td>
<td valign=top>  <a href="http://www.biomedcentral.com/1471-2105/8/474/abstract">High-throughput sequence alignment using Graphics Processing Units.</a><br>
<b>Schatz, MC</b>, Trapnell, C, Delcher, AL, Varshney, A. (2007)  <i>BMC Bioinformatics</i> 8:474.</td>
<td valign=top><a href="http://www.biomedcentral.com/1471-2105/8/474/abstract"><img width=50 height=70 src="/images/node_reorder.png"></a></td></tr>
<tr><td></td></tr>

<tr><td valign=top align=right>8.</td>
<td valign=top><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&Cmd=ShowDetailView&TermToSearch=17994087&ordinalpos=1&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_RVDocSum">Evolution of genes and genomes on the Drosophila phylogeny.</a><br> 
Drosophila 12 Genomes Consortium. (2007) <i>Nature</i> Nov 8;450(7167):203-18.</td>
<td valign=top><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&Cmd=ShowDetailView&TermToSearch=17994087&ordinalpos=1&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_RVDocSum"><img width="50" height="70" src="/images/flycover.jpg"></a></td></tr>
<tr><td></td></tr>

<tr><td valign=top align=right>7.</td>
<td valign=top>  <a href="http://www.sciencemag.org/cgi/content/abstract/317/5845/1756">Draft Genome of the Filarial Nematode Parasite Brugia malayi.</a><br>
Ghedin, E, et al. (2007)  <i>Science</i> 317(5845):1756-1760.</td>
<td valign=top><a href="http://www.sciencemag.org/cgi/content/abstract/317/5845/1756"><img width=50 height=70 src="/images/brugia.png"></a></td></tr>
<tr><td></td></tr>

<tr><td valign=top align=right>6.</td>
<td valign=top>  <a href="http://www.biomedcentral.com/1471-2180/7/61/abstract">Structure and evolution of a proviral locus of Glyptapanteles indiensis bracovirus.</a><br>
Desjardins, CA, et al. (2007)  <i>BMC Microbiology</i> 7:61</td>
<td valign=top><a href="http://www.biomedcentral.com/1471-2180/7/61/abstract"><img width=50 height=70 src="/images/proviral.png"></a></td></tr>
<tr><td></td></tr>

<tr><td valign=top align=right>5.</td>
<td valign=top>  <a href="http://www.sciencemag.org/cgi/content/full/sci;316/5832/1718">Genome Sequence of Aedes aegypti, a Major Arbovirus Vector.</a> <br>
Nene, V et al. (2007) <i>Science</i> 316(5832), 1718-1723.</td>
<td valign=top><a href="http://www.sciencemag.org/content/vol316/issue5832/cover.dtl"><img width=50 height=70 src="/images/aedes.cover.gif"></a></td></tr>
<tr><td></td></tr>

<tr><td valign=top align=right>4.</td>
<td valign=top>  <a href="http://genomebiology.com/2007/8/3/R34">Hawkeye: a visual analytics tool for genome assemblies.</a> <br>
<b>Schatz, MC</b>, Phillippy, AM, Shneiderman, B, Salzberg, SL. (2007) <i>Genome Biology</i> 8:R34.</td>
<td valign=top><a href="http://genomebiology.com/2007/8/3/R34"><img width=50 height=70 src="/images/scaffold.png"></a></td></tr>
<tr><td></td></tr>

<tr><td valign=top align=right>3.</td>
<td valign=top>  <a href="http://www.sciencemag.org/cgi/content/abstract/315/5809/207">Draft Genome Sequence of the Sexually Transmitted Pathogen Trichomonas vaginalis.</a> <br>
Carlton, JM, Hirt, RP, Silva, JC, Delcher, AL, <b>Schatz, M</b>, et al. (2007)  <i>Science</i> 315 (5809), 207-212.</td>
<td valign=top><a href="http://www.sciencemag.org/content/vol315/issue5809/cover.dtl"><img width=50 height=70 src="/images/tvg.cover.gif"></a></td></tr>
<tr><td></td></tr>

<tr><td colspan=3><br><h3>2006 & earlier</h3><hr></td></tr>

<tr><td valign=top align=right>2.</td>
<td valign=top>  <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=15660156">Major structural differences and novel potential virulence mechanisms from the genomes of multiple campylobacter species.</a> <br>
Fouts, DE et al. (2005)  <i>PLoS Biology</i> 3 (1):e15.</td>
<td valign=top><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?cmd=Retrieve&db=PubMed&dopt=Citation&list_uids=15660156"><img width=50 height=70 src="/images/camp.jpg"></a></td></tr>
<tr><td></td></tr>

<tr><td valign=top align=right>1.</td>
<td valign=top>  <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=14744981">Automated correction of genome sequence errors.</a> <br>
Gajer, P, <b>Schatz, M</b>, Salzberg, SL. (2004)  <i>Nucleic Acids Research</i> 32 (2):562-569.</td>
<td valign=top><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?cmd=Retrieve&db=PubMed&dopt=Citation&list_uids=14744981"><img width=50 height=70 src="/images/autoeditor.png"></a></td></tr>
<tr><td></td></tr>

</table>

<br>
<br>


<!------------------------------------------------ ------ -->
<a name="thesis"><table width="100%" bgcolor="#e6e6e6"> <tr><th style="font-variant:small-caps; font-size:12pt; color:#003399;" align="left">Thesis</th></tr></table></a><br>

<table class="intro" width=100%>

<tr><td valign=top align=right>2.</td>
  <td valign=top>  <a href="http://cs.nyu.edu/web/Research/theses.html">Scoring-and-Unfolding Trimmed Tree Assembler: Algorithms for Assembling Genome Sequences Accurately and Efficiently.</a> <br>
Narzisi, G. (2011) <i>Ph.D. Disseration</i>, Department of Computer Science, New York University</td>
<td valign=top><a href="http://cs.nyu.edu/web/Research/theses.html"><img width=50 height=70 src="thumb/2011-NarzisiThesis.png"></a></td></tr>
<tr><td></td></tr>

<tr><td valign=top align=right>1.</td>
<td valign=top>  <a href="http://drum.lib.umd.edu/handle/1903/10351">High Performance Computing for DNA Sequence Alignment and Assembly.</a> <br>
Schatz, M.C. (2010) <i>Ph.D. Disseration</i>, Department of Computer Science, University of Maryland</td>
<td valign=top><a href="http://drum.lib.umd.edu/handle/1903/10351"><img width=50 height=70 src="/images/thesis.png"></a></td></tr>
<tr><td></td></tr>

</table>

<br>
<br>


See more in <a href="http://scholar.google.com/citations?user=rcs6IKwAAAAJ&hl=en">Google Scholar</a>.

<br>
<br>
<br>
<br>


