---
layout: default
title: Teaching - 2013
---

<a name="2013"><table width="100%" bgcolor="#e6e6e6"> <tr><th style="font-variant:small-caps; font-size:12pt; color:#003399;" align="left">2013</th></tr></table></a><br>


<!-- ########################################################################################### -->

<a name="2013QB"><h3>CSHL Quantitive Biology</h3></a> <hr>

<a href="QB.IFX0a.Schatzlab Research Projects.pdf">IFX0a. High level intro to genome assembly</a>
<p>
Introduction to de Bruijn graphs for genome assembly
</p>

<a href="QB.IFX0b.CSHL Inhouse.pdf">IFX0b. CSHL Inhouse</a>
<p>
De novo assembly with long PacBio reads, detection of indels with Scalpel
</p>

<a href="QB.IFX1.Exact Matching.pdf">IFX1. Computational Thinking: Sorting, Searching, and Indexing</a>
<p>
Introduction to binary search, suffix arrays, hashing, and the BWT
</p>

<a href="QB.IFX1.BWT Notes.pdf">IFX1b. Notes on the BWT</a>
<p>
Scribed notes on computing and searching with the BWT
</p>

<a href="QB.IFX2.Dynamic Programming.pdf">IFX2. Dynamic Programming: LIS and Sequence Alignment</a>
<p>
Application of dynamic programming for sequence alignment: longest increasing subsequence, edit distance, sequence similarity, BLAST, Dynamic Time-warping
</p>

<a href="QB.IFX2.DP Notes.pdf">IFX2b. Notes on Dynamic Programming</a>
<p>
Scribed notes on dynamic programming
</p>

<a href="QB.IFX3.Graphs and Genomes.pdf">IFX3. Graphs and Genomes</a>
<p>
Algorithms for graph searching, detailed look at genome assembly.
</p>

<a href="QB.IFX4.Gene Finding and HMMs.pdf">IFX4. Gene Finding and HMMs</a>
<p>
Approaches to prokayotic and eukaryotic gene finding, hidden markov models, forward algorithm, viterbi algorithm.
</p>

<br>
<br>

<!-- ########################################################################################### -->

<a name="2013QBBootcamp"><h3>CSHL Quantitative Biology Bootcamp</h3></a> <hr>

<a href="QB Bootcamp 1. Programming 101.pdf">Lecture 1: Biology, Computers, and Python</a><br>
<p>
Milestones in Molecular Biology and the rise of sequencing.
Overview of computer systems, introduction to python and scripting.
</p>

<a href="QB Bootcamp 2. Sequence Analysis.pdf">Lecture 2: Sequence Alignment and Computational Thinking</a><br>
<p>
Introduction to Alignment and Algorithms, Suffix Arrays, Binary Search
</p>

<a href="QB Bootcamp 3. Genome Resources.pdf">Lecture 3: Genomic Resources</a><br>
<p>
NCBI, UCSC, CSHL Meetings and Courses, Galaxy.
</p>

<a href="QB Bootcamp 4. Unix Scripting.pdf">Lecture 4: Unix Scripting</a><br>
<p>
Introduction to Unix, searching the human genome annotation
</p>

<a href="QB Bootcamp 5. ORC Exercise.pdf">Lecture 5: Dicovering Origins of Replication</a><br>
<p>
Problem by <a href="http://www.cshl.edu/Faculty/justin-kinney.html">Justin Kinney</a>.
Background on tracking DNA replication with next-gen sequencing,
Walk-through of analysis steps, Visualization of discovered replication sites.
</p>

<a href="replication_analysis.ipynb.gz">Lecture 6: Advanced Origins of Replication Analyis [iPython Notebook]</a><br>
<p>
Problem by <a href="http://www.cshl.edu/Faculty/justin-kinney.html">Justin Kinney</a>.
Plotting, smoothing, and analyzying data
</p>

<a href="tf_binding_site_analysis.ipynb.gz">Lecture 7: Transcription Factor Binding Sites [iPython Notebook]</a><br>
<p>
Problem by <a href="http://www.cshl.edu/Faculty/justin-kinney.html">Justin Kinney</a>.
Parsing, and discovering transcription factor binding sites
</p>


<a href="exercises.zip">Python Exercises</a><br>
<p>
Exercises on working with python
</p>

<a href="exercises_solutions.zip">Python Exercise Solutions</a><br>
<p>
Solutions to exercises
</p>


<br>
<br>

<!-- ########################################################################################### -->

<a name="2013AdvSeq"><h3>CSHL Advanced Sequencing Course</h3></a> <hr>

<a href="http://bioinformatics.bc.edu/wikis/csh13/index.php/Main_Page">Course Wiki</a><br>
<p>
Schedule and archive of presentations.
</p>

<a href="AdvSeq.Whole Genome Assembly and Alignment.pdf">Whole Genome Assembly and Alignment</a><br>
<p>
De novo assembly theory and practice; whole genome alignment with MUMmer
</p>

<a href="AdvSeq.Assembly Tutorial.pdf">Assembly Tutorial</a><br>
<p>
Assembly tutorial to detect a secret message embedded into a microbial genome
</p>

<a href="AdvSeq.asm.tgz">AdvSeq.asm.tgz</a><br>
<p>
Data for assembly tutorial
</p>

<br>
<br>

<!-- ########################################################################################### -->

<a name="2013P4B"><h3>CSHL Programming for Biology</h3></a> <hr>

<a href="P4B.Whole Genome Assembly and Alignment.pdf">Whole Genome Assembly and Alignment</a><br>
<p>
De novo assembly theory and practice; whole genome alignment with MUMmer
</p>

<a href="P4B.Assembly Tutorial.pdf">Assembly Tutorial</a><br>
<p>
Assembly tutorial to detect a secret message embedded into a microbial genome
</p>

<a href="P4B.asm.challenge.tgz">P4B.asm.challenge.tgz</a><br>
<p>
Data for assembly tutorial
</p>

<br>
<br>


<!-- ########################################################################################### -->

<a name="2013URP"><h3>CSHL Undergraduate Research Program in Bioinformatics</h3></a> <hr>

<a href="2013.URP.CS Fundamentals.pdf">Lecture 1: Sequence Alignment and Computational Thinking</a><br>
<p>
In this class we explored the problem of finding exact occurrences of a query
sequence in a large genome or database of sequences.  Under this theme, we
started by analyzing the brute force approach introducing the concepts of
algorithm, complexity analysis, and E-values.  Next we discussed suffix arrays
as an index for accelerating the search, including analyzing the performance of
binary search. We also considered two traditional algorithms for sorting
(Selection Sort versus QuickSort) and their relative performance.  In the second
half of the class we discussed finding approximate occurrences of a short query
sequence in a large genome or database of sequences. We first defined the
problem by considering various metrics of an approximate occurrence such as
hamming distance, or edit distance.  We then considered different methods for
computing inexact alignments including brute force global &amp; local
alignments, and seed-and-extend algorithms. Finally we discussed Bowtie as a
Burrows-Wheeler transform based short read mapping algorithm for discovering
alignments to reference genome.
</p>

<br>

<a href="2013.URP.Sequencing Lunch.pdf">Lecture 2: Sequencing Pitfalls</a><br>
<p>
In this session we reviewed the currently available sequencing technologies and best practices,
focusing on the widely used Illumina sequencing platform, the up and coming PacBio
sequencing platform, and the recently announced Oxford Nanopore instruments. Special
attention was placed on the complexities and biases with Illumina.
</p>

<br>

<a href="2013.URP.Graphs and Genomes.pdf">Lecture 3: Graphs and Genomes</a><br>
<p>
The theme of this class was graphs and methods for graph analysis. The emphasis
was on genome assembly but included a discussion of other biological networks
including PPI networks, regulation networks, neuron interaction networks, and
cell cycle graphs. In the class, we considered fundamental properties of graphs,
such as nodes, edges, degrees, and shortest paths. We then examined in detail
algorithms for searching graphs with a  with breadth-first-search, and then
approaches for finding minimum cost paths through weighed graphs (traveling
salesman problem), including exhaustive search, greedy algorithms, and
branch-and-bound. This lead to a discussion of the intractable nature of
NP-complete problems, and reviewed several important examples (vertex cover,
clique finding, knapsack problem, Hamiltonian cycle).
</p>

<br>
<br>

<!-- ########################################################################################### -->

<a name="2013SBUGG"><h3>SBU Graduate Genetics</h3></a> <hr>

<a href="2013.10.29.SBU.Grad Genetics.NextGen Sequence Analysis.pdf">Next gen sequence analysis</a>
<p>
Rise of sequencing; brute-force matching, binary search, genetics of autism.
</p>

<a href="2013.11.05.SBU.Grad Genetics.Assembly.pdf">Whole genome assembly</a>
<p>
Review of *-seq assays, assembly theory, ALLPATHS-LG, Celera Assembler, PacBio.
</p>

<br>
<br>

<!-- ########################################################################################### -->

<a name="2013SBUCS"><h3>SBU Intro to Computational Biology</h3></a> <hr>

<a href="2013.10.24.SBU Comp Bio BWT.pdf">Next-gen sequence analysis</a><br>
<p>
Rise of sequencing; alignment with the BWT; genetic of autism
</p>

<a href="2013.10.24.SBU.BWT Notes.pdf">Lecture notes on the BWT</a><br>
<p>
BWT construction, unwinding, exact match
</p>

<br>
<br>


<!-- ########################################################################################### -->

<a name="2013iPQB"><h3>SBU Introduction to Physical and Quantitative Biology</h3></a> <hr>

<a href="2013.09.23.SBU.Intro to Physical and QB.pdf">Sequence Alignment and Computational Thinking</a><br>
<p>
Milestones in Molecular Biology and the rise of sequencing. Algorithms for searching and aligning sequences
</p>

<br>
<br>

