---
layout: default
title: Schatzlab - Breast Cancer Organoid
---

<span class="contenthead">Comprehensive analysis of structural variants in breast cancer genomes using single molecule sequencing</span><br>
<hr>

Sergey Aganezov, Sara Goodwin, Rachel Sherman, Fritz J. Sedlazeck, Gayatri Arun, Sonam Bhatia, Isac Lee, Melanie Kirsche, Robert Wappel, Melissa Kramer, Karen Kostroff, David L. Spector, Winston Timp, W. Richard McCombie, Michael C. Schatz<br><br>


<table class="intro">
<tr>
<td valign=top>
Improved identification of structural variants (SVs) in cancer can lead to more targeted and effective treatment options as well as advance our basic understanding of disease progression. We performed whole genome sequencing of the SKBR3 breast cancer cell-line and patient-derived tumor and normal organoids from two breast cancer patients using 10X/Illumina, PacBio, and Oxford Nanopore sequencing. We then inferred SVs and large-scale allele-specific copy number variants (CNVs) using an ensemble of methods. Our findings demonstrate that long-read sequencing allows for substantially more accurate and sensitive SV detection, with between 90% and 95% of variants supported by each long-read technology also supported by the other. We also report high accuracy for long-reads even at relatively low coverage (25x-30x). Furthermore, we inferred karyotypes from these data using our enhanced RCK algorithm to present a more accurate representation of the mutated cancer genomes, and find hundreds of variants affecting known cancer-related genes detectable only through long-read sequencing. These findings highlight the need for long-read sequencing of cancer genomes for the precise analysis of their genetic instability.</td>
<td width=150 valign="top">
<img src="overview.jpg" width=150>
</td></tr></table>
  
  
<br><br><br>

<span class="contenthead">Analysis Manuscripts</span> <br>

<table class="intro" width=100%>
  
<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top><a href="https://www.biorxiv.org/content/10.1101/847855v1">Comprehensive analysis of structural variants in breast cancer genomes using single molecule sequencing</a><br>
  Aganezov et al. (2019) bioRxiv. doi: https://doi.org/10.1101/847855<td>
</tr> <tr><td height=8 colspan=2></td></tr>

<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top><a href="https://www.biorxiv.org/content/10.1101/560839v1">Reconstruction of clone- and haplotype-specific cancer genome karyotypes from bulk tumor samples</a><br>
  Aganezov et al. (2019) bioRxiv. doi: https://doi.org/10.1101/560839<td>
</tr> <tr><td height=8 colspan=2></td></tr>

</table>

<br>

<span class="contenthead">Method Availability</span> <br>

The SV inference and comparison workflow is implemented with Snakemake v5.5.4 and is available at <a href="http://github.com/aganezov/EnsembleSV">github.com/aganezov/EnsembleSV</a><br>
RCK v1.1 utilized for cancer genome karyotype inference is available at <a href="http://github.com/aganezov/EnsembleSV">github.com/aganezov/RCK</a><br>

<br>

<span class="contenthead">Data Availability</span> <br>

Raw sequencing data for SKBR3 are available within the SRA under BioProject <a href="https://www.ncbi.nlm.nih.gov/bioproject/476239">PRJNA476239</a>. <br>
Patient data are available within dbGaP under accession phs038843.v1<br><br>

<span class="contenthead">SKBR3 Merged Analysis</span> <br>
<table class="intro" width=100%>
<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top>SKBR3.specific.all.vcf: <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/bcorganoid/SKBR3.specific.all.vcf">SKBR3.specific.all.vcf</a></td>
</tr> <tr><td height=8 colspan=2></td></tr>
</table>  

<span class="contenthead">SKBR3 PacBio-based Analysis</span> <br>
<table class="intro" width=100%>
<tr> <td valign=top>PacBio reads aligned with NGMLR:</td> <td><a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/reads_lr_skbr3.fa_ngmlr-0.2.3_mapped.bam">reads_lr_skbr3.fa_ngmlr-0.2.3_mapped.bam</a></td> </tr>
<tr> <td valign=top>PacBio pbsv VCF:</td>                 <td><a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/bcorganoid/SKBR3_PACBIO_pbsv.vcf">SKBR3_PACBIO_pbsv.vcf</a></td></tr>
<tr> <td valign=top>PacBio Sniffles VCF:</td>             <td><a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/bcorganoid/SKBR3_PACBIO_sniffles.vcf">SKBR3_PACBIO_sniffles.vcf</a></td></tr>
</table>  

<br>

<span class="contenthead">SKBR3 Oxford Nanopore-based Analysis</span> <br>
<table class="intro" width=100%>
<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top>Oxford Nanopore reads aligned with NGMLR:<br> <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/skbr3.ont.sort.bam">skbr3.ont.sort.bam</a><td>
  <td valign=top>ONT pbsv VCF:<br>     <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/bcorganoid/SKBR3_ONT_pbsv.vcf">SKBR3_ONT_pbsv.vcf</a><td>
  <td valign=top>ONT Sniffles VCF:<br> <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/bcorganoid/SKBR3_ONT_sniffles.vcf">SKBR3_ONT_sniffles.vcf</a><td>
</tr> <tr><td height=8 colspan=2></td></tr>
</table>

<br>

<span class="contenthead">SKBR3 10X Genomics-based Analysis</span> <br>
<table class="intro" width=100%>
<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top>10X Genomics Linked Reads aligned with LongRanger:<br> <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/skbr3.10x.sort.bam">skbr3.10x.sort.bam</a><td>
  <td valign=top>10X GrocSV VCF:<br> <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/bcorganoid/SKBR3_10x_grocsvs.vcf">SKBR3_10x_grocsvs.vcf</a><td>
  <td valign=top>10X SVABA Indel VCF:<br> <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/bcorganoid/SKBR3_10x_indel_svaba.vcf">SKBR3_10x_indel_svaba.vcf</a><td>
  <td valign=top>10X SVABA SVs VCF:<br> <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/bcorganoid/SKBR3_10x_sv_svaba.vcf">SKBR3_10x_sv_svaba.vcf</a><td>
  <td valign=top>10X LongRanger Dels VCF:<br> <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/bcorganoid/SKBR3_10x_longranger_dels.vcf">SKBR3_10x_longranger_dels.vcf</a><td>
  <td valign=top>10X LongRanger Large SVs VCF:<br> <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/bcorganoid/SKBR3_10x_longranger_large_svs.vcf">SKBR3_10x_longranger_large_svs.vcf</a><td>
  <td valign=top>10X Lumpy SVs VCF:<br> <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/bcorganoid/SKBR3_10x_lumpy.vcf">SKBR3_10x_lumpy.vcf</a><td>
  <td valign=top>10X Manta SVs VCF:<br> <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/bcorganoid/SKBR3_10x_manta.vcf">SKBR3_10x_manta.vcf</a><td>
</tr> <tr><td height=8 colspan=2></td></tr>
</table>

<br>

<span class="contenthead">SKBR3 Illumina PE-based Analysis</span> <br>
<table class="intro" width=100%>
<tr>
  <td valign=top width=10>&raquo;</td>
  <td valign=top>Illumina PE550 reads aligned with BWA-MEM:<br>
    <a href="http://labshare.cshl.edu/shares/schatzlab/www-data/skbr3/SKBR3_550bp_pcrFREE_S1_L001_AND_L002_R1_001.101bp.bwamem.ill.mapped.sort.bam">SKBR3_550bp_pcrFREE_S1_L001_AND_L002_R1_001.101bp.bwamem.ill.mapped.sort.bam</a><td>
</tr> <tr><td height=8 colspan=2></td></tr>
</table>
